<!--?xml version="1.0" encoding="utf-8"?--><!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd"><html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="plosone"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4280143/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Metabolomics Data Normalization with EigenMS</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4280143/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="PLoS ONE"><meta name="citation_title" content="Metabolomics Data Normalization with EigenMS"><meta name="citation_authors" content="Yuliya V. Karpievitch, Sonja B. Nikolic, Richard Wilson, James E. Sharman, Lindsay M. Edwards"><meta name="citation_date" content="2014"><meta name="citation_issue" content="12"><meta name="citation_volume" content="9"><meta name="citation_doi" content="10.1371/journal.pone.0116221"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4280143/?report=abstract"><meta name="citation_pmid" content="25549083"><meta name="DC.Title" content="Metabolomics Data Normalization with EigenMS"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Public Library of Science"><meta name="DC.Contributor" content="Yuliya V. Karpievitch"><meta name="DC.Contributor" content="Sonja B. Nikolic"><meta name="DC.Contributor" content="Richard Wilson"><meta name="DC.Contributor" content="James E. Sharman"><meta name="DC.Contributor" content="Lindsay M. Edwards"><meta name="DC.Date" content="2014"><meta name="DC.Identifier" content="10.1371/journal.pone.0116221"><meta name="DC.Language" content="en"><meta property="og:title" content="Metabolomics Data Normalization with EigenMS"><meta property="og:type" content="article"><meta property="og:description" content="Liquid chromatography mass spectrometry has become one of the analytical platforms of choice for metabolomics studies. However, LC-MS metabolomics data can suffer from the effects of various systematic biases. These include batch effects, day-to-day variations ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4280143/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-plosone.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4280143/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4280143/pdf/pone.0116221.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="8A1BE16EE8E37CD100000000021E021E.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.ncbi-alerts { background: none repeat scroll 0% 0% #671414;color: #fff;padding: .231em 1.231em;}.ncbi-alerts p { color: #fff; }.ncbi-alerts a, .ncbi-alerts a:visited {color: #fff; background: none;}.ncbi-alerts p {margin: 1em 0;padding: 0 0 0 1.5em; }.ncbi-alerts p:nth-child(1) {background-image: url('/core/icons/information.png');background-position: left top;background-repeat: no-repeat }.ncbi-alerts.subclosed p:nth-child(1) {background-image: url('/core/alerts/warning.png');}.ncbi-alerts.subclosed p { padding-left: 2em;}.ncbi-alerts.pubmed, .ncbi-alerts.pubmed p {background-color: #97B0C8; color: #000;}.ncbi-alerts a { text-decoration: underline !important; }.ncbi-alerts.pubmed a, .ncbi-alerts.pubmed a:visited { color: #000; }.ncbi-alerts, .ncbi-alerts p {background-color: #97B0C8; color: #000;}.ncbi-alerts a, .ncbi-alerts a:visited { color: #000; }</style></head>
    <body class="article" id="ui-ncbiexternallink-4">
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	&lt;p class="nojs"&gt;
	&lt;strong&gt;Warning:&lt;/strong&gt;
	The NCBI web site requires JavaScript to function. 
	&lt;a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank"&gt;more...&lt;/a&gt;
	&lt;/p&gt;
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4280143%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header" style="margin-bottom: 0px;">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/440/">PLoS One</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/245922/">v.9(12); 2014</a></li><li class="accid">PMC4280143</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-plosone.gif" alt="Logo of plosone" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,0,438,55" alt="PLoS One" title="PLoS One" href="http://www.plosone.org/" ref="reftype=publisher&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,56,104,74" alt="View this Article" title="View this Article" href="//dx.doi.org/10.1371/journal.pone.0116221" ref="reftype=publisher&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article%7CBanner&amp;TO=Content%20Provider%7CArticle%7CFullText" target="pmc_ext"><area shape="rect" coords="106,56,202,74" alt="Submit to PLoS" title="Submit to PLoS" href="http://www.plosone.org/static/checklist.action" ref="reftype=publisher&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="204,56,302,74" alt="Get E-mail Alerts" title="Get E-mail Alerts" href="http://journals.plos.org/alerts" ref="reftype=publisher&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="304,56,374,74" alt="Contact Us" title="Contact Us" href="http://www.plosone.org/static/contact.action" ref="reftype=publisher&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="440,0,500,55" alt="Public Library of Science (PLoS)" title="Public Library of Science (PLoS)" href="http://www.plos.org/" ref="reftype=publisher&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">PLoS One</a></span>. 2014; 9(12): e116221. </span></div><div><span class="fm-vol-iss-date">Published online 2014 Dec 30. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1371%2Fjournal.pone.0116221" target="pmc_ext" ref="reftype=other&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1371/journal.pone.0116221</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4280143</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/25549083">25549083</a></div></div></div></div><h1 class="content-title">Metabolomics Data Normalization with EigenMS</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Karpievitch%20YV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25549083" class="affpopup" co-rid="_co_idm139681143941088" co-class="co-affbox">Yuliya V. Karpievitch</a>,<sup>
1
,</sup><sup>
*
</sup> <a href="/pubmed/?term=Nikolic%20SB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25549083" class="affpopup" co-rid="_co_idm139681143938448" co-class="co-affbox">Sonja B. Nikolic</a>,<sup>
2
</sup> <a href="/pubmed/?term=Wilson%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25549083" class="affpopup" co-rid="_co_idm139681183101808" co-class="co-affbox">Richard Wilson</a>,<sup>
3
</sup> <a href="/pubmed/?term=Sharman%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25549083" class="affpopup" co-rid="_co_idm139681183099888" co-class="co-affbox">James E. Sharman</a>,<sup>
3
</sup> and  <a href="/pubmed/?term=Edwards%20LM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25549083" class="affpopup" co-rid="_co_idm139681180941184" co-class="co-affbox">Lindsay M. Edwards</a><sup>
4
,</sup><sup>
5
</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139681143941088"><h3 class="no_margin">Yuliya V. Karpievitch</h3><p>  <sup>1</sup>  School of Mathematics and Physics, University of Tasmania, Hobart, TAS, Australia,   </p><div>Find articles by <a href="/pubmed/?term=Karpievitch%20YV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25549083">Yuliya V. Karpievitch</a></div></div><div id="_co_idm139681143938448"><h3 class="no_margin">Sonja B. Nikolic</h3><p>  <sup>2</sup>  Menzies Research Institute Tasmania, University of Tasmania, Hobart, TAS, Australia,   </p><div>Find articles by <a href="/pubmed/?term=Nikolic%20SB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25549083">Sonja B. Nikolic</a></div></div><div id="_co_idm139681183101808"><h3 class="no_margin">Richard Wilson</h3><p>  <sup>3</sup>  Central Science Laboratory, University of Tasmania, Hobart, TAS, Australia,   </p><div>Find articles by <a href="/pubmed/?term=Wilson%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25549083">Richard Wilson</a></div></div><div id="_co_idm139681183099888"><h3 class="no_margin">James E. Sharman</h3><p>  <sup>3</sup>  Central Science Laboratory, University of Tasmania, Hobart, TAS, Australia,   </p><div>Find articles by <a href="/pubmed/?term=Sharman%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25549083">James E. Sharman</a></div></div><div id="_co_idm139681180941184"><h3 class="no_margin">Lindsay M. Edwards</h3><p>  <sup>4</sup>  Centre of Human &amp; Aerospace Physiological Sciences, King’s College London, London, United Kingdom,   </p><p>  <sup>5</sup>  Fibrosis Discovery Performance Unit, GlaxoSmithKline R&amp;D, Stevenage, United Kingdom,   </p><div>Find articles by <a href="/pubmed/?term=Edwards%20LM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25549083">Lindsay M. Edwards</a></div></div></div></div><div class="contrib-group half_rhythm fm-editor">Andrew R. Dalby, <span class="fm-role">Editor</span><sup></sup></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139681141202096_ai idm139681143941216_ai idm139681140251024_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139681141202096_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139681141202096_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139681141202096_ai" style="display:none"><div class="fm-affl" lang="en" id="aff1">  <sup>1</sup>  School of Mathematics and Physics, University of Tasmania, Hobart, TAS, Australia,   </div><div class="fm-affl" lang="en" id="aff2">  <sup>2</sup>  Menzies Research Institute Tasmania, University of Tasmania, Hobart, TAS, Australia,   </div><div class="fm-affl" lang="en" id="aff3">  <sup>3</sup>  Central Science Laboratory, University of Tasmania, Hobart, TAS, Australia,   </div><div class="fm-affl" lang="en" id="aff4">  <sup>4</sup>  Centre of Human &amp; Aerospace Physiological Sciences, King’s College London, London, United Kingdom,   </div><div class="fm-affl" lang="en" id="aff5">  <sup>5</sup>  Fibrosis Discovery Performance Unit, GlaxoSmithKline R&amp;D, Stevenage, United Kingdom,   </div><div class="fm-affl" lang="en" id="edit1">  University of Westminster, United Kingdom,   </div><div id="cor1">* E-mail: <a href="mailto:dev@null" data-email="ua.ude.satu@hctiveiprak.ayiluy" class="oemail">ua.ude.satu@hctiveiprak.ayiluy</a></div><div id="idm139681146254480"><strong>Competing Interests: </strong>One of the authors is employed by a commercial company GlaxoSmithKline R&amp;D. We note that this does not alter the authors’ adherence to PLOS ONE policies on sharing data and materials.</div><div><p id="__p2" class="p p-first-last">Conceived and designed the experiments: YVK SBN RW JES LME. Performed the experiments: SBN RW. Analyzed the data: YVK SBN LME. Contributed reagents/materials/analysis tools: YVK SBN RW JES LME. Contributed to the writing of the manuscript: YVK SBN RW JES LME.</p></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm139681141202096_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2014 Jul 4; Accepted 2014 Dec 3.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm139681141202096_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © 2014 Karpievitch et al</div><div class="license half_rhythm">This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are properly credited.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC4280143/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-length="222" class="box-data-avail whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-avl-stmnt" data-ga-action="click_feat_toggler" data-ga-label="Data Availability Statement" class="pmctoggle">Data Availability Statement</a></dt><dd id="data-avl-stmnt" style="display: none;"><p>The authors confirm that all data underlying the findings are fully available without restriction. The data necessary to reproduce our results is available on the EigenMS SourceForge page (<a href="http://eigenms.sourceforge.net/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">http://eigenms.sourceforge.net/</a>).</p></dd></dl></div><div id="idm139681178961488" lang="en" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139681178961488title">Abstract</h2><!--article-meta--><div><p id="__p3" class="p p-first-last">Liquid chromatography mass spectrometry has become one of the analytical platforms of choice for metabolomics studies. However, LC-MS metabolomics data can suffer from the effects of various systematic biases. These include batch effects, day-to-day variations in instrument performance, signal intensity loss due to time-dependent effects of the LC column performance, accumulation of contaminants in the MS ion source and MS sensitivity among others. In this study we aimed to test a singular value decomposition-based method, called EigenMS, for normalization of metabolomics data. We analyzed a clinical human dataset where LC-MS serum metabolomics data and physiological measurements were collected from thirty nine healthy subjects and forty with type 2 diabetes and applied EigenMS to detect and correct for any systematic bias. EigenMS works in several stages. First, EigenMS preserves the treatment group differences in the metabolomics data by estimating treatment effects with an ANOVA model (multiple fixed effects can be estimated). Singular value decomposition of the residuals matrix is then used to determine bias trends in the data. The number of bias trends is then estimated via a permutation test and the effects of the bias trends are eliminated. EigenMS removed bias of unknown complexity from the LC-MS metabolomics data, allowing for increased sensitivity in differential analysis. Moreover, normalized samples better correlated with both other normalized samples and corresponding physiological data, such as blood glucose level, glycated haemoglobin, exercise central augmentation pressure normalized to heart rate of 75, and total cholesterol. We were able to report 2578 discriminatory metabolite peaks in the normalized data (p&lt;0.05) as compared to only 1840 metabolite signals in the raw data. Our results support the use of singular value decomposition-based normalization for metabolomics data.</p></div></div><div id="s1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s1title">Introduction</h2><p id="__p5" class="p p-first-last">Along with nuclear magnetic resonance, liquid chromatography coupled to mass spectrometry (LC-MS) has become one of the most common analytical platforms for studying cell, tissue or body fluid metabolomes <a href="#pone.0116221-Zelena1" rid="pone.0116221-Zelena1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_689806568" role="button" aria-expanded="false" aria-haspopup="true">[1]</a>–<a href="#pone.0116221-Metz1" rid="pone.0116221-Metz1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_689806559" role="button" aria-expanded="false" aria-haspopup="true">[7]</a>. Advantages of the method include high sensitivity and the ability to discriminate thousands of features in a single experiment. Yet, as with any high-throughput technology, systematic biases are often observed in LC-MS metabolomics data <a href="#pone.0116221-Want1" rid="pone.0116221-Want1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_689806572" role="button" aria-expanded="false" aria-haspopup="true">[8]</a>. As the number of samples in the dataset increases so does the possibility of a time-dependent variation in the resulting metabolite data. Time-dependent trends in LC-MS metabolomics datasets typically result from analyte retention time drift due to changes in LC column performance or variations in signal intensity caused by fluctuations in MS sensitivity. While these issues can be addressed in part by careful experimental design and the use of quality control samples, there remains a need for robust post-acquisition data normalization. Normalization methods need to be flexible enough to capture biases of arbitrary complexity, while avoiding overfitting that would invalidate downstream statistical inference. Careful normalization of metabolite peak intensities enables greater accuracy in quantitative comparisons between disease groups as well as better correlation of metabolite signals to physiological or phenotypic data collected in tandem. We report here the application of a singular value decomposition-based method, called EigenMS, to remove systematic biases from metabolomics data in the presence of missing observations <a href="#pone.0116221-Karpievitch1" rid="pone.0116221-Karpievitch1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_689806566" role="button" aria-expanded="false" aria-haspopup="true">[9]</a>. This normalization method, previously shown to be effective in normalizing LC-MS proteomics data <a href="#pone.0116221-Karpievitch1" rid="pone.0116221-Karpievitch1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_689806558" role="button" aria-expanded="false" aria-haspopup="true">[9]</a>, improved downstream differential analysis and increased correlation of the metabolite peak intensities with corresponding physiological measurements of what we call clinical biochemistry.</p></div><div id="s2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s2title">Materials and Methods</h2><p id="__p6" class="p p-first">To demonstrate the utility of our approach, we used a recently-acquired metabolomic dataset examining the serum of subjects with type 2 diabetes (n = 40) and control subjects without diabetes (n = 39). The age range of the human subjects was 57±11 years with standard error of the mean of 1.2. Biological sample preparation and data acquisition followed the same protocol as reported in Nikolic et al. <a href="#pone.0116221-Nikolic1" rid="pone.0116221-Nikolic1" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">[10]</a>. LC-MS data were acquired in positive ion mode using an Orbitrap XL mass spectrometer (Thermo Scientific) controlled by XCalibur v2 software. Chromatographic separation was carried out using C18 reverse-phase HPLC as described in <a href="#pone.0116221-Want1" rid="pone.0116221-Want1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_689806571" role="button" aria-expanded="false" aria-haspopup="true">[8]</a>.</p><p id="__p7">The study was approved by the Human Research Ethics Committee of the University of Tasmania and all procedures conformed to the Declaration of Helsinki. Study participants gave written informed consent prior to participation.</p><p id="__p8">We generated a pooled quality control (QC) sample in order to monitor LC and MS performance across sample runs by combining small aliquots (10 uL) of every sample in the study, as recommended by Sangster et al. <a href="#pone.0116221-Sangster1" rid="pone.0116221-Sangster1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_689806567" role="button" aria-expanded="false" aria-haspopup="true">[11]</a>. This pooled QC sample was then used throughout the experiment as a process control. Because all QC injections originated from the same mixture and thus should be chemically identical, QC samples allow detection of variations in the observed intensities that may affect downstream statistical analyses.</p><p id="__p9">We monitored system performance using blocks of four experimental samples flanked by the QC samples between MS source and inlet cleaning. Thus our basic experimental running order included cleaning of the ion spray cone and exterior surface of the ion transfer capillary with 50∶50 methanol/water between each block. At the end of each day of operation we used 50∶50 methanol/water to flush the sample transfer tube and atmospheric pressure ionization probe, according to the guidelines for daily operation of LTQ XL specified by the manufacturer.</p><p id="__p10">Mass spectral peak deconvolution and retention time correction were carried out in R using XCMS. The parameters for deconvolution were: method = ‘centWave’, ppm = 3, peakwidth = 5–30, snthresh = 6, mzdiff = 0.01; and for RT correction: method = “obiwarp” and profStep = 0.1, yielding ∼7000 peaks.</p><p id="__p11" class="p p-last">EigenMS uses a combination of ANOVA and singular value decomposition to capture and remove biases from LC-MS metabolomic peak intensity measurements while preserving the variation of interest. ANOVA is used first to capture and preserve the variation attributable to the treatment effect(s) under study. EigenMS utilizes built-in R function lm() to estimate treatment effects and produce the matrix of residuals which accessed via the residuals() function. Singular value decomposition (SVD) is then applied to a matrix of residuals to find any systematic trends attributable to bias. The number of bias trends is determined by a permutation test and the effects of the bias trends are then removed from the data. We found that in metabolomic studies the number of bias trends to be eliminated should be set to about 20% of the number of samples. This is a heuristic approach that produces better normalization than the automatically determined number, which works well for proteomic studies. EigenMS is based on the surrogate variable analysis of Leek and Storey <a href="#pone.0116221-Leek1" rid="pone.0116221-Leek1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_689806565" role="button" aria-expanded="false" aria-haspopup="true">[12]</a>, with modifications including analysis of data with missing values that are typical in LC-MS experiments <a href="#pone.0116221-Karpievitch1" rid="pone.0116221-Karpievitch1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_689806562" role="button" aria-expanded="false" aria-haspopup="true">[9]</a>. EigenMS is available as a stand-alone set of two functions implemented in R from SourceForge: <a href="http://sourceforge.net/projects/eigenms/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://sourceforge.net/projects/eigenms/</a>. Current version of software was implemented in RStudio version 0.98.953 and R version 3.0.1.</p></div><div id="s3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s3title">Results and Discussion</h2><p id="__p12" class="p p-first">A characteristic of biofluid metabolite analysis via LC-MS is progressive signal intensity loss due to the accumulation of contaminants within the MS ion source, sample transfer lines and the heated ion transfer capillary <a href="#pone.0116221-Lai1" rid="pone.0116221-Lai1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_689806563" role="button" aria-expanded="false" aria-haspopup="true">[13]</a>. We designed our experiment based on the guidelines outlined in Lai et al. 2009 <a href="#pone.0116221-Lai1" rid="pone.0116221-Lai1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_689806564" role="button" aria-expanded="false" aria-haspopup="true">[13]</a> as well as based on in-house experimentation to establish the number of samples that could be run without large signal intensity loss on our instrumentation. We conditioned the LC column using QC samples as suggested by Want <em>et al.</em> and others prior to running any experimental samples to avoid high signal variation during the first few runs <a href="#pone.0116221-Want1" rid="pone.0116221-Want1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_689806570" role="button" aria-expanded="false" aria-haspopup="true">[8]</a>, <a href="#pone.0116221-Sangster1" rid="pone.0116221-Sangster1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_689806560" role="button" aria-expanded="false" aria-haspopup="true">[11]</a>.</p><p id="__p13">Within each block, different treatment groups were matched and run order randomized. However, even using frequent cleaning, some signal loss was observed as is evident from the declining abundance profile within each day (<a href="/pmc/articles/PMC4280143/figure/pone-0116221-g001/" target="figure" class="fig-table-link figpopup" rid-figpopup="pone-0116221-g001" rid-ob="ob-pone-0116221-g001" co-legend-rid="lgnd_pone-0116221-g001"><span>Fig. 1</span></a>). These data were acquired on the instrument operating in positive ion mode. Intensity loss was not as obvious when the instrument was operating in negative ion mode, however, we still observed variation that could not be easily explained. Careful experimental design, such as within-block treatment group matching, alleviated the influence of some of the biases and signal intensity loss that we and others have observed <a href="#pone.0116221-Lai1" rid="pone.0116221-Lai1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_689806561" role="button" aria-expanded="false" aria-haspopup="true">[13]</a>. We still suggest that normalization be used to correct for any remaining intensity loss as well as any other known and unknown systematic biases. Further, if one wishes to identify relationships between LC-MS metabolomics data and other variables, normalization becomes essential.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="pone-0116221-g001" co-legend-rid="lgnd_pone-0116221-g001"><a href="/pmc/articles/PMC4280143/figure/pone-0116221-g001/" target="figure" rid-figpopup="pone-0116221-g001" rid-ob="ob-pone-0116221-g001"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139681180414128" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is pone.0116221.g001.jpg" title="An external file that holds a picture, illustration, etc.
Object name is pone.0116221.g001.jpg" src="/pmc/articles/PMC4280143/bin/pone.0116221.g001.jpg"></div></a><div id="largeobj_idm139681180414128" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC4280143/figure/pone-0116221-g001/" target="figure" rid-figpopup="pone-0116221-g001" rid-ob="ob-pone-0116221-g001"></a><a target="object" href="/pmc/articles/PMC4280143/figure/pone-0116221-g001/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_pone-0116221-g001"><div><a class="figpopup" href="/pmc/articles/PMC4280143/figure/pone-0116221-g001/" target="figure" rid-figpopup="pone-0116221-g001" rid-ob="ob-pone-0116221-g001">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Boxplots of sample intensities.</strong><p id="__p14">Boxplots of sample intensities in run order on the instrument for positive ion mode. Each box represents a sample, each of five days is presented in different color. Decline in intensity indicates signal intensity loss within each day.</p></div></div></div><p id="__p15">Sample intensity loss makes comparisons between experimental groups more difficult, it also leads to an increased number of missing values in subsequent samples. Thus careful experimental design and sample run order randomization are required to minimize the introduction of systematic biases and any possible confounding of the results. Our experience leads us to recommend the use of QC samples to monitor performance of the LC-MS instrument. For example, we embedded QC samples throughout the experimental runs to help us monitor instrument performance. We utilized an incomplete block design where each block consisted of four samples, two diabetes and two control samples in randomized order. Each block was bracketed by QC samples to allow the operator to monitor the experiment and perform system diagnostics if variation in the signal intensity of the QC runs was observed. In this study QC samples were not used for data processing or normalization.</p><p id="__p16">We normalized the data with EigenMS. EigenMS identified 12 systematic bias trends and eliminated their effects from the data. <a href="/pmc/articles/PMC4280143/figure/pone-0116221-g002/" target="figure" class="fig-table-link figpopup" rid-figpopup="pone-0116221-g002" rid-ob="ob-pone-0116221-g002" co-legend-rid="lgnd_pone-0116221-g002"><span>Fig. 2</span></a> shows boxplots of the intensities for the disease and control groups before (top panel) and after (bottom panel) normalization for the same data as in <a href="/pmc/articles/PMC4280143/figure/pone-0116221-g001/" target="figure" class="fig-table-link figpopup" rid-figpopup="pone-0116221-g001" rid-ob="ob-pone-0116221-g001" co-legend-rid="lgnd_pone-0116221-g001"><span>Fig. 1</span></a>. Samples in <a href="/pmc/articles/PMC4280143/figure/pone-0116221-g002/" target="figure" class="fig-table-link figpopup" rid-figpopup="pone-0116221-g002" rid-ob="ob-pone-0116221-g002" co-legend-rid="lgnd_pone-0116221-g002"><span>Fig. 2</span></a> are grouped by disease group (red vs. green with QC samples not shown) and within each group they appear in the run order on the instrument, such that the first sample in red was run right next to the first sample in green and so on. Even with regular cleaning of the inlet we encountered some signal loss as is evident from the downward trend in the means (middle bars) of the boxplots. <a href="/pmc/articles/PMC4280143/figure/pone-0116221-g002/" target="figure" class="fig-table-link figpopup" rid-figpopup="pone-0116221-g002" rid-ob="ob-pone-0116221-g002" co-legend-rid="lgnd_pone-0116221-g002"><span>Fig. 2</span></a> bottom panel shows that normalization successfully adjusted for the signal intensity loss and any other systematic biases and placed the means of each sample almost on a straight line.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="pone-0116221-g002" co-legend-rid="lgnd_pone-0116221-g002"><a href="/pmc/articles/PMC4280143/figure/pone-0116221-g002/" target="figure" rid-figpopup="pone-0116221-g002" rid-ob="ob-pone-0116221-g002"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139681183754608" class="figure"><a href="/pmc/articles/PMC4280143/figure/pone-0116221-g002/" target="figure" rid-figpopup="pone-0116221-g002" rid-ob="ob-pone-0116221-g002"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4280143_pone.0116221.g002.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is pone.0116221.g002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4280143/bin/pone.0116221.g002.jpg"></a></div><div id="largeobj_idm139681183754608" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4280143/figure/pone-0116221-g002/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_pone-0116221-g002"><div><a class="figpopup" href="/pmc/articles/PMC4280143/figure/pone-0116221-g002/" target="figure" rid-figpopup="pone-0116221-g002" rid-ob="ob-pone-0116221-g002">Figure 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Boxplots of the raw and normalized intensities.</strong><p id="__p17">Boxplots of the intensities before (top panel) and after (bottom panel) normalization. Each box represents a sample. Samples are grouped by disease group (red vs. green, QC samples are omitted) and are in chronological run order on the instrument within each group.</p></div></div></div><p id="__p18">
<a href="/pmc/articles/PMC4280143/figure/pone-0116221-g003/" target="figure" class="fig-table-link figpopup" rid-figpopup="pone-0116221-g003" rid-ob="ob-pone-0116221-g003" co-legend-rid="lgnd_pone-0116221-g003"><span>Fig. 3</span></a> shows SVD trends in raw (left 3 panels) and normalized (right 3 panels) diabetes data. Samples are ordered by group and appear in chronological run order within each group in the same way as <a href="/pmc/articles/PMC4280143/figure/pone-0116221-g002/" target="figure" class="fig-table-link figpopup" rid-figpopup="pone-0116221-g002" rid-ob="ob-pone-0116221-g002" co-legend-rid="lgnd_pone-0116221-g002"><span>Fig. 2</span></a>. Trends are ordered from top to bottom by the decreasing amount of variation explained by each trend. Due to the nature of SVD, each trend is orthogonal to every other. Notably, 20 percent of the variation in the raw data is attributable to the signal loss that appears as the top trend. Note that SVD trends can be rotated around the x-axis, thus the top trend in the raw data represents signal intensity loss. The top trend also shows a jump in each group which occurred between days 3 and 4 of the experiment. All the processing was done following the same protocol, but we still observed a variation due to the day effect where samples run on days 4 and 5 were affected differently than samples run on days 1–3. The rest of the trends in the raw data are not easily interpretable and are attributed to unknown systematic biases. <a href="/pmc/articles/PMC4280143/figure/pone-0116221-g003/" target="figure" class="fig-table-link figpopup" rid-figpopup="pone-0116221-g003" rid-ob="ob-pone-0116221-g003" co-legend-rid="lgnd_pone-0116221-g003"><span>Fig. 3</span></a> (right panel) shows the normalized data. The top trend is representative of the differences between the disease and control groups. Only 3.7% of variation is attributable to the differences between the disease groups, but nonetheless this is a major trend in the normalized data as compared to the signal intensity loss trend in the raw data. Most importantly we were able to report 2578 discriminatory metabolite peaks in the normalized data as compared to only 1840 metabolite signals in the raw data (compared using an unpaired t-test; with a significance cutoff α = 0.05 after Benjamini-Hochberg multiple testing adjustment) <a href="#pone.0116221-Benjamini1" rid="pone.0116221-Benjamini1" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">[14]</a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="pone-0116221-g003" co-legend-rid="lgnd_pone-0116221-g003"><a href="/pmc/articles/PMC4280143/figure/pone-0116221-g003/" target="figure" rid-figpopup="pone-0116221-g003" rid-ob="ob-pone-0116221-g003"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139681141114544" class="figure"><a href="/pmc/articles/PMC4280143/figure/pone-0116221-g003/" target="figure" rid-figpopup="pone-0116221-g003" rid-ob="ob-pone-0116221-g003"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4280143_pone.0116221.g003.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is pone.0116221.g003.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4280143/bin/pone.0116221.g003.jpg"></a></div><div id="largeobj_idm139681141114544" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4280143/figure/pone-0116221-g003/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_pone-0116221-g003"><div><a class="figpopup" href="/pmc/articles/PMC4280143/figure/pone-0116221-g003/" target="figure" rid-figpopup="pone-0116221-g003" rid-ob="ob-pone-0116221-g003">Figure 3</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>SVD trends in raw and normalized data.</strong><p id="__p19">SVD trends in raw (left panel) and normalized (right panel) clinical study data. Percentage at the top of each subplot shows the percent of the variation in the data explained by each trend. On the <em>x</em>-axis is sample index from 1 to 79, where each circle represents a sample. Samples are grouped by disease group with 39 control samples numbered 1<strong>–</strong>39 followed by 40 diabetes samples numbered 40<strong>–</strong>79. Samples are in chronological run order on the instrument within each group. Values on the y-axis satisfy equation: <em>R</em> = <em>UDV</em>’ where <em>R</em> is a matrix of residuals (left panel) and matrix of normalized intensities (right panel); columns of <em>V</em> represent trends observed in the data. Three trends that explain the most amount of variation are plotted for raw and normalized data.</p></div></div></div><p id="__p20">We saw an improvement in correlations of normalized metabolite intensities to the physiology variables we measured for each subject. We selected 1100 peaks that were found to be significantly different between two groups with Benjamini-Hochberg adjusted p-value&lt;0.001 in the normalized data. We correlated these 1100 MS peaks using Spearman correlation in ‘Hmisc’ package to the following clinical measurements we obtained for each subject: blood glucose level, glycated haemoglobin, exercise central augmentation pressure normalized to heart rate of 75, and total cholesterol <a href="#pone.0116221-Climie1" rid="pone.0116221-Climie1" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">[15]</a>. The first three of these are expected to be elevated in diabetics while cholesterol is usually lower than controls due to statin therapy. If EigenMS works well we expect to see increased correlation between the changes observed in normalized metabolites and physiological variables.</p><p id="__p21" class="p">
<a href="/pmc/articles/PMC4280143/figure/pone-0116221-g004/" target="figure" class="fig-table-link figpopup" rid-figpopup="pone-0116221-g004" rid-ob="ob-pone-0116221-g004" co-legend-rid="lgnd_pone-0116221-g004"><span>Fig. 4</span></a> shows correlations of raw (x-axis) and normalized (y-axis) metabolite intensities to glucose (top left), glycated haemoglobin (top right), exercise central augmentation pressure (bottom left) and cholesterol (bottom right). The blue lines show correlations of zero. The red lines indicate no difference between correlations obtained from normalized and raw data. The scatter plots show that we obtained higher correlations of normalized metabolite peak intensities to the variables measured in the laboratory. For glucose, for example, correlations for most of the positively correlated peaks increased as is evident from the dots falling above the diagonal line. For negatively correlated metabolites most of the correlations fall near the diagonal line still producing more scatter below the diagonal line indicating increased correlations. Similar patterns are observed for the rest of the physiological variables.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="pone-0116221-g004" co-legend-rid="lgnd_pone-0116221-g004"><a href="/pmc/articles/PMC4280143/figure/pone-0116221-g004/" target="figure" rid-figpopup="pone-0116221-g004" rid-ob="ob-pone-0116221-g004"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139681146118480" class="figure"><a href="/pmc/articles/PMC4280143/figure/pone-0116221-g004/" target="figure" rid-figpopup="pone-0116221-g004" rid-ob="ob-pone-0116221-g004"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4280143_pone.0116221.g004.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is pone.0116221.g004.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4280143/bin/pone.0116221.g004.jpg"></a></div><div id="largeobj_idm139681146118480" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4280143/figure/pone-0116221-g004/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_pone-0116221-g004"><div><a class="figpopup" href="/pmc/articles/PMC4280143/figure/pone-0116221-g004/" target="figure" rid-figpopup="pone-0116221-g004" rid-ob="ob-pone-0116221-g004">Figure 4</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Correlations to physiology data.</strong><p id="__p22">Correlations of raw (x-axis) and normalized (y-axis) metabolite intensities to physiology data. Metabolites with p-value&lt;0.001 in the normalized data (1100 metabolites) were selected to be plotted. The same metabolites are plotted for raw data. Blue lines represent correlation of zero. Red diagonal line shows the line on which all dots would fall if there was no difference between correlations of raw and normalized data. Observed counter-clockwise shift of points indicates better correlation with physiological data.</p></div></div></div></div><div id="s4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s4title">Conclusions</h2><p id="__p23" class="p p-first-last">Normalization is an important step in MS data analysis, but it is complicated by the high complexity of biases. EigenMS has been shown to remove biases of arbitrary complexity from proteomics data <a href="#pone.0116221-Karpievitch1" rid="pone.0116221-Karpievitch1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_689806569" role="button" aria-expanded="false" aria-haspopup="true">[9]</a>. Here we show that it works equally well for metabolomics data. By extension, the method should reasonably be expected to work equally well for any omics data where the variation of interest can be preserved via the fixed effects in an ANOVA model and the matrix of residuals analyzed for the presence of bias trends. The ability of EigenMS to capture complex biases and eliminate them preserves the validity of any downstream statistical analysis. The software is implemented in R and is freely available from SourceForge: <a href="http://sourceforge.net/projects/eigenms/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://sourceforge.net/projects/eigenms/</a>.</p></div><div id="idm139681131514064" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139681131514064title">Funding Statement</h2><p>GlaxoSmithKline R&amp; D provided support in the form of salaries for authors [LME], but did not have any additional role in the study design, data collection and analysis, decision to publish, or preparation of the manuscript. The specific roles of these authors are articulated in the ‘author contributions’ section.</p></div><div id="idm139681129431088" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139681129431088title">Data Availability</h2><p id="__p4">The authors confirm that all data underlying the findings are fully available without restriction. The data necessary to reproduce our results is available on the EigenMS SourceForge page (<a href="http://eigenms.sourceforge.net/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">http://eigenms.sourceforge.net/</a>).</p></div><div id="idm139681145547168" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139681145547168title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="pone.0116221-Zelena1">1. <span class="mixed-citation">
Zelena E, Dunn WB, Broadhurst D, Francis-McIntyre S, Carroll KM, et al. (2009) <span class="ref-title">Development of a robust and repeatable UPLC-MS method for the long-term metabolomic study of human serum</span>. <span class="ref-journal">Anal Chem</span>
<span class="ref-vol">81</span>:1357–1364. [<a href="/pubmed/19170513" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Anal+Chem&amp;title=Development+of+a+robust+and+repeatable+UPLC-MS+method+for+the+long-term+metabolomic+study+of+human+serum&amp;author=E+Zelena&amp;author=WB+Dunn&amp;author=D+Broadhurst&amp;author=S+Francis-McIntyre&amp;author=KM+Carroll&amp;volume=81&amp;publication_year=2009&amp;pages=1357-1364&amp;pmid=19170513&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0116221-Gika1">2. <span class="mixed-citation">
Gika HG, Theodoridis GA, Wilson ID (2008) <span class="ref-title">Liquid chromatography and ultra-performance liquid chromatography-mass spectrometry fingerprinting of human urine: sample stability under different handling and storage conditions for metabonomics studies</span>. <span class="ref-journal">J Chromatogr A</span>
<span class="ref-vol">1189</span>:314–322. [<a href="/pubmed/18096175" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Chromatogr+A&amp;title=Liquid+chromatography+and+ultra-performance+liquid+chromatography-mass+spectrometry+fingerprinting+of+human+urine:+sample+stability+under+different+handling+and+storage+conditions+for+metabonomics+studies&amp;author=HG+Gika&amp;author=GA+Theodoridis&amp;author=ID+Wilson&amp;volume=1189&amp;publication_year=2008&amp;pages=314-322&amp;pmid=18096175&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0116221-Guy1">3. <span class="mixed-citation">
Guy PA, Tavazzi I, Bruce SJ, Ramadan Z, Kochhar S (2008) <span class="ref-title">Global metabolic profiling analysis on human urine by UPLC-TOFMS: issues and method validation in nutritional metabolomics</span>. <span class="ref-journal">J Chromatogr B Analyt Technol Biomed Life Sci</span>
<span class="ref-vol">871</span>:253–260. [<a href="/pubmed/18490203" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Chromatogr+B+Analyt+Technol+Biomed+Life+Sci&amp;title=Global+metabolic+profiling+analysis+on+human+urine+by+UPLC-TOFMS:+issues+and+method+validation+in+nutritional+metabolomics&amp;author=PA+Guy&amp;author=I+Tavazzi&amp;author=SJ+Bruce&amp;author=Z+Ramadan&amp;author=S+Kochhar&amp;volume=871&amp;publication_year=2008&amp;pages=253-260&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0116221-Bordbar1">4. <span class="mixed-citation">
Bordbar A, Mo ML, Nakayasu ES, Schrimpe-Rutledge AC, Kim YM, et al. (2012) <span class="ref-title">Model-driven multi-omic data analysis elucidates metabolic immunomodulators of macrophage activation</span>. <span class="ref-journal">Mol Syst Biol</span>
<span class="ref-vol">8</span>:558. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3397418/">PMC free article</a>]</span> [<a href="/pubmed/22735334" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Syst+Biol&amp;title=Model-driven+multi-omic+data+analysis+elucidates+metabolic+immunomodulators+of+macrophage+activation&amp;author=A+Bordbar&amp;author=ML+Mo&amp;author=ES+Nakayasu&amp;author=AC+Schrimpe-Rutledge&amp;author=YM+Kim&amp;volume=8&amp;publication_year=2012&amp;pages=558&amp;pmid=22735334&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0116221-DeatherageKaiser1">5. <span class="mixed-citation">
Deatherage Kaiser BL, Li J, Sanford JA, Kim YM, Kronewitter SR, et al. (2013) <span class="ref-title">A Multi-Omic View of Host-Pathogen-Commensal Interplay in -Mediated Intestinal Infection</span>. <span class="ref-journal">PLoS One</span>
<span class="ref-vol">8</span>:e67155. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3694140/">PMC free article</a>]</span> [<a href="/pubmed/23840608" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=A+Multi-Omic+View+of+Host-Pathogen-Commensal+Interplay+in+-Mediated+Intestinal+Infection&amp;author=BL+Deatherage+Kaiser&amp;author=J+Li&amp;author=JA+Sanford&amp;author=YM+Kim&amp;author=SR+Kronewitter&amp;volume=8&amp;publication_year=2013&amp;pages=e67155&amp;pmid=23840608&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0116221-Kim1">6. <span class="mixed-citation">
Kim YM, Schmidt BJ, Kidwai AS, Jones MB, Deatherage Kaiser BL, et al. (2013) <span class="ref-title">Salmonella modulates metabolism during growth under conditions that induce expression of virulence genes</span>. <span class="ref-journal">Mol Biosyst</span>
<span class="ref-vol">9</span>:1522–1534. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3665296/">PMC free article</a>]</span> [<a href="/pubmed/23559334" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Biosyst&amp;title=Salmonella+modulates+metabolism+during+growth+under+conditions+that+induce+expression+of+virulence+genes&amp;author=YM+Kim&amp;author=BJ+Schmidt&amp;author=AS+Kidwai&amp;author=MB+Jones&amp;author=BL+Deatherage+Kaiser&amp;volume=9&amp;publication_year=2013&amp;pages=1522-1534&amp;pmid=23559334&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0116221-Metz1">7. <span class="mixed-citation">
Metz TO, Page JS, Baker ES, Tang K, Ding J, et al. (2008) <span class="ref-title">High Resolution Separations and Improved Ion Production and Transmission in Metabolomics</span>. <span class="ref-journal">Trends Analyt Chem</span>
<span class="ref-vol">27</span>:205–214. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2625305/">PMC free article</a>]</span> [<a href="/pubmed/19255623" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Analyt+Chem&amp;title=High+Resolution+Separations+and+Improved+Ion+Production+and+Transmission+in+Metabolomics&amp;author=TO+Metz&amp;author=JS+Page&amp;author=ES+Baker&amp;author=K+Tang&amp;author=J+Ding&amp;volume=27&amp;publication_year=2008&amp;pages=205-214&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0116221-Want1">8. <span class="mixed-citation">
Want EJ, Wilson ID, Gika H, Theodoridis G, Plumb RS, et al. (2010) <span class="ref-title">Global metabolic profiling procedures for urine using UPLC–MS</span>. <span class="ref-journal">Nature Protocols</span>
<span class="ref-vol">5</span>:1005–1018. [<a href="/pubmed/20448546" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Protocols&amp;title=Global+metabolic+profiling+procedures+for+urine+using+UPLC–MS&amp;author=EJ+Want&amp;author=ID+Wilson&amp;author=H+Gika&amp;author=G+Theodoridis&amp;author=RS+Plumb&amp;volume=5&amp;publication_year=2010&amp;pages=1005-1018&amp;pmid=20448546&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0116221-Karpievitch1">9. <span class="mixed-citation">
Karpievitch YV, Taverner T, Adkins JN, Callister SJ, Anderson GA, et al. (2009) <span class="ref-title">Normalization of Peak Intensities in Bottom-Up MS-Based Proteomics Using Singular Value Decomposition</span>. <span class="ref-journal">Bioinformatics</span>
<span class="ref-vol">25</span>:2573–2580. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2752608/">PMC free article</a>]</span> [<a href="/pubmed/19602524" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Normalization+of+Peak+Intensities+in+Bottom-Up+MS-Based+Proteomics+Using+Singular+Value+Decomposition&amp;author=YV+Karpievitch&amp;author=T+Taverner&amp;author=JN+Adkins&amp;author=SJ+Callister&amp;author=GA+Anderson&amp;volume=25&amp;publication_year=2009&amp;pages=2573-2580&amp;pmid=19602524&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0116221-Nikolic1">10. <span class="mixed-citation">
Nikolic S, Wilson R, Hare J, Adams M, Edwards L, et al. (2014) <span class="ref-title">Spironolactone reduces aortic stiffness via blood pressure-dependent effects of canrenoate</span>. <span class="ref-journal">Metabolomics</span>
<span class="ref-vol">10</span>:105–113. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Metabolomics&amp;title=Spironolactone+reduces+aortic+stiffness+via+blood+pressure-dependent+effects+of+canrenoate&amp;author=S+Nikolic&amp;author=R+Wilson&amp;author=J+Hare&amp;author=M+Adams&amp;author=L+Edwards&amp;volume=10&amp;publication_year=2014&amp;pages=105-113&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0116221-Sangster1">11. <span class="mixed-citation">
Sangster T, Major H, Plumb R, Wilson AJ, Wilson ID (2006) <span class="ref-title">A pragmatic and readily implemented quality control strategy for HPLC-MS and GC-MS-based metabolomic analysis</span>. <span class="ref-journal">The Analyst</span>
<span class="ref-vol">131</span>:1075–1078. [<a href="/pubmed/17003852" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=The+Analyst&amp;title=A+pragmatic+and+readily+implemented+quality+control+strategy+for+HPLC-MS+and+GC-MS-based+metabolomic+analysis&amp;author=T+Sangster&amp;author=H+Major&amp;author=R+Plumb&amp;author=AJ+Wilson&amp;author=ID+Wilson&amp;volume=131&amp;publication_year=2006&amp;pages=1075-1078&amp;pmid=17003852&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0116221-Leek1">12. <span class="mixed-citation">Leek JT, Storey JD (2007) Capturing heterogeneity in gene expression studies by surrogate variable analysis. PLoS Genetics 3. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1994707/">PMC free article</a>]</span> [<a href="/pubmed/17907809" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="pone.0116221-Lai1">13. <span class="mixed-citation">
Lai L, Michopoulos F, Gika H, Theodoridis G, Wilkinson RW, et al. (2010) <span class="ref-title">Methodological considerations in the development of HPLC-MS methods for the analysis of rodent plasma for metabonomic studies</span>. <span class="ref-journal">Mol Biosyst</span>
<span class="ref-vol">6</span>:108–120. [<a href="/pubmed/20024072" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Biosyst&amp;title=Methodological+considerations+in+the+development+of+HPLC-MS+methods+for+the+analysis+of+rodent+plasma+for+metabonomic+studies&amp;author=L+Lai&amp;author=F+Michopoulos&amp;author=H+Gika&amp;author=G+Theodoridis&amp;author=RW+Wilkinson&amp;volume=6&amp;publication_year=2010&amp;pages=108-120&amp;pmid=20024072&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0116221-Benjamini1">14. <span class="mixed-citation">
Benjamini Y, Hochberg Y (1995) <span class="ref-title">Controlling the false discovery rate: a practical and powerful approach to multiple testing</span>. <span class="ref-journal">Journal of the Royal Statistical Society</span>
<span class="ref-vol">57</span>:289–300. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Journal+of+the+Royal+Statistical+Society&amp;title=Controlling+the+false+discovery+rate:+a+practical+and+powerful+approach+to+multiple+testing&amp;author=Y+Benjamini&amp;author=Y+Hochberg&amp;volume=57&amp;publication_year=1995&amp;pages=289-300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="pone.0116221-Climie1">15. <span class="mixed-citation">
Climie RED, Nikolic SB, Otahal P, Keith LJ, Sharman JE (2013) <span class="ref-title">Augmentation index and arterial stiffness in patients with type 2 diabetes mellitus</span>. <span class="ref-journal">Artery Research</span>
<span class="ref-vol">7</span>:194–200. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Artery+Research&amp;title=Augmentation+index+and+arterial+stiffness+in+patients+with+type+2+diabetes+mellitus&amp;author=RED+Climie&amp;author=SB+Nikolic&amp;author=P+Otahal&amp;author=LJ+Keith&amp;author=JE+Sharman&amp;volume=7&amp;publication_year=2013&amp;pages=194-200&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2" style=""><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139681178961488title">Abstract</a></li><li><a href="#s1title">Introduction</a></li><li><a href="#s2title">Materials and Methods</a></li><li><a href="#s3title">Results and Discussion</a></li><li><a href="#s4title">Conclusions</a></li><li><a href="#idm139681131514064title">Funding Statement</a></li><li><a href="#idm139681129431088title">Data Availability</a></li><li><a href="#idm139681145547168title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">PLoS ONE</span> are provided here courtesy of <strong>Public Library of Science</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4280143/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4280143/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4280143/pdf/pone.0116221.pdf">PDF (1.7M)</a></li> | <li><a href="#" data-citationid="PMC4280143" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4280143%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4280143%2F&amp;text=Metabolomics%20Data%20Normalization%20with%20EigenMS" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4280143%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="25549083" data-aiid="4280143" data-aid="4280143" data-iid="245922" data-domainid="440" data-domain="plosone" data-accid="PMC4280143" data-md5="a59a257ed6f6c746499f1de0ec3f8b73">

<div>







</div>   
    </div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"><div id="body-link-popper-pone.0116221-Zelena1" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Development of a robust and repeatable UPLC-MS method for the long-term metabolomic study of human serum.<div class="authors">Zelena E, Dunn WB, Broadhurst D, Francis-McIntyre S, Carroll KM, Begley P, O'Hagan S, Knowles JD, Halsall A, HUSERMET Consortium., Wilson ID, Kell DB</div><div class="citation">Anal Chem. 2009 Feb 15; 81(4):1357-64.</div><p>[<a href="/pubmed/19170513/" ref="reftype=pubmed&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#pone.0116221-Zelena1">Ref list</a>]</p></div><div id="body-link-popper-pone.0116221-Metz1" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">High Resolution Separations and Improved Ion Production and Transmission in Metabolomics.<div class="authors">Metz TO, Page JS, Baker ES, Tang K, Ding J, Shen Y, Smith RD</div><div class="citation">Trends Analyt Chem. 2008 Mar; 27(3):205-214.</div><p>[<a href="/pubmed/19255623/" ref="reftype=pubmed&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#pone.0116221-Metz1">Ref list</a>]</p></div><div id="body-link-popper-pone.0116221-Want1" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Global metabolic profiling procedures for urine using UPLC-MS.<div class="authors">Want EJ, Wilson ID, Gika H, Theodoridis G, Plumb RS, Shockcor J, Holmes E, Nicholson JK</div><div class="citation">Nat Protoc. 2010 Jun; 5(6):1005-18.</div><p>[<a href="/pubmed/20448546/" ref="reftype=pubmed&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#pone.0116221-Want1">Ref list</a>]</p></div><div id="body-link-popper-pone.0116221-Karpievitch1" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Normalization of peak intensities in bottom-up MS-based proteomics using singular value decomposition.<div class="authors">Karpievitch YV, Taverner T, Adkins JN, Callister SJ, Anderson GA, Smith RD, Dabney AR</div><div class="citation">Bioinformatics. 2009 Oct 1; 25(19):2573-80.</div><p>[<a href="/pubmed/19602524/" ref="reftype=pubmed&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#pone.0116221-Karpievitch1">Ref list</a>]</p></div><div id="body-link-popper-pone.0116221-Nikolic1" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">10. <span class="mixed-citation">
Nikolic S, Wilson R, Hare J, Adams M, Edwards L, et al. (2014) <span class="ref-title">Spironolactone reduces aortic stiffness via blood pressure-dependent effects of canrenoate</span>. <span class="ref-journal">Metabolomics</span>
<span class="ref-vol">10</span>:105–113. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Metabolomics&amp;title=Spironolactone+reduces+aortic+stiffness+via+blood+pressure-dependent+effects+of+canrenoate&amp;author=S+Nikolic&amp;author=R+Wilson&amp;author=J+Hare&amp;author=M+Adams&amp;author=L+Edwards&amp;volume=10&amp;publication_year=2014&amp;pages=105-113&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#pone.0116221-Nikolic1">Ref list</a>]</div><div id="body-link-popper-pone.0116221-Sangster1" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">A pragmatic and readily implemented quality control strategy for HPLC-MS and GC-MS-based metabonomic analysis.<div class="authors">Sangster T, Major H, Plumb R, Wilson AJ, Wilson ID</div><div class="citation">Analyst. 2006 Oct; 131(10):1075-8.</div><p>[<a href="/pubmed/17003852/" ref="reftype=pubmed&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#pone.0116221-Sangster1">Ref list</a>]</p></div><div id="body-link-popper-pone.0116221-Leek1" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Capturing heterogeneity in gene expression studies by surrogate variable analysis.<div class="authors">Leek JT, Storey JD</div><div class="citation">PLoS Genet. 2007 Sep; 3(9):1724-35.</div><p>[<a href="/pubmed/17907809/" ref="reftype=pubmed&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#pone.0116221-Leek1">Ref list</a>]</p></div><div id="body-link-popper-pone.0116221-Lai1" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Methodological considerations in the development of HPLC-MS methods for the analysis of rodent plasma for metabonomic studies.<div class="authors">Lai L, Michopoulos F, Gika H, Theodoridis G, Wilkinson RW, Odedra R, Wingate J, Bonner R, Tate S, Wilson ID</div><div class="citation">Mol Biosyst. 2010 Jan; 6(1):108-20.</div><p>[<a href="/pubmed/20024072/" ref="reftype=pubmed&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#pone.0116221-Lai1">Ref list</a>]</p></div><div id="body-link-popper-pone.0116221-Benjamini1" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">14. <span class="mixed-citation">
Benjamini Y, Hochberg Y (1995) <span class="ref-title">Controlling the false discovery rate: a practical and powerful approach to multiple testing</span>. <span class="ref-journal">Journal of the Royal Statistical Society</span>
<span class="ref-vol">57</span>:289–300. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Journal+of+the+Royal+Statistical+Society&amp;title=Controlling+the+false+discovery+rate:+a+practical+and+powerful+approach+to+multiple+testing&amp;author=Y+Benjamini&amp;author=Y+Hochberg&amp;volume=57&amp;publication_year=1995&amp;pages=289-300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#pone.0116221-Benjamini1">Ref list</a>]</div><div id="body-link-popper-pone.0116221-Climie1" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">15. <span class="mixed-citation">
Climie RED, Nikolic SB, Otahal P, Keith LJ, Sharman JE (2013) <span class="ref-title">Augmentation index and arterial stiffness in patients with type 2 diabetes mellitus</span>. <span class="ref-journal">Artery Research</span>
<span class="ref-vol">7</span>:194–200. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Artery+Research&amp;title=Augmentation+index+and+arterial+stiffness+in+patients+with+type+2+diabetes+mellitus&amp;author=RED+Climie&amp;author=SB+Nikolic&amp;author=P+Otahal&amp;author=LJ+Keith&amp;author=JE+Sharman&amp;volume=7&amp;publication_year=2013&amp;pages=194-200&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280143&amp;issue-id=245922&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#pone.0116221-Climie1">Ref list</a>]</div></div>
                        </div>
                        <div class="bottom">
                            
                            

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 64px; left: 459px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript>&lt;img alt="statistics" src="/stat?jsdisabled=true&amp;amp;ncbi_db=pmc&amp;amp;ncbi_pdid=article&amp;amp;ncbi_acc=&amp;amp;ncbi_domain=plosone&amp;amp;ncbi_report=record&amp;amp;ncbi_type=fulltext&amp;amp;ncbi_objectid=&amp;amp;ncbi_pcid=/articles/PMC4280143/&amp;amp;ncbi_app=pmc" /&gt;</noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- 95B63906E70AD421_0000SID /projects/PMC/PMCViewer@4.46 ptpmc102 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></htm
