{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "name": "Custom NER.ipynb",
      "provenance": [],
      "collapsed_sections": []
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    }
  },
  "cells": [
    {
      "cell_type": "code",
      "metadata": {
        "id": "suVad9QdIuNh"
      },
      "source": [
        "# CONFIG File \r\n",
        "# Data annotations and data set configurations\r\n",
        "assay_list_location = \"/content/drive/My Drive/Imperial college London/alla.csv\"\r\n",
        "dataset_path_html = r\"/content/drive/My Drive/Imperial college London/data\" # Processed raw html with  beautifulsoup \r\n",
        "dataset_path_json = \"\" # Processed with autoscoopyst\r\n",
        "# PROCCESED data\r\n",
        "train_data_path=\"\" # Where to save the JSON file with all the annontated data\r\n",
        "\r\n",
        "\r\n",
        "# Spacy model \r\n",
        "TRAIN_DATA =\"\" # JSON file with final output \r\n",
        "test_data=\"\"\r\n"
      ],
      "execution_count": 14,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "K7RnUN8FIm1Z",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 537
        },
        "outputId": "59a6fbc0-4f76-43f4-ae15-e85d56ca3935"
      },
      "source": [
        "#1\n",
        "#Importing drive contents \n",
        "from google.colab import drive\n",
        "drive.mount('/content/drive')\n",
        "import pandas as pd\n",
        "#Read assay list\n",
        "assay=[]\n",
        "df = pd.read_csv(\"/content/drive/My Drive/Imperial college London/alla.csv\",dtype= str) # READ CSV AS STRING !!!\n",
        "assay = df.values.tolist()\n",
        "assay = list(map(''.join, assay)) # convert list of lists to list of strings "
      ],
      "execution_count": 15,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Drive already mounted at /content/drive; to attempt to forcibly remount, call drive.mount(\"/content/drive\", force_remount=True).\n"
          ],
          "name": "stdout"
        },
        {
          "output_type": "error",
          "ename": "ParserError",
          "evalue": "ignored",
          "traceback": [
            "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
            "\u001b[0;31mParserError\u001b[0m                               Traceback (most recent call last)",
            "\u001b[0;32m<ipython-input-15-7edcd87abfb0>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      6\u001b[0m \u001b[0;31m#Read assay list\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      7\u001b[0m \u001b[0massay\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 8\u001b[0;31m \u001b[0mdf\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mpd\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mread_csv\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"/content/drive/My Drive/Imperial college London/alla.csv\"\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0mdtype\u001b[0m\u001b[0;34m=\u001b[0m \u001b[0mstr\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;31m# READ CSV AS STRING !!!\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      9\u001b[0m \u001b[0massay\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mdf\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mvalues\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mtolist\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     10\u001b[0m \u001b[0massay\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mlist\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mmap\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m''\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mjoin\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0massay\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;31m# convert list of lists to list of strings\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;32m/usr/local/lib/python3.6/dist-packages/pandas/io/parsers.py\u001b[0m in \u001b[0;36mread_csv\u001b[0;34m(filepath_or_buffer, sep, delimiter, header, names, index_col, usecols, squeeze, prefix, mangle_dupe_cols, dtype, engine, converters, true_values, false_values, skipinitialspace, skiprows, skipfooter, nrows, na_values, keep_default_na, na_filter, verbose, skip_blank_lines, parse_dates, infer_datetime_format, keep_date_col, date_parser, dayfirst, cache_dates, iterator, chunksize, compression, thousands, decimal, lineterminator, quotechar, quoting, doublequote, escapechar, comment, encoding, dialect, error_bad_lines, warn_bad_lines, delim_whitespace, low_memory, memory_map, float_precision)\u001b[0m\n\u001b[1;32m    686\u001b[0m     )\n\u001b[1;32m    687\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 688\u001b[0;31m     \u001b[0;32mreturn\u001b[0m \u001b[0m_read\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mfilepath_or_buffer\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mkwds\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    689\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    690\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;32m/usr/local/lib/python3.6/dist-packages/pandas/io/parsers.py\u001b[0m in \u001b[0;36m_read\u001b[0;34m(filepath_or_buffer, kwds)\u001b[0m\n\u001b[1;32m    452\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    453\u001b[0m     \u001b[0;31m# Create the parser.\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 454\u001b[0;31m     \u001b[0mparser\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mTextFileReader\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mfp_or_buf\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkwds\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    455\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    456\u001b[0m     \u001b[0;32mif\u001b[0m \u001b[0mchunksize\u001b[0m \u001b[0;32mor\u001b[0m \u001b[0miterator\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;32m/usr/local/lib/python3.6/dist-packages/pandas/io/parsers.py\u001b[0m in \u001b[0;36m__init__\u001b[0;34m(self, f, engine, **kwds)\u001b[0m\n\u001b[1;32m    946\u001b[0m             \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0moptions\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m\"has_index_names\"\u001b[0m\u001b[0;34m]\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mkwds\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m\"has_index_names\"\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    947\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 948\u001b[0;31m         \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_make_engine\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mengine\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    949\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    950\u001b[0m     \u001b[0;32mdef\u001b[0m \u001b[0mclose\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;32m/usr/local/lib/python3.6/dist-packages/pandas/io/parsers.py\u001b[0m in \u001b[0;36m_make_engine\u001b[0;34m(self, engine)\u001b[0m\n\u001b[1;32m   1178\u001b[0m     \u001b[0;32mdef\u001b[0m \u001b[0m_make_engine\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mengine\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;34m\"c\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1179\u001b[0m         \u001b[0;32mif\u001b[0m \u001b[0mengine\u001b[0m \u001b[0;34m==\u001b[0m \u001b[0;34m\"c\"\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 1180\u001b[0;31m             \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_engine\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mCParserWrapper\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mf\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0moptions\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1181\u001b[0m         \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1182\u001b[0m             \u001b[0;32mif\u001b[0m \u001b[0mengine\u001b[0m \u001b[0;34m==\u001b[0m \u001b[0;34m\"python\"\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;32m/usr/local/lib/python3.6/dist-packages/pandas/io/parsers.py\u001b[0m in \u001b[0;36m__init__\u001b[0;34m(self, src, **kwds)\u001b[0m\n\u001b[1;32m   2008\u001b[0m         \u001b[0mkwds\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m\"usecols\"\u001b[0m\u001b[0;34m]\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0musecols\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   2009\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 2010\u001b[0;31m         \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_reader\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mparsers\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mTextReader\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0msrc\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkwds\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   2011\u001b[0m         \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0munnamed_cols\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_reader\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0munnamed_cols\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   2012\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;32mpandas/_libs/parsers.pyx\u001b[0m in \u001b[0;36mpandas._libs.parsers.TextReader.__cinit__\u001b[0;34m()\u001b[0m\n",
            "\u001b[0;32mpandas/_libs/parsers.pyx\u001b[0m in \u001b[0;36mpandas._libs.parsers.TextReader._get_header\u001b[0;34m()\u001b[0m\n",
            "\u001b[0;32mpandas/_libs/parsers.pyx\u001b[0m in \u001b[0;36mpandas._libs.parsers.TextReader._tokenize_rows\u001b[0;34m()\u001b[0m\n",
            "\u001b[0;32mpandas/_libs/parsers.pyx\u001b[0m in \u001b[0;36mpandas._libs.parsers.raise_parser_error\u001b[0;34m()\u001b[0m\n",
            "\u001b[0;31mParserError\u001b[0m: Error tokenizing data. C error: Calling read(nbytes) on source failed. Try engine='python'."
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 235
        },
        "id": "S_l_2FYBIyt1",
        "outputId": "5c15ea7a-1686-4fd2-9279-0e8acaa144e4"
      },
      "source": [
        "#2\n",
        "def html_loader(path=dataset_path_html):  \n",
        "  ''' \n",
        "  Reads html files and makes the text more readable \n",
        "  using beautiful soup's get text method.\n",
        "  Returns a list of strings, where each element is one html file's text\n",
        "  '''\n",
        "  import os\n",
        "  from bs4 import BeautifulSoup\n",
        "  #import urllib2\n",
        "  content=[]\n",
        "  #url = \"/content/drive/My Drive/Imperial college London/data/PMC3908470.html\"\n",
        "  for filename in os.listdir(path):\n",
        "      if filename.endswith(\".html\"):\n",
        "          fullpath = os.path.join(path, filename)\n",
        "          page = open(fullpath, errors='ignore') # encoding='mac_roman' because the files were created in mac , errors ignore if it can't open some files\n",
        "          soup = BeautifulSoup(page)\n",
        "          [s.extract() for s in soup(['style', 'script', '[document]', 'head', 'title'])]\n",
        "          soup=soup.get_text()\n",
        "          items = ' '.join(soup.split())\n",
        "          content.append(items)\n",
        "  print(\"Number of papers processed: \",len(content))\n",
        "  return content\n",
        "#print(html_loader())  \n",
        "content=html_loader()\n",
        "\n",
        "  "
      ],
      "execution_count": 13,
      "outputs": [
        {
          "output_type": "error",
          "ename": "NameError",
          "evalue": "ignored",
          "traceback": [
            "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
            "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
            "\u001b[0;32m<ipython-input-13-94c881806a23>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0;31m#2\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 2\u001b[0;31m \u001b[0;32mdef\u001b[0m \u001b[0mhtml_loader\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mpath\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mdataset_path_html\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      3\u001b[0m   ''' \n\u001b[1;32m      4\u001b[0m   \u001b[0mReads\u001b[0m \u001b[0mhtml\u001b[0m \u001b[0mfiles\u001b[0m \u001b[0;32mand\u001b[0m \u001b[0mmakes\u001b[0m \u001b[0mthe\u001b[0m \u001b[0mtext\u001b[0m \u001b[0mmore\u001b[0m \u001b[0mreadable\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      5\u001b[0m   \u001b[0musing\u001b[0m \u001b[0mbeautiful\u001b[0m \u001b[0msoup\u001b[0m\u001b[0;31m'\u001b[0m\u001b[0ms\u001b[0m \u001b[0mget\u001b[0m \u001b[0mtext\u001b[0m \u001b[0mmethod\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;31mNameError\u001b[0m: name 'dataset_path_html' is not defined"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "CRJPqtFdeCiX",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 270
        },
        "outputId": "a6647f73-a2bf-4fdf-ac14-3ae4a7709bac"
      },
      "source": [
        "#3\n",
        "# Easy way but computentianlly expensive\n",
        "import spacy\n",
        "nlp = spacy.load('en_core_web_sm')\n",
        "nlp.max_length=7000000\n",
        "#txt = '''I really like NMR spectroscopy. Electrospray ionization is my favorite form of ionizing things because it can be very versatile, and as such Electrospray ionization is widely used. The word chroma comes from the greek word for color, that is why Gas chromatography is called like that!'''\n",
        "#for i in range(0, len(content)):\n",
        "text = ''.join([str(elem) for elem in content]) \n",
        "#txt= train\n",
        "#print(txt)\n",
        "#txt= \"This is my NMR i like it. This is a random sentence. This is NMRSQPREWE, to try and confuse the model. I have some QQ wanna try ?\"\n",
        "tokens = nlp(text, disable=['ner'])\n",
        "sentences=[]\n",
        "for sent in tokens.sents:\n",
        "    sentences.append(sent.string.strip())\n",
        "print(sentences)\n"
      ],
      "execution_count": 12,
      "outputs": [
        {
          "output_type": "error",
          "ename": "NameError",
          "evalue": "ignored",
          "traceback": [
            "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
            "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
            "\u001b[0;32m<ipython-input-12-53d5b9104856>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      6\u001b[0m \u001b[0;31m#txt = '''I really like NMR spectroscopy. Electrospray ionization is my favorite form of ionizing things because it can be very versatile, and as such Electrospray ionization is widely used. The word chroma comes from the greek word for color, that is why Gas chromatography is called like that!'''\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      7\u001b[0m \u001b[0;31m#for i in range(0, len(content)):\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 8\u001b[0;31m \u001b[0mtext\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m''\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mjoin\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mstr\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0melem\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0melem\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mcontent\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      9\u001b[0m \u001b[0;31m#txt= train\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     10\u001b[0m \u001b[0;31m#print(txt)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;31mNameError\u001b[0m: name 'content' is not defined"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "o9WWjwnfJE8i",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 194
        },
        "outputId": "a43ccd0d-5ce3-4a6f-cdae-d371f8b808fc"
      },
      "source": [
        "#4\n",
        "# Beta V 1.1\n",
        "# IN PROGRESS\n",
        "# Version incldes finding nouns and then doing the matching \n",
        "# Efficient vs non Efficient way\n",
        "'''\n",
        "This code reads into a list all the assay methods from a csv file.\n",
        "A function takes a text as an input an a python list of assays. \n",
        "The function performs matching of all sentences that contain a word which \n",
        "is in the list. This is done using regular expression, the word is found \n",
        "and then the expression looks for puntcuation signs left from the word and \n",
        "right from the word so that it can extract the sentence which contains the word.\n",
        "\n",
        "The user can choose the given assay when running the function, \n",
        "or if he doesn't input a parameter it will go through the list of all assays in the file.\n",
        "\n",
        "'''\n",
        "#from google.colab import drive\n",
        "#drive.mount('/content/drive')\n",
        "\n",
        "import re #time complexity issues of finite state machines ?\n",
        "import pandas as pd\n",
        "import spacy\n",
        "import numpy as np\n",
        "\n",
        "assay=[]\n",
        "df = pd.read_csv(\"/content/drive/My Drive/Imperial college London/alla.csv\",dtype= str) # READ CSV AS STRING !!!\n",
        "assay = df.values.tolist()\n",
        "assay = list(map(''.join, assay)) # convert list of lists to list of strings \n",
        "\n",
        "\n",
        "def finder(text,user_assay):    \n",
        "    pattern1=r'[^.?!]*(?<=[.?\\s!])%s(?=[\\s.?!])[^.?!]*[.?!]' #extracts full sentences that contain a word \n",
        "    pattern2=r'\\b%s\\b' #extracts a given word \n",
        "    index=[]\n",
        "    for i in range(len(assay)):\n",
        "      tmp=re.findall(pattern2 %assay[i],text)  \n",
        "      if (len(tmp)>0):\n",
        "       index.append(i)\n",
        "    res_list = [assay[j] for j in index]\n",
        "    print(\"The assays mentioned are: \\n \", res_list, type(res_list))\n",
        "    return res_list\n",
        "    '''\n",
        "     #if the user chooses a custom assay not from the list\n",
        "     # One big if is needed to check whether user_assay is None or not\n",
        "    else:  \n",
        "     out2=re.findall(pattern % user_assay , text, re.MULTILINE )\n",
        "     if (len(out2) == 0):\n",
        "      print(\"No matching assays\")\n",
        "     else:\n",
        "      output.append(out2) \n",
        "      print(\"At least one match was found :\")\n",
        "      print('\\n'.join(out2))\n",
        "   '''\n",
        "\n",
        "# Run the program outside of the function  \n",
        "# Expected assay output: NMR spectroscopy, liquid chromatography, (LC)-mass spectrometry, gas chromatography-mass spectrometry, 1H NMR spectroscopy, NMR spcetrospcy  \n",
        "\n",
        "\n",
        "\n",
        "\n",
        "\n",
        "x=finder(text,\"\")\n",
        "\n",
        "#x=finder(text,\"\") # You should write \"\" instead of None for the input of the function\n",
        "'''\n",
        "nlp=spacy.load(\"en_core_web_sm\")\n",
        "doc=nlp(text)\n",
        "nouns=list(doc.noun_chunks)\n",
        "nouns=''.join(str(nouns)) # putting the nouns in a string because my function takes in a string\n",
        "nouns=nouns.replace(',','.') # I join them with dots because I tried to fix the regex to recognize commas but it didn't work (argument agains regex)\n",
        "in_txt=sentences # copy the list sentences to another list in_txt - sentences keeps intackt as a list of the sentences\n",
        "in_txt=''.join(str(in_txt)) # convert in_txt to string by joining it from self, I need it all as one string for the finder function \n",
        "# I keep sentences as a list but create a string which is my input text becuase I need a string for it to be readable with the regex, but a list of the loops\n",
        "#x=finder(in_txt,\"\") # You should write \"\" instead of None for the input of the function \n",
        "# x is a list of the assays found in the text , \n",
        "# in_txt is only needed to find the mentioned assays \n",
        "# Find span of all texts\n",
        "#Let's see if it works with overlapping matches \n",
        "'''\n"
      ],
      "execution_count": 20,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "The assays mentioned are: \n",
            "  ['(1)H NMR', 'NMR', '1H-NMR', '1H-NMR', 'NMR spectroscopy', '13C NMR', '13C NMR', 'COSY', 'CPMG', 'Carr-Purcell-Meiboom-Gill', 'Mass spectrometry', 'mass spectrometer', 'MS', 'tandem mass spectrometry', 'MS/MS', 'FID', 'Liquid chromatography', 'LC-MS', 'LC/MS', 'UPLC-MS', 'HPLC-MS', 'Ultra Performance LC', 'reverse-phase HPLC', 'reversed-phase HPLC', 'RP', 'LC-MS/MS', 'Electrospray ionization', 'ESI', 'TOF', 'QTOF', 'QqQ', 'DI', 'Orbitrap', 'quadrupole ion trap', 'SPE', 'Lipidomics', 'Thermo', 'Waters', 'Agilent', 'Thermo Fisher', 'Bruker', 'Varian', 'SCIEX'] <class 'list'>\n"
          ],
          "name": "stdout"
        },
        {
          "output_type": "execute_result",
          "data": {
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "string"
            },
            "text/plain": [
              "'\\nnlp=spacy.load(\"en_core_web_sm\")\\ndoc=nlp(text)\\nnouns=list(doc.noun_chunks)\\nnouns=\\'\\'.join(str(nouns)) # putting the nouns in a string because my function takes in a string\\nnouns=nouns.replace(\\',\\',\\'.\\') # I join them with dots because I tried to fix the regex to recognize commas but it didn\\'t work (argument agains regex)\\nin_txt=sentences # copy the list sentences to another list in_txt - sentences keeps intackt as a list of the sentences\\nin_txt=\\'\\'.join(str(in_txt)) # convert in_txt to string by joining it from self, I need it all as one string for the finder function \\n# I keep sentences as a list but create a string which is my input text becuase I need a string for it to be readable with the regex, but a list of the loops\\n#x=finder(in_txt,\"\") # You should write \"\" instead of None for the input of the function \\n# x is a list of the assays found in the text , \\n# in_txt is only needed to find the mentioned assays \\n# Find span of all texts\\n#Let\\'s see if it works with overlapping matches \\n'"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 20
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "Dza4Hbc-4x8k",
        "outputId": "0dce3204-576c-4f6a-8acb-8e5ede64248d"
      },
      "source": [
        "print(type(content))"
      ],
      "execution_count": 62,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "<class 'str'>\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "_spZMgZeY8-H",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 235
        },
        "outputId": "1ab3eaa0-24ef-4642-8ed7-5ec9f993b02c"
      },
      "source": [
        "#5\n",
        "# INPUT: List x of all mentioned assays , List sentences of all sentences in the text\n",
        "#=============================================================================\n",
        "\n",
        "\n",
        "#Saving to a JSON file that is compatible with Spacy NER\n",
        "#import json \n",
        "label=\"ASSAY\" # za ako nema entity isto bitno e !!, i ke treba starts i ends za sekoja recenica, ako se prazni is okay\n",
        "# ako e prva staj zagrada ako ne, nishto\n",
        "# Try to save in dictironary, each sentence has a seperate dictionary\n",
        "# Za ova so dolnava crta da naucam bolje kako se praj sho kako\n",
        "\n",
        "dicts = [{} for _ in range(len(sentences))] # create a list of dictionaries that has the same number of diconaries as sentences \n",
        "for k in range (len(sentences)):\n",
        "  d={\"content\":None,\"entities\":[]} # Specify the dictionary format \n",
        "  dicts[k].update(d)              # Write an empty list of dictionaries \n",
        "# Functions to append multiple values to a key in dictionary\n",
        "\n",
        "def add_content_in_dict(sample_dict, key, list_of_values):\n",
        "    \"\"\"Append multiple values to a key in the given dictionary\"\"\"\n",
        "    if key not in sample_dict:\n",
        "        sample_dict[key] = \"\"\n",
        "    sample_dict[key]=(list_of_values) # because it initializes, it replaces the first key value pair\n",
        "    return sample_dict\n",
        "\n",
        "def add_entites_in_dict(sample_dict, key, list_of_values):\n",
        "    \"\"\"Append multiple values to a key in the given dictionary\"\"\"\n",
        "    if key not in sample_dict:\n",
        "        sample_dict[key] = []\n",
        "    #if sample_dict.get(key)== []:\n",
        "        #sample_dict[key]=[list_of_values]\n",
        "    sample_dict[key].append(list_of_values)  # it adds to the first key value pair\n",
        "    return sample_dict\n",
        "\n",
        "# This loop finds the matches \"found entities in ALL sentences\" saved in the output of the finder function which is a list x[]\n",
        "# It looks for the span of matches and the m\n",
        "# OVERLAPPING ISSUE FIX\n",
        "s=0\n",
        "e=0\n",
        "for k in range(len(sentences)): # cycle through sentences\n",
        "  #print(\"Matches in\" , k , \"sentence:\")\n",
        "  add_content_in_dict(dicts[k],'content',sentences[k])\n",
        "  for j in range(len(x)): # look for all entites in a given sentence # LEN X VO ZAGRADATA ; len(x) e ustvari dolzianta na toj celiot tekst od funkcijta sho iskacha od finder\n",
        "    for m in re.finditer(x[j], sentences[k]):  # cycle through all charaters in a given sentence with a given entity\n",
        "        #print(m.start(), m.end(), m.group()) # print the starting char, ending char , and match IN THE given sentence\n",
        "        #temp=m.end() - m.start()\n",
        "        if (m.end() - m.start() > e - s):\n",
        "          add_entites_in_dict(dicts[k],'entities',(m.start(),m.end(),label)) # updatni entities za TOJ istiot content    \n",
        "          e=m.end()\n",
        "          s=m.start()\n",
        "          continue\n",
        "        else:\n",
        "          add_entites_in_dict(dicts[k],'entities',(s,e,label))\n",
        "          e=m.end()\n",
        "          s=m.start()\n",
        "          continue\n",
        "        \n",
        "        \n",
        "\n",
        "        #[19,30,\"subject\"]\n",
        "        #add_entites_in_dict(dicts[k],'entities',[])\n",
        "        #starts.append(m.start())  # add to a list of all starting characters\n",
        "        #ends.append(m.end())      # add to a list of all ending charachters \n",
        "\n",
        "#starts.sort()  # sort the lists because I need the pairs of elements to find the span of the match\n",
        "#ends.sort()    # I will first try with overlapping entites but I might also correct that \n",
        "#print(starts,ends)\n",
        "\n",
        "# Try to save in dictironary, each sentence has a seperate dictionary\n",
        "# Za ova so dolnava crta da naucam bolje kako se praj sho kako\n",
        "\n",
        "# Granicniot slucaj za koa nema najdeno entities sho da stai\n",
        "fulls=[]   # list of all the data with found entities  \n",
        "empties=[]  # list of all the data with empty entities \n",
        "fin=[]   # final list with 80% fulls and 20% of empties\n",
        "\n",
        "dr = [{} for _ in range(len(sentences))] #\n",
        "for k in range (len(sentences)):\n",
        "  d={'entities':dicts[k].get('entities')}\n",
        "  dr[k].update(d) \n",
        "\n",
        "\n",
        "\n",
        "for i in range (len(sentences)):\n",
        "  elem=(dicts[i].get('content'),dr[i])\n",
        "  if (dr[i].get('entities')!=[]):  # append only the ones that have enttites in them\n",
        "   fulls.append(elem)\n",
        "  else:\n",
        "    empties.append(elem)\n",
        "\n",
        "\n",
        "#====================================\n",
        "#\n",
        "# TUKA DA SE STAVI FUNKCIJATA NEGDE\n",
        "#\n",
        "#======================================\n",
        "\n",
        "\n",
        "if (len(empties)<=(len(fulls)//10)):\n",
        "  fin = fulls + empties\n",
        "else:\n",
        "  fin = fulls + empties[(len(fulls)//10):len(fulls)//5]   # I put 10% of the empties beginnign from the first 10 percent and until the second 10%\n",
        "# fin  =   [  (get.key, {'entities': [(),()]}) , (get.key, {'entities': [(),()]})   ]\n",
        "#dicts[0].get('content') # ova e getkey\n",
        "#print(len(fin))\n",
        "print(fin)\n",
        "\n",
        "\n",
        "import json\n",
        "\n",
        "with open('test.json', 'w') as fout:\n",
        "    json.dump(fin , fout, sort_keys=True, indent=1 ) # save json in pretty format\n",
        "\n",
        "\n",
        "with open('test.json', \"r\") as read_file:   # read json file \n",
        "    data_train = json.load(read_file) \n",
        "print(data_train)\n"
      ],
      "execution_count": 27,
      "outputs": [
        {
          "output_type": "error",
          "ename": "NameError",
          "evalue": "ignored",
          "traceback": [
            "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
            "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
            "\u001b[0;32m<ipython-input-27-b7c20d538f1b>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m     11\u001b[0m \u001b[0;31m# Za ova so dolnava crta da naucam bolje kako se praj sho kako\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     12\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 13\u001b[0;31m \u001b[0mdicts\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0;34m{\u001b[0m\u001b[0;34m}\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0m_\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mrange\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mlen\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0msentences\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m]\u001b[0m \u001b[0;31m# create a list of dictionaries that has the same number of diconaries as sentences\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     14\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mk\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mrange\u001b[0m \u001b[0;34m(\u001b[0m\u001b[0mlen\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0msentences\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     15\u001b[0m   \u001b[0md\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;34m{\u001b[0m\u001b[0;34m\"content\"\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;32mNone\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\"entities\"\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m}\u001b[0m \u001b[0;31m# Specify the dictionary format\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;31mNameError\u001b[0m: name 'sentences' is not defined"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "AjiMFC_MaevP",
        "outputId": "248ef0d3-533e-4aa2-971e-1efaec4ebea1"
      },
      "source": [
        "# 21 load files and clear out\r\n",
        "from google.colab import drive\r\n",
        "drive.mount('/content/drive')\r\n",
        "import json\r\n",
        "with open('/content/drive/My Drive/Imperial college London/test_1.json', \"r\") as read_file:   # read json file \r\n",
        "    data_train = json.load(read_file) \r\n",
        "\r\n",
        "test=data_train\r\n",
        "tmp=[]\r\n",
        "\r\n",
        "for i in range (0, len(test)):\r\n",
        "  tmp.append(test[i][1].get('entities')) # [[3, 6, 'ASSAY'], [0, 6, 'ASSAY'], [0, 6, 'ASSAY']]  ;   [[111, 113, 'ASSAY']]\r\n",
        "\r\n",
        "\r\n",
        "# we creating a list called tmp, it will have all the entities, each element is one sentence. \r\n",
        "# tmp[i] is the input to the function \r\n",
        "'''\r\n",
        "the function would return what tmp[i] should have --> data train[i][1].get('entities'))\r\n",
        "'''\r\n",
        "\r\n",
        "#print(tmp[0][0][0]) # this one is 3 ;  1. which list 2. whcih element in the list 3. 0 - start 1- ends - 2- label \r\n",
        "\r\n",
        "\r\n",
        "\r\n",
        "\r\n",
        "def jac(args):\r\n",
        "  '''\r\n",
        "  input is [[3, 6, 'ASSAY'], [0, 6, 'ASSAY'], [0, 6, 'ASSAY']] -> matches in first sentence\r\n",
        "  It should be ran for each sentence!\r\n",
        "  If the function had variable parameters \r\n",
        "  *args should be put instead of args and read as follows:\r\n",
        "  lis=[]\r\n",
        "  for arg in args:\r\n",
        "    lis.append(arg)\r\n",
        "  print(lis[0][0])\r\n",
        "  ''' \r\n",
        "  lis=[]\r\n",
        "  lis.append(args)\r\n",
        "  # lis = [[3, 6, 'ASSAY'], [0, 6, 'ASSAY'],[0, 6, 'ASSAY']]\r\n",
        "  '''\r\n",
        "  # In order to use set theory operations we first \r\n",
        "  # make a list  of lists, where each element is a list \r\n",
        "  # with elements between two values e.g [3,6, \"ASSAY\"] = [3,4,5,6]\r\n",
        "  # Creates a list between the ranges od the starting and ending match character\r\n",
        "  '''\r\n",
        "\r\n",
        "  dlis=[] \r\n",
        "  for i in range (0, len(lis)):  \r\n",
        "    for j in range (0, len(lis[0])):\r\n",
        "      dlis.append(list(range(lis[i][j][0],lis[i][j][1],1)))  \r\n",
        "              \r\n",
        "  #********************************************************\r\n",
        "\r\n",
        "  dlis = set ( tuple (x) for x in dlis) # clearning out repetitions\r\n",
        "  dlis = list (list (x) for x in dlis)  # back to list to access elements by indexing\r\n",
        "  #no repeating arrays\r\n",
        "  dlis=(sorted(dlis, key=len)) # sort inner arrays by length\r\n",
        "  \r\n",
        "  to_remove=[] # list of elements needed to be removed\r\n",
        "  ind=[]   # list of indices of dlis that need to be removed\r\n",
        "\r\n",
        "  ###############################################################################\r\n",
        "    # On to detecting overlapping matches\r\n",
        "  for i in range(0, len(dlis)):    # calucalte Jacardi index between evey element and each forward element\r\n",
        "    for j in range(i+1,len(dlis)):   # if there is an overlap, keep the longest \r\n",
        "        #print(dlis[i],dlis[j])\r\n",
        "        J=len(set(dlis[i]) & set(dlis[j])) / len(set(dlis[i]) | set(dlis[j]))\r\n",
        "        #print(J)\r\n",
        "        if (J!=0.0):   # if there is an overlap, write the element that should be removed \r\n",
        "           to_remove.append(dlis[i])\r\n",
        "           ind.append(i)    # it is the i-th element because the list is sorted\r\n",
        "           break # If it is similar with at least one, break the cycle\r\n",
        "        \r\n",
        "  result_list=[]\r\n",
        "  for i in range (0 , len(dlis)):\r\n",
        "     if (i not in ind):    # if \"i\" is not in \"to_remove\" then append it to results\r\n",
        "       result_list.append(dlis[i])  # taking the indecies\r\n",
        "   \r\n",
        "   \r\n",
        "   # just create a nested list because that's the needet output format \r\n",
        "   #\r\n",
        "  out=[]\r\n",
        "  for i in range (len(result_list)): \r\n",
        "    out.append([]) \r\n",
        "\r\n",
        "   # adapt the output to make it in the right format\r\n",
        "   # and turn the sets back to [start,end,label] format\r\n",
        "  label='ASSAY'  \r\n",
        "  for i in range (0, len(result_list)):\r\n",
        "      out[i].append(result_list[i][0])\r\n",
        "      out[i].append(result_list[i][0] + len(result_list[i]))\r\n",
        "      out[i].append(label)\r\n",
        "  out = sorted (out)\r\n",
        "  \r\n",
        "  return out\r\n",
        "\r\n",
        "z=21\r\n",
        "print(tmp[z])\r\n",
        "print(\"The processed output is:\")\r\n",
        "print(jac(tmp[z]))\r\n",
        "print(type(test[1][1]))\r\n",
        "\r\n",
        "# MEsto test mozda train za da ne se editne\r\n",
        "for i in range (0,len(test)):\r\n",
        "  test[i][1].update( entities = jac(tmp[i]) )\r\n",
        "\r\n",
        "#for i in range (0, len(test)):\r\n",
        "  #print(\"New data:\" , test[i][1] )\r\n",
        "\r\n",
        "print(test)   \r\n",
        "data_train=test"
      ],
      "execution_count": 62,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Drive already mounted at /content/drive; to attempt to forcibly remount, call drive.mount(\"/content/drive\", force_remount=True).\n",
            "[[84, 87, 'ASSAY'], [108, 111, 'ASSAY']]\n",
            "The processed output is:\n",
            "[[84, 87, 'ASSAY'], [108, 111, 'ASSAY']]\n",
            "<class 'dict'>\n",
            "[['1H-NMR-Based Metabolomic Study for Identifying Serum Profiles Associated with t...', {'entities': [[0, 6, 'ASSAY']]}], ['1H-NMR-Based Metabolomic Study for Identifying Serum Profiles Associated with the Response to Etanercept in Patients with Rheumatoid ArthritisPLoS ONE.', {'entities': [[0, 6, 'ASSAY']]}], ['Discriminating high-risk cervical Human Papilloma Virus infections with urinary biomarkers via non-targeted GC-MS-based metabolomicsPLoS ONE.', {'entities': [[111, 113, 'ASSAY']]}], ['2018; 13(12) Prognosis Biomarkers of Severe Sepsis and Septic Shock by 1H NMR Urine Metabolom...', {'entities': [[71, 77, 'ASSAY']]}], ['Prognosis Biomarkers of Severe Sepsis and Septic Shock by 1H NMR Urine Metabolomics in the Intensive Care', {'entities': [[58, 64, 'ASSAY']]}], ['PMC5724867PMID: 29228049Correction: Assessment of gold nanoparticles on human peripheral blood cells by metabolic profiling with 1H-NMR spectroscopy, a novel translational approach on a patient-specific basisMartina Palomino-Schtzlein, Hermenegildo Garca, Patricia Gutirrez-Carcedo, Antonio Pineda-Lucena, and Jos Raul Herance Copyright and License information', {'entities': [[132, 148, 'ASSAY']]}], ['This corrects the article \"Assessment of gold nanoparticles on human peripheral blood cells by metabolic profiling with 1H-NMR spectroscopy, a novel translational approach on a patient-specific basis\" in volume 12, e0182985.There is an error in affiliation 3 for authors Patricia Gutirrez-Carcedo and Jos Raul Herance.', {'entities': [[123, 139, 'ASSAY']]}], ['Palomino-Schtzlein M, Garca H, Gutirrez-Carcedo P, Pineda-Lucena A, Herance JR (2017) Assessment of gold nanoparticles on human peripheral blood cells by metabolic profiling with 1H-NMR spectroscopy, a novel translational approach on a patient-specific basis.', {'entities': [[182, 198, 'ASSAY']]}], ['Manage collections... loading Similar articles in PubMed Assessment of gold nanoparticles on human peripheral blood cells by metabolic profiling with 1H-NMR spectroscopy, a novel translational approach on a patient-specific basis.[PLoS One.', {'entities': [[153, 169, 'ASSAY']]}], ['On Correction: Assessment of gold nanoparticles on human peripheral blood cells by ...Correction: Assessment of gold nanoparticles on human peripheral blood cells by metabolic profiling with 1H-NMR spectroscopy, a novel translational approach on a patient-specific basisPLoS ONE.', {'entities': [[194, 210, 'ASSAY']]}], ['2014; 9(10) Development and validation of a multiplex UHPLC-MS/MS method for the determinati...', {'entities': [[57, 65, 'ASSAY']]}], ['Development and validation of a multiplex UHPLC-MS/MS method for the determination of the investigational antibiotic against multi-resistant tuberculosis macozinone', {'entities': [[45, 53, 'ASSAY']]}], ['Here, a NMR-based metabonomic approach was used to preliminarily identify sex-specific urinary metabolite biomarkers for diagnosing male and female BD patients.', {'entities': [[8, 11, 'ASSAY']]}], ['Nuclear magnetic resonance spectroscopy (NMR), one of three major analytical platforms with proven utility in metabonomic applications, has proven to be a well-established strategy for identifying disease-specific biomarkers because of its highly sensitive, high-throughput molecular screening capability [8].', {'entities': [[41, 44, 'ASSAY']]}], ['Using this method, our group has identified a panel of potential metabolite biomarkers capable of discriminating major depressive disorder (MDD) subjects from healthy controls (HC) with both high sensitivity and specificity [9], [10].In regards to BD, several researchers also performed studies using NMR-based metabolomic platforms.', {'entities': [[301, 304, 'ASSAY']]}], ['Using a NMR-based metabolomic platform, our group has identified a panel of metabolite biomarkers capable of discriminating BD subjects from HC with a sensitivity of 79.1% and a specificity of 79.2% [13].However, these previous studies have not taken sex-based differences into consideration.', {'entities': [[8, 11, 'ASSAY']]}], ['Therefore, in this study, a NMR-based metabonomic approach was applied to preliminarily identify sex-specific urinary metabolite biomarkers for separately diagnosing male and female BD patients.', {'entities': [[28, 31, 'ASSAY']]}], ['The resulting supernatant, after centrifugation at 1500 g for 10 min, was immediately divided into equal aliquots and stored at −80°C.NMR AcquisitionPrior to NMR analysis, urine samples were thawed and centrifuged at 1500 g for 10 min to remove precipitation.', {'entities': [[134, 137, 'ASSAY'], [158, 161, 'ASSAY']]}], ['After centrifugation at 12000 rpm for 10 min, 500 µl samples of supernatant were transferred into 5 mm NMR tubes.', {'entities': [[103, 106, 'ASSAY']]}], ['The proton spectra of the urine samples were collected on a Bruker Avance 600 spectrometer operating at a 600.13 MHz', {'entities': [[60, 66, 'ASSAY']]}], ['Typically, 64 transients were collected into 16K data points with a spectral width of 8000 Hz, an acquisition time of 0.945 s, and a relaxation delay of 2 s. Prior to Fourier transformation, the free induction decay (FID) was zero-filled and multiplied by an exponential function corresponding to a line-broadening factor of 0.3 Hz in frequency domain.', {'entities': [[217, 220, 'ASSAY']]}], ['Urine resonance assignments were performed according to the previous literature and NMR databases [6], [10].NMR Spectral ProcessingWe used the 3-trimethylsilyl-1-[2,2,3,3-2H4] propionate (TSP) as an internal standard to confirm the change in the urinary levels of the metabolites.', {'entities': [[84, 87, 'ASSAY'], [108, 111, 'ASSAY']]}], ['The NMR spectra (0.5–9.5 ppm) were segmented into equal widths (0.005 ppm) using the AMIX package (Bruker Biospin, Germany).', {'entities': [[4, 7, 'ASSAY'], [99, 105, 'ASSAY']]}], ['One study reported that synthesis and degradation of ketone bodies was the most influenced metabolic pathways associated with CUMS-induced depression [25].', {'entities': [[128, 130, 'ASSAY']]}], ['Finally, due to the diverse physicochemical properties and wide concentration range of metabolite biomarkers, future studies should employ additional metabolomic methods or multiple metabolomic platforms [29], [30].Go to:ConclusionsIn conclusion, using a NMR-based metabonomic approach, two sex-specific urinary metabolite biomarker panels for diagnosing male and female BD patients were identified here.', {'entities': [[255, 258, 'ASSAY']]}], ['employing 1H NMR spectroscopy and chemometrics.', {'entities': [[13, 29, 'ASSAY']]}], ['(2005) Evaluation of the orthogonal projection on latent structure model limitations caused by chemical shift variability and improved visualization of biomarker changes in 1H NMR spectroscopic metabonomic studies.', {'entities': [[173, 179, 'ASSAY']]}], ['(2013) Integration of 1H NMR and UPLC-Q-TOF/MS for a comprehensive urinary metabonomics study on a rat model of depression induced by chronic unpredictable mild stress.', {'entities': [[22, 28, 'ASSAY'], [40, 43, 'ASSAY'], [44, 46, 'ASSAY']]}], ['Combined Application of NMR- and GC-MS-Based Metabonomics Yields a Superior Urinary Biomarker Panel for Bipolar Disorder.', {'entities': [[24, 27, 'ASSAY'], [36, 38, 'ASSAY']]}], ['employing 1H NMR spectroscopy and chemometrics.', {'entities': [[13, 29, 'ASSAY']]}], ['[Ref list]Evaluation of the orthogonal projection on latent structure model limitations caused by chemical shift variability and improved visualization of biomarker changes in 1H NMR spectroscopic metabonomic studies.', {'entities': [[176, 182, 'ASSAY']]}], ['[Ref list]SOME STEP-DOWN PROCEDURES CONTROLLING THE FALSE DISCOVERY RATE UNDER DEPENDENCE.Ge Y, Sealfon SC, Speed TPStat Sin.', {'entities': [[58, 60, 'ASSAY']]}], ['[Ref list]Integration of ¹H NMR and UPLC-Q-TOF/MS for a comprehensive urinary metabonomics study on a rat model of depression induced by chronic unpredictable mild stress.', {'entities': [[28, 31, 'ASSAY'], [43, 46, 'ASSAY'], [47, 49, 'ASSAY']]}], ['list]Combined application of NMR- and GC-MS-based metabonomics yields a superior urinary biomarker panel for bipolar disorder.', {'entities': [[29, 32, 'ASSAY'], [41, 43, 'ASSAY']]}], [': PMC2951909PMID: 20949041 1H-NMR-Based Metabolomic Profiling of CSF in Early Amyotrophic Lateral SclerosisHélène Blasco, 1 , 2 , 3 , *', {'entities': [[27, 33, 'ASSAY']]}], ['The aim of this study was to analyze the CSF of patients with ALS by 1H NMR (Nuclear Magnetic Resonance)', {'entities': [[69, 75, 'ASSAY']]}], ['One and two-dimensional 1H NMR analyses were performed and metabolites were quantified by the ERETIC method.', {'entities': [[24, 30, 'ASSAY']]}], ['screening by NMR spectroscopy could be a useful, simple and low cost tool to improve the early diagnosis of ALS.', {'entities': [[13, 29, 'ASSAY']]}], ['Metabolomic studies have been performed via different analytical methods such as high performance liquid chromatography followed by electrochemical detection [8] or high resolution 1H NMR (Nuclear Magnetic Resonance)', {'entities': [[181, 187, 'ASSAY']]}], ['NMR spectroscopy appears to be cost-effective, useful in routine care, and screening [10].', {'entities': [[0, 16, 'ASSAY']]}], ['The aim of this study was to search for a metabolic signature of ALS in the CSF using high resolution NMR spectroscopy.', {'entities': [[102, 118, 'ASSAY']]}], ['We compared 1H NMR spectra of CSF samples collected from ALS patients and from patients without a neurodegenerative disease.', {'entities': [[12, 18, 'ASSAY']]}], ['For our study, we used a part of CSF sample to perform complementary analysis with NMR.', {'entities': [[83, 86, 'ASSAY']]}], ['Before the NMR experiments, samples were thawed, and centrifugated at 3000g for 5 minutes.', {'entities': [[11, 14, 'ASSAY']]}], ['Then 0.5 mL of CSF were mixed with 0.1 mL of deuterium oxyde solution to calibrate the NMR spectrometer.', {'entities': [[87, 90, 'ASSAY']]}], ['Finally, 0.6 mL of the prepared solutions were transferred to a 5-mm NMR tube (CortecNet, Paris, France) for 1H NMR analysis.', {'entities': [[69, 72, 'ASSAY'], [109, 115, 'ASSAY']]}], ['1H NMR spectra were performed on a Bruker DRX-500 spectrometer (Bruker SADIS, Wissembourg, France), operating at 11.7 T, with a Broad Band Inverse (BBI) probehead equipped with Z gradient coil.', {'entities': [[0, 6, 'ASSAY'], [35, 41, 'ASSAY'], [64, 70, 'ASSAY'], [73, 75, 'ASSAY']]}], ['NMR measurements were done at 298K, with non-spinning samples.', {'entities': [[0, 3, 'ASSAY']]}], ['CPMG', {'entities': [[0, 4, 'ASSAY']]}], ['1H NMR spectra were recorded with a spin-spin relaxation delay of 80ms (echo time 200 µs repeated 200 times), collected with 128 transients, eight dummy scans and a d1 of 10s into 32k data points with a spectral width of 7500 Hz and an acquisition time of 2.04s.', {'entities': [[0, 6, 'ASSAY']]}], ['COSY spectra were acquired with 3s relaxation delay, 6000Hz spectral width in both directions.', {'entities': [[0, 4, 'ASSAY']]}], ['The ERETIC pulse was generated by a second radio frequency channel phase-synchronized with the NMR signal of the sample.', {'entities': [[95, 98, 'ASSAY']]}], ['Data were processed using XWinNMR version 3.5 software (Bruker Daltonik, Karlsruhe, Germany).', {'entities': [[30, 33, 'ASSAY'], [56, 62, 'ASSAY']]}], ['Prior to Fourier transformation (FT), the 1D FIDs were zero-filled to 64K data points which provided sufficient data points for each resonance, and a line broadening factor of 0.3Hz was applied.', {'entities': [[45, 48, 'ASSAY']]}], ['Each spectrum was integrated using WinNMR software integral function.', {'entities': [[38, 41, 'ASSAY']]}], ['Among the 53 metabolites identifiable in CSF by NMR [17], we quantified (by XWin NMR software) those having the highest signal, i.e quantifiable with the highest accuracy.', {'entities': [[48, 51, 'ASSAY'], [81, 84, 'ASSAY']]}], ['(0.24–6.23)0.89Glucose (mmol/L)3.3(0.56–7.16)3.2 (0.15–8.9)0.51Open in a separate windowData are expressed in median (range).Qualitative 1H-NMR analysis and identificationThe 1H-NMR signals of all common metabolites including amino-acids, organic acids, and carbohydrates were assigned according to previous publications.', {'entities': [[137, 143, 'ASSAY'], [175, 181, 'ASSAY']]}], ['in a separate windowFigure 1CSF spectra by 1H NMR from ALS patient (A) and non ALS patient (B).Univariate data analysisWe analysed 17 CSF metabolites in ALS and non ALS patients, defined as follows: amino-acids (alanine, glutamine, tyrosine), organic acids (citrate, acetate, α-hydroxybutyrate (AHBT)), ketone bodies (β-hydroxybutyrate (BHBT), acetone, acetoacetate), glucose, fructose, metabolites involved in glucose metabolism (pyruvate, lactate, creatinine and creatine, recently identified as markers of mitochondrial dysfunction [18], the anti-oxidant molecule ascorbate, and formate as well as ethanol.', {'entities': [[43, 49, 'ASSAY']]}], ['Table 2CSF metabolite profiles in ALS and control subjects measured by 1H NMR spectroscopy.', {'entities': [[74, 90, 'ASSAY']]}], ['PCA of the 1H-NMR metabolite differences between ALS and non-ALS patientsPCA analysis was applied to the NMR data using the 17 metabolites integrated in all patients.', {'entities': [[11, 17, 'ASSAY'], [105, 108, 'ASSAY']]}], ['The PCA scores plot for the quantitative NMR data showed separated clusters for ALS and non ALS patients.', {'entities': [[41, 44, 'ASSAY']]}], ['NMR spectroscopy is a non-invasive technique, needs little sample preparation and gives an overview of the principal metabolic pathways.', {'entities': [[0, 16, 'ASSAY']]}], ['To our knowledge, the present study is the first to address the potential use of 1H NMR spectroscopy in analysing CSF from ALS patients.', {'entities': [[84, 100, 'ASSAY']]}], ['Importantly, glutamate could not be detected in CSF of patients and controls by NMR technique.', {'entities': [[80, 83, 'ASSAY']]}], ['Even if there may be an increase of glutamate in ALS patients, the NMR methodology is not able to pick up such very low concentrations.', {'entities': [[67, 70, 'ASSAY']]}], ['However, metabolism of acetone is different than for the other ketones, perhaps explaining the still moderately higher level of acetone in ALS patients [31].Based on these data, we used two statistical approaches to assess diagnostic consistency in ALS.First, we performed the multivariate analysis PCA on the 17 metabolites detected by 1H NMR spectroscopy to provide a metabolic signature for ALS.', {'entities': [[340, 356, 'ASSAY']]}], ['The scatter plot of PCA results derived from the NMR spectra discriminated between ALS and non-ALS populations.', {'entities': [[49, 52, 'ASSAY']]}], ['This study demonstrates that 1H NMR spectroscopy combined with multivariate analysis can detect changes in the concentrations of some CSF metabolites in ALS, and highlights specific metabolic pathway perturbations.', {'entities': [[32, 48, 'ASSAY']]}], ['To our knowledge, this study is the first one reporting in CSF the use of 1H NMR spectroscopy to explore metabolic pathways involved in the pathogenesis of ALS.', {'entities': [[77, 93, 'ASSAY']]}], ['This study focuses on the NMR profile, but we are aware that accuracy of early diagnosis of ALS will depend on combination of several approaches like imaging, electrophysiological and biological markers.', {'entities': [[26, 29, 'ASSAY']]}], ['Metabolomic analysis of serum by (1) H NMR spectroscopy in amyotrophic lateral sclerosis.', {'entities': [[39, 55, 'ASSAY']]}], ['Ala-Korpela M. Potential role of body fluid 1H NMR metabonomics as a prognostic and diagnostic tool.', {'entities': [[44, 50, 'ASSAY']]}], ['; 1(2):101-108.[PubMed] [Ref list]Metabolomic analysis of serum by (1) H NMR spectroscopy in amyotrophic lateral sclerosis.', {'entities': [[73, 89, 'ASSAY']]}], ['[Ref list]Review Potential role of body fluid 1H NMR metabonomics as a prognostic and diagnostic tool.', {'entities': [[46, 52, 'ASSAY']]}], ['MCB MS GR PJB.', {'entities': [[4, 6, 'ASSAY']]}], ['Wrote the paper: MCB MS GR PJB.', {'entities': [[21, 23, 'ASSAY']]}], ['Mass spectra of catechol estrogen quinones from Schistosoma haematobium infected individuals by MS (parent m/z) and MS/MS', {'entities': [[96, 98, 'ASSAY'], [116, 121, 'ASSAY']]}], ['3 and obtained by LC/DAD-ESI/MS.', {'entities': [[25, 28, 'ASSAY'], [29, 31, 'ASSAY']]}], ['The maximum peak of each (arrows) was submitted to CID/MS/MS, where the collisions released the mass spectra of the “daughters” (bottom image: encircled).', {'entities': [[55, 60, 'ASSAY']]}], ['Urine was collected from the participants for egg-based parasitological assessment of schistosome infection, and for liquid chromatography diode array detection electron spray ionization mass spectrometry (LC/UV-DAD/ESI-MSn) to investigate estrogen metabolites in the urine.', {'entities': [[216, 219, 'ASSAY'], [220, 222, 'ASSAY']]}], ['(LC/UV-DAD/ESI-MSn)Analysis using LC/UV-DAD/ESI-MSn was performed on a Finnigan Surveyor Plus High-Performance Liquid Chromatography (HPLC) instrument (ThermoFinnigan, San Jose, CA) equipped with a diode-array detector and a mass detector.', {'entities': [[11, 14, 'ASSAY'], [15, 17, 'ASSAY'], [44, 47, 'ASSAY'], [48, 50, 'ASSAY'], [152, 158, 'ASSAY']]}], ['The HPLC system included a quaternary pump, autosampler, degasser, photodiode-array detector and automatic thermostatic column compartment, and was controlled by the Xcalibur software package (ThermoFinnigan).', {'entities': [[193, 199, 'ASSAY']]}], ['The mass detector was a Finnigan Surveyor LCQ XP MAX quadrupole ion trap mass spectrometer (ThermoFinnigan) equipped with an electron spray ionization (ESI) interface.', {'entities': [[53, 72, 'ASSAY'], [73, 90, 'ASSAY'], [92, 98, 'ASSAY'], [152, 155, 'ASSAY']]}], ['The instrument was operated in a negative-ion mode with an ESI needle voltage of 5.00 kV and capillary temperature of 325 °C.', {'entities': [[59, 62, 'ASSAY']]}], ['Mass spectrometry (MSn) data were simultaneously acquired for the selected precursor ion.', {'entities': [[0, 17, 'ASSAY'], [19, 21, 'ASSAY']]}], ['Collision Induced Dissociation-Tandem Mass Spectrometry (CID-MS/MS) and MSn analyses were performed using helium with collision energy of 25–35 eV.', {'entities': [[61, 66, 'ASSAY'], [72, 74, 'ASSAY']]}], ['The flow rate was 0.3 ml min−1 followed to split out in 200 µl min−1 to MS.', {'entities': [[72, 74, 'ASSAY']]}], ['The instrument was calibrated with caffeine (Sigma-Aldrich), Met-Arg-Phe-Ala (MRFA) (tetrapeptide, Thermo Finnigan), and Ultramark 1621 (Lancaster Synthesis, Ward Hill, MA) in the mass range of 195–1,821 m/z.', {'entities': [[99, 105, 'ASSAY']]}], ['An advantage of inclusion of a MS detector over conventional HPLC analysis is that the number of channels in the detector can be set to specifically and separately identify all estrogen related compounds in a single injection of the urine sample.', {'entities': [[31, 33, 'ASSAY']]}], ['Novel estrogen-like metabolites detected in urine of females positive for eggs of S. haematobium Analysis of urine was undertaken using by LC/DAD/ESI-MSn.', {'entities': [[146, 149, 'ASSAY'], [150, 152, 'ASSAY']]}], ['We analyzed the estrogen-related compounds using LC/DAD/ESI-MSn to yield the typical intensity spectra of the representative estrogen derivatives obtained by LC (Fig. 3).', {'entities': [[56, 59, 'ASSAY'], [60, 62, 'ASSAY']]}], ['This produced the ‘parent’ mass spectrum by MS (parent m/z).', {'entities': [[44, 46, 'ASSAY']]}], ['The corresponding RT of each peak was submitted to LC/DAD-ESI/MS and revealed a unique high peak in each -- 305.33, 269.33, 481.33 and 495.33 mz, respectively.', {'entities': [[58, 61, 'ASSAY'], [62, 64, 'ASSAY']]}], ['Thereafter, each of these peaks was submitted to ICD/MS/MS, which produced the mass spectra for the ‘daughter’ ions (daughters m/z) (Fig. S1).', {'entities': [[53, 58, 'ASSAY']]}], ['Open in a separate windowFigure 4Chemical structures, retention times (RT) and m/z values for the four main components identified by LC–MS analysis of urine of Schistosoma haematobium-infected females.', {'entities': [[136, 138, 'ASSAY']]}], ['Here, estrogen-like metabolites similar to those identified previously in adult worm and eggs stages of S. haematobium [13], [14] were detected by LC–MS in urine of study participants with FGS, and were statistically associated with self-reported infertility in the study volunteers.', {'entities': [[150, 152, 'ASSAY']]}], ['Go to:Supporting InformationFigure S1 Mass spectra of catechol estrogen quinones from Schistosoma haematobium infected individuals by MS (parent m/z) and MS/MS', {'entities': [[134, 136, 'ASSAY'], [154, 159, 'ASSAY']]}], ['3 and obtained by LC/DAD-ESI/MS.', {'entities': [[25, 28, 'ASSAY'], [29, 31, 'ASSAY']]}], ['The maximum peak of each (arrows) was submitted to CID/MS/MS, where the collisions released the mass spectra of the “daughters” (bottom image: encircled).', {'entities': [[55, 60, 'ASSAY']]}], ['Conceived and designed the experiments: AT GL JM MS JdL RG.', {'entities': [[49, 51, 'ASSAY']]}], ['AT GL JM MS JdL', {'entities': [[9, 11, 'ASSAY']]}], ['(LC-MS).ResultsPacing-stress resulted in a 225% increase in the median rate·pressure product from baseline.', {'entities': [[1, 6, 'ASSAY']]}], ['Metabolite AnalysisMetabolite profiling was performed by liquid chromatography-mass spectrometry (LC-MS) methods as previously described [4] and also summarized in Text S1 in File S1.', {'entities': [[98, 103, 'ASSAY']]}], ['Sabatine MS, Liu E, Morrow DA, Heller E, McCarroll R, et al.', {'entities': [[9, 11, 'ASSAY']]}], ['Turer AT, Addo TA, Martin JL, Sabatine MS, Lewis GD, et al.', {'entities': [[39, 41, 'ASSAY']]}], ['Sabatine MS, Liu E, Morrow DA, Heller E, McCarroll R, Wiegand R, Berriz GF, Roth FP, Gerszten RECirculation.', {'entities': [[9, 11, 'ASSAY']]}], ['Lewis GD, Wei R, Liu E, Yang E, Shi X, Martinovic M, Farrell L, Asnani A, Cyrille M, Ramanathan A, Shaham O, Berriz G, Lowry PA, Palacios IF, Taşan M, Roth FP, Min J, Baumgartner C, Keshishian H, Addona T, Mootha VK, Rosenzweig A, Carr SA, Fifer MA, Sabatine MS, Gerszten REJ Clin Invest.', {'entities': [[259, 261, 'ASSAY']]}], ['Lewis GD, Farrell L, Wood MJ, Martinovic M, Arany Z, Rowe GC, Souza A, Cheng S, McCabe EL, Yang E, Shi X, Deo R, Roth FP, Asnani A, Rhee EP, Systrom DM, Semigran MJ, Vasan RS, Carr SA, Wang TJ, Sabatine MS, Clish CB, Gerszten RESci Transl Med.', {'entities': [[203, 205, 'ASSAY']]}], ['Turer AT, Addo TA, Martin JL, Sabatine MS, Lewis GD, Gerszten RE, Keeley EC, Cigarroa JE, Lange RA, Hillis LD, de Lemos JAJ Am Coll Cardiol.', {'entities': [[39, 41, 'ASSAY']]}], ['Fig: Representative 1H NMR spectrum of a urine sample.', {'entities': [[20, 26, 'ASSAY']]}], ['1H NMR data matrix.', {'entities': [[0, 6, 'ASSAY']]}], ['1H NMR binned spectral data.', {'entities': [[0, 6, 'ASSAY']]}], ['The aim of this work was to investigate urine samples collected from patients suffering from pain of different nature by a metabolomics approach based on 1H NMR spectroscopy and multivariate statistical analysis.', {'entities': [[157, 173, 'ASSAY']]}], ['High Resolution 1H Nuclear Magnetic Resonance (NMR) spectroscopy is very attractive since it is highly reproducible and requires minimal sample preparation.', {'entities': [[47, 50, 'ASSAY']]}], ['Recent evidences, based on metabolomics with different analytical platforms (1H NMR, Mass Spectrometry), on different biological matrices (plasma, urine, cerebrospinal fluid, and tissues), from either humans or animal models [12–18], suggested that this approach could be a promising tool to help in the diagnostic assessment of pain.', {'entities': [[77, 83, 'ASSAY']]}], ['To this goal, we applied a metabolomics approach based on 1H NMR analysis of urine collected from patients affected by these two different types of pain and from a control group.', {'entities': [[58, 64, 'ASSAY']]}], ['Sample collection and preparation for 1H NMR spectroscopyAt', {'entities': [[41, 57, 'ASSAY']]}], ['Samples were immediately frozen at -80°C until NMR analysis.', {'entities': [[47, 50, 'ASSAY']]}], ['For 1H NMR analysis, 1 mL of thawed urine was centrifuged again.', {'entities': [[4, 10, 'ASSAY']]}], ['(TSP, 98 atom % D, Sigma-Aldrich, Milan) at a 0.6 mM final concentration, and 650 μl of the obtained solution were transferred into a 5 mm NMR tube.1H NMR data acquisition and processing1H NMR experiments were carried out on a Varian UNITY INOVA', {'entities': [[139, 142, 'ASSAY'], [148, 154, 'ASSAY'], [186, 192, 'ASSAY'], [227, 233, 'ASSAY']]}], ['500 spectrometer (Agilent Technologies, CA, USA).', {'entities': [[18, 25, 'ASSAY']]}], ['Before Fourier transformation, the free induction decays (FID) were zero-filled to 64k and an exponential weighting function was applied with a line-broadening factor of 0.5 Hz.', {'entities': [[58, 61, 'ASSAY']]}], ['The assignment of major resonances was performed on the basis of data published in the literature [19] and using the database implemented in Chenomx NMR suite 7.1 (Chenomx Inc., Edmonton, Alberta, Canada).', {'entities': [[149, 152, 'ASSAY']]}], ['The 1H NMR spectra were reduced to consecutive integrated spectral regions (bins) of equal width (0.04 ppm) corresponding to the region 0.5–9.5 ppm.', {'entities': [[4, 10, 'ASSAY']]}], ['Statistical analysisMultivariate statistical analysis of 1H NMR spectral data Multivariate statistical analysis of 1H NMR spectral data were performed by the unsupervised Principal components Analysis (PCA) and the supervised Partial Least Square Discriminant Analysis (PLS-DA) and its Orthogonal variant (OPLS-DA), as implemented in SIMCA software package.', {'entities': [[57, 63, 'ASSAY'], [115, 121, 'ASSAY']]}], ['1).cvalues expressed as the median (range).dvalues are expressed as the mean ± standard deviation.1H NMR spectra were recorded for all samples.', {'entities': [[98, 104, 'ASSAY']]}], ['A representative 1H NMR spectrum of a urine sample with major assignments is reported in S1 Fig.', {'entities': [[17, 23, 'ASSAY']]}], ['At first, to ascertain whether subjects with pain syndromes have a different 1H NMR urine metabolite profile with respect to the random control population, we performed a two classes OPLS-DA, where both NP and NC samples were merged in an unique class: “pain”, and compared to C class.', {'entities': [[77, 83, 'ASSAY']]}], ['Open in a separate windowFig 1OPLS-DA score plots in the predictive (x-axis) and orthogonal (y-axis) components of 1H NMR spectral data of urine samples.', {'entities': [[115, 121, 'ASSAY']]}], ['Based on these results, we can state that the 1H NMR metabolite profile can well classify subjects with pain', {'entities': [[46, 52, 'ASSAY']]}], ['Two-classes OPLS-DA of 1H NMR metabolite profiles of subjects diagnosed as suffering of neuropathic pain were compared to the controls, matched for age and gender.', {'entities': [[23, 29, 'ASSAY']]}], ['Analogously, pairwise OPLS-DA of 1H NMR metabolite profiles of subjects diagnosed as suffering of nociceptive pain were compared to the controls matched for age and gender.', {'entities': [[33, 39, 'ASSAY']]}], ['1H NMR metabolite profile of NP and NC samples were compared.', {'entities': [[0, 6, 'ASSAY']]}], ['Analyzing the overall results summarized in Table 2, it is clear that a good level of classification can be obtained by pairwise OPLS-DA of 1H NMR', {'entities': [[140, 146, 'ASSAY']]}], [':DiscussionAim of this study was to investigate whether a 1H NMR metabolomics approach can help in formulate a definite pain diagnosis, as it is well known that obtaining unambiguous results from clinical scores is a major issue for patients suffering from chronic pain.', {'entities': [[58, 64, 'ASSAY']]}], ['Go to:ConclusionsIn this exploratory study, for the first time, we investigated the ability of a 1H NMR metabolomics approach to differentiate between two types of chronic pain.', {'entities': [[97, 103, 'ASSAY']]}], ['As a suggestion for future development, we “throw the gauntlet” of creating a network for the best possible definition of a 1H NMR metabolite profile associated with different kinds of pain.', {'entities': [[124, 130, 'ASSAY']]}], ['NMR spectrometer is relatively easy to find in Health facilities.', {'entities': [[0, 3, 'ASSAY']]}], ['The high reproducibility of NMR data among different laboratories may allow the collection of a huge data bank.', {'entities': [[28, 31, 'ASSAY']]}], ['FigRepresentative 1H NMR spectrum of a urine sample.', {'entities': [[18, 24, 'ASSAY']]}], ['file.(19K, docx)S3 Table1H NMR data matrix.', {'entities': [[24, 30, 'ASSAY']]}], ['1H NMR binned spectral data.', {'entities': [[0, 6, 'ASSAY']]}], ['An NMR-based metabolic profiling study of inflammatory pain using the rat FCA model Metabolomics.', {'entities': [[3, 6, 'ASSAY']]}], ['1H-NMR metabolic profiling of cerebrospinal fluid in patients with complex regional pain syndrome-related dystonia.', {'entities': [[0, 6, 'ASSAY']]}], ['Metabolomic study of biochemical changes in the plasma and urine of primary dysmenorrhea patients using UPLC-MS coupled with a pattern recognition approach.', {'entities': [[104, 111, 'ASSAY']]}], ['NMR brain metabonomics of scrapie exposed sheep.', {'entities': [[0, 3, 'ASSAY']]}], ['Identifying hypoxia in a newborn piglet model using urinary NMR metabolomic profiling.', {'entities': [[60, 63, 'ASSAY']]}], ['Melzack RPain.', {'entities': [[8, 10, 'ASSAY']]}], ['[Ref list]Metabolomic study of biochemical changes in the plasma and urine of primary dysmenorrhea patients using UPLC-MS coupled with a pattern recognition approach.', {'entities': [[114, 121, 'ASSAY']]}], ['[Ref list](1)H NMR brain metabonomics of scrapie exposed sheep.', {'entities': [[15, 18, 'ASSAY']]}], ['[Ref list]Identifying hypoxia in a newborn piglet model using urinary NMR metabolomic profiling.', {'entities': [[70, 73, 'ASSAY']]}], ['PMC4280143PMID: 25549083Metabolomics Data Normalization with EigenMSYuliya V. Karpievitch, 1 ,', {'entities': [[66, 68, 'ASSAY']]}], ['The data necessary to reproduce our results is available on the EigenMS SourceForge page (http://eigenms.sourceforge.net/).Go to:', {'entities': [[69, 71, 'ASSAY']]}], ['AbstractLiquid chromatography mass spectrometry has become one of the analytical platforms of choice for metabolomics studies.', {'entities': [[8, 29, 'ASSAY']]}], ['However, LC-MS metabolomics data can suffer from the effects of various systematic biases.', {'entities': [[9, 14, 'ASSAY']]}], ['These include batch effects, day-to-day variations in instrument performance, signal intensity loss due to time-dependent effects of the LC column performance, accumulation of contaminants in the MS ion source and MS sensitivity among others.', {'entities': [[196, 198, 'ASSAY'], [214, 216, 'ASSAY']]}], ['In this study we aimed to test a singular value decomposition-based method, called EigenMS, for normalization of metabolomics data.', {'entities': [[88, 90, 'ASSAY']]}], ['We analyzed a clinical human dataset where LC-MS serum metabolomics data and physiological measurements were collected from thirty nine healthy subjects and forty with type 2 diabetes and applied EigenMS to detect and correct for any systematic bias.', {'entities': [[43, 48, 'ASSAY'], [201, 203, 'ASSAY']]}], ['EigenMS works in several stages.', {'entities': [[5, 7, 'ASSAY']]}], ['First, EigenMS preserves the treatment group differences in the metabolomics data by estimating treatment effects with an ANOVA model (multiple fixed effects can be estimated).', {'entities': [[12, 14, 'ASSAY']]}], ['EigenMS removed bias of unknown complexity from the LC-MS metabolomics data, allowing for increased sensitivity in differential analysis.', {'entities': [[5, 7, 'ASSAY'], [52, 57, 'ASSAY']]}], ['IntroductionAlong with nuclear magnetic resonance, liquid chromatography coupled to mass spectrometry (LC-MS) has become one of the most common analytical platforms for studying cell, tissue or body fluid metabolomes [1]–[7].', {'entities': [[103, 108, 'ASSAY']]}], ['Yet, as with any high-throughput technology, systematic biases are often observed in LC-MS metabolomics data [8].', {'entities': [[85, 90, 'ASSAY']]}], ['Time-dependent trends in LC-MS metabolomics datasets typically result from analyte retention time drift due to changes in LC column performance or variations in signal intensity caused by fluctuations in MS sensitivity.', {'entities': [[25, 30, 'ASSAY'], [204, 206, 'ASSAY']]}], ['We report here the application of a singular value decomposition-based method, called EigenMS, to remove systematic biases from metabolomics data in the presence of missing observations [9].', {'entities': [[91, 93, 'ASSAY']]}], ['This normalization method, previously shown to be effective in normalizing LC-MS proteomics data [9], improved downstream differential analysis and increased correlation of the metabolite peak intensities with corresponding physiological measurements of what we call clinical biochemistry.', {'entities': [[75, 80, 'ASSAY']]}], ['LC-MS data were acquired in positive ion mode using an Orbitrap XL mass spectrometer (Thermo Scientific) controlled by XCalibur v2 software.', {'entities': [[0, 5, 'ASSAY'], [55, 63, 'ASSAY'], [67, 84, 'ASSAY'], [86, 92, 'ASSAY']]}], ['Chromatographic separation was carried out using C18 reverse-phase HPLC as described in [8].The study was approved by the Human Research Ethics Committee of the University of Tasmania and all procedures conformed to the Declaration of Helsinki.', {'entities': [[53, 71, 'ASSAY']]}], ['We generated a pooled quality control (QC) sample in order to monitor LC and MS performance across sample runs by combining small aliquots (10 uL) of every sample in the study, as recommended by Sangster et al.', {'entities': [[77, 79, 'ASSAY']]}], ['We monitored system performance using blocks of four experimental samples flanked by the QC samples between MS source and inlet cleaning.', {'entities': [[108, 110, 'ASSAY']]}], ['Mass spectral peak deconvolution and retention time correction were carried out in R using XCMS.', {'entities': [[93, 95, 'ASSAY']]}], ['EigenMS uses a combination of ANOVA and singular value decomposition to capture and remove biases from LC-MS metabolomic peak intensity measurements while preserving the variation of interest.', {'entities': [[5, 7, 'ASSAY'], [103, 108, 'ASSAY']]}], ['EigenMS utilizes built-in R function lm() to estimate treatment effects and produce the matrix of residuals which accessed via the residuals() function.', {'entities': [[5, 7, 'ASSAY']]}], ['EigenMS is based on the surrogate variable analysis of Leek and Storey [12], with modifications including analysis of data with missing values that are typical in LC-MS experiments [9].', {'entities': [[5, 7, 'ASSAY'], [163, 168, 'ASSAY']]}], ['EigenMS is available as a stand-alone set of two functions implemented in R from SourceForge:', {'entities': [[5, 7, 'ASSAY']]}], ['Results and DiscussionA characteristic of biofluid metabolite analysis via LC-MS is progressive signal intensity loss due to the accumulation of contaminants within the MS ion source, sample transfer lines and the heated ion transfer capillary [13].', {'entities': [[75, 80, 'ASSAY'], [169, 171, 'ASSAY']]}], ['Further, if one wishes to identify relationships between LC-MS metabolomics data and other variables, normalization becomes essential.', {'entities': [[57, 62, 'ASSAY']]}], ['Our experience leads us to recommend the use of QC samples to monitor performance of the LC-MS instrument.', {'entities': [[89, 94, 'ASSAY']]}], ['We normalized the data with EigenMS.', {'entities': [[33, 35, 'ASSAY']]}], ['EigenMS identified 12 systematic bias trends and eliminated their effects from the data.', {'entities': [[5, 7, 'ASSAY']]}], ['We correlated these 1100 MS peaks using Spearman correlation in ‘Hmisc’ package to the following clinical measurements we obtained for each subject: blood glucose level, glycated haemoglobin, exercise central augmentation pressure normalized to heart rate of 75, and total cholesterol [15].', {'entities': [[25, 27, 'ASSAY']]}], ['If EigenMS works well we expect to see increased correlation between the changes observed in normalized metabolites and physiological variables.', {'entities': [[8, 10, 'ASSAY']]}], [':ConclusionsNormalization is an important step in MS data analysis, but it is complicated by the high complexity of biases.', {'entities': [[50, 52, 'ASSAY']]}], ['EigenMS has been shown to remove biases of arbitrary complexity from proteomics data [9].', {'entities': [[5, 7, 'ASSAY']]}], ['The ability of EigenMS to capture complex biases and eliminate them preserves the validity of any downstream statistical analysis.', {'entities': [[20, 22, 'ASSAY']]}], ['The data necessary to reproduce our results is available on the EigenMS SourceForge page (http://eigenms.sourceforge.net/).Go to:References1.', {'entities': [[69, 71, 'ASSAY']]}], ['Development of a robust and repeatable UPLC-MS method for the long-term metabolomic study of human serum.', {'entities': [[39, 46, 'ASSAY']]}], ['Liquid chromatography and ultra-performance liquid chromatography-mass spectrometry fingerprinting of human urine: sample stability under different handling and storage conditions for metabonomics studies.', {'entities': [[0, 21, 'ASSAY']]}], ['profiling analysis on human urine by UPLC-TOFMS: issues and method validation in nutritional metabolomics.', {'entities': [[42, 45, 'ASSAY'], [45, 47, 'ASSAY']]}], ['Global metabolic profiling procedures for urine using UPLC–MS.', {'entities': [[59, 61, 'ASSAY']]}], ['(2009) Normalization of Peak Intensities in Bottom-Up MS-Based Proteomics Using Singular Value Decomposition.', {'entities': [[54, 56, 'ASSAY']]}], ['A pragmatic and readily implemented quality control strategy for HPLC-MS and GC-MS-based metabolomic analysis.', {'entities': [[65, 72, 'ASSAY'], [80, 82, 'ASSAY']]}], ['Methodological considerations in the development of HPLC-MS methods for the analysis of rodent plasma for metabonomic studies.', {'entities': [[52, 59, 'ASSAY']]}], ['Development of a robust and repeatable UPLC-MS method for the long-term metabolomic study of human serum.', {'entities': [[39, 46, 'ASSAY']]}], ['[Ref list]Global metabolic profiling procedures for urine using UPLC-MS.Want EJ, Wilson ID, Gika H, Theodoridis G, Plumb RS, Shockcor J, Holmes E, Nicholson JKNat Protoc.', {'entities': [[64, 71, 'ASSAY']]}], ['[Ref list]Normalization of peak intensities in bottom-up MS-based proteomics using singular value decomposition.', {'entities': [[57, 59, 'ASSAY']]}], ['[Ref list]A pragmatic and readily implemented quality control strategy for HPLC-MS and GC-MS-based metabonomic analysis.', {'entities': [[75, 82, 'ASSAY'], [90, 92, 'ASSAY']]}], ['[Ref list]Methodological considerations in the development of HPLC-MS methods for the analysis of rodent plasma for metabonomic studies.', {'entities': [[62, 69, 'ASSAY']]}], ['PMC5538707PMID: 28763497Diagnosis of coinfection by schistosomiasis and viral hepatitis B or C using 1H NMR-based metabonomicsLiana Ribeiro Gouveia, Data curation, Formal analysis, Methodology, Software, Validation, Writing – original draft,#1,', {'entities': [[101, 107, 'ASSAY']]}], ['Nuclear Magnetic Resonance (NMR)-based', {'entities': [[28, 31, 'ASSAY']]}], ['The aim of this study was to diagnose periportal fibrosis due to schistosomiasis mansoni in patients with chronic HBV or HCV infection through NMR-based metabonomics models.', {'entities': [[143, 146, 'ASSAY']]}], ['The serum samples were analyzed through H-1 NMR spectroscopy and the models were based on Principal Component Analysis (PCA) and Partial Least Squares—Discriminant Analysis (PLS-DA).', {'entities': [[44, 60, 'ASSAY']]}], ['As an alternative, the Nuclear Magnetic Resonance (NMR)-based metabonomics strategy has proved its value in disease diagnosis.', {'entities': [[51, 54, 'ASSAY']]}], ['This strategy is often used when there is interest in classifying samples by their biochemical status in response to diseases [11,12], since NMR spectroscopy is a powerful tool for identifying changes in metabolite profiling as a result of changes in the biochemical status.', {'entities': [[141, 157, 'ASSAY']]}], ['A disadvantage is the fact that an NMR spectrometer is expensive.', {'entities': [[35, 38, 'ASSAY']]}], ['However, comparing to US, NMR is not a subjective technique.', {'entities': [[26, 29, 'ASSAY']]}], ['Moreover, NMR offers a non-invasive analysis and with only one serum sample it would be possible to find out with high accuracy if the patient has coinfection or not.', {'entities': [[10, 13, 'ASSAY']]}], ['The use of specific statistical tools for an efficient interpretation of NMR spectra is necessary, due to complexity of the process [13].Many encouraging results have been obtained using NMR-based metabonomics to investigate diseases, including studies on hepatitis C [14], urological cancer [15] and schistosomiasis [', {'entities': [[73, 76, 'ASSAY'], [187, 190, 'ASSAY']]}], ['16,17].Therefore, the aim of this study was to diagnose periportal fibrosis due to schistosomiasis mansoni in patients with chronic HBV or HCV infection through 1H NMR-based metabonomics models.', {'entities': [[161, 167, 'ASSAY']]}], ['Part of the sera was stored at minus 20°C for a short period of time until the NMR analysis could be performed.', {'entities': [[79, 82, 'ASSAY']]}], ['NMR analysisSerum samples were prepared by mixing 400 μl of serum and 200 μl of D2O, using the signal attribute to lactate (δ 1.33 ppm) as a chemical shift reference.', {'entities': [[0, 3, 'ASSAY']]}], ['All 1H NMR spectra were recorded in a Varian Unity Plus spectrometer operating at 300 MHz, using NMR tubes of 5 mm.', {'entities': [[4, 10, 'ASSAY'], [38, 44, 'ASSAY'], [97, 100, 'ASSAY']]}], ['After homogenization, the 1H NMR spectrum was acquired using a sequence of pulses with presaturation (PRESAT) of the water signal hyphenated to the Carr-Purcell-Meiboom-Gill (CPMG) pulse sequence, which was employed as a T2 filter.', {'entities': [[26, 32, 'ASSAY'], [148, 173, 'ASSAY'], [175, 179, 'ASSAY']]}], ['One matrix was built with 40 rows (cases) and 105 variables (bins of the 1H NMR spectrum plus class variable), and then was submitted to multivariate analysis.', {'entities': [[73, 79, 'ASSAY']]}], ['Fig 4 shows 1H PRESAT-CPMG NMR spectra of serum samples, with indication of spectral regions used in the metabonomics modeling.', {'entities': [[22, 26, 'ASSAY'], [27, 30, 'ASSAY']]}], ['Open in a separate windowFig 41H NMR spectra (PRESAT-CPMG, 300 MHz) of serum, with attribution of signals responsible for discrimination.', {'entities': [[30, 36, 'ASSAY'], [53, 57, 'ASSAY']]}], ['In fact, Wang et al., using a S. mansoni–mouse model based on 1H NMR spectroscopy, reported that the lactate concentration varied between animals but was not found to be significant in discriminating between infected and control mice.', {'entities': [[65, 81, 'ASSAY']]}], ['Furthermore, 1H NMR-based metabonomics has been used successfully to discriminated patients with HCV infection with high sensitivity and specificity [14]; to distinguish S. mansoni-infected from uninfected individuals before and after chemotherapy with praziquantel [17]; and to characterize metabolic profiles of liver cirrhosis and hepatocellular carcinoma [33].', {'entities': [[13, 19, 'ASSAY']]}], ['These results suggest that NMR spectra combined with pattern recognition analysis techniques offer a great benefit to early diagnosis of human diseases.', {'entities': [[27, 30, 'ASSAY']]}], ['The proposed approach was based on exploratory PCA and subsequent PLS-DA of pre-processed binned 1H NMR spectra of serum samples.', {'entities': [[97, 103, 'ASSAY']]}], ['Exploratory PCA for 1H NMR data showed a clear separation between coinfected and monoinfected samples.', {'entities': [[20, 26, 'ASSAY']]}], ['Meyer D. Applying biofluid 1H NMR-based metabonomic techniques to distinguish between HIV-1', {'entities': [[27, 33, 'ASSAY']]}], ['Araújo L, Pinto F, Costa T, Silva R, Lima S, Silva R. Diagnosis of Urological Cancer by 1H NMR', {'entities': [[88, 94, 'ASSAY']]}], ['Proposed minimum reporting standards for chemical analysis Chemical Analysis Working Group (CAWG) Metabolomics Standards Inititative (MSI).', {'entities': [[134, 136, 'ASSAY']]}], ['Article NMR based serum metabolomics discriminates Takayasu Arteritis from Healthy Individuals :', {'entities': [[8, 11, 'ASSAY']]}], ['A proof of principle study NMR based serum metabolomics discriminates Takayasu Arteritis from Healthy Individuals :', {'entities': [[27, 30, 'ASSAY']]}], ['1HNMR-based metabolomic profile of rats with experimental acute pancreatitis.', {'entities': [[2, 5, 'ASSAY']]}], ['Prediction of gestational diabetes through NMR metabolomics of maternal blood.', {'entities': [[43, 46, 'ASSAY']]}], ['Metabolic characterization of advanced liver fibrosis in HCV patients as studied by serum 1H-NMR spectroscopy.', {'entities': [[93, 109, 'ASSAY']]}], ['Application of 1H NMR-based metabonomics in the study of metabolic profiling of human hepatocellular carcinoma and liver cirrhosis.', {'entities': [[15, 21, 'ASSAY']]}], ['biofluid 1H NMR-based metabonomic techniques to distinguish between HIV-1', {'entities': [[9, 15, 'ASSAY']]}], ['Araújo L, Pinto F, Costa T, Silva R, Lima S, Silva R. Diagnosis of Urological Cancer by 1H NMR', {'entities': [[88, 94, 'ASSAY']]}], ['(CAWG) Metabolomics Standards Initiative (MSI).Sumner LW, Amberg A, Barrett D, Beale MH, Beger R, Daykin CA, Fan TW, Fiehn O, Goodacre R, Griffin JL, Hankemeier T, Hardy N, Harnly J, Higashi R, Kopka J, Lane AN, Lindon JC, Marriott P, Nicholls AW, Reily MD, Thaden JJ, Viant MRMetabolomics.', {'entities': [[42, 44, 'ASSAY']]}], ['[Ref list]1HNMR-based metabolomic profile of rats with experimental acute pancreatitis.', {'entities': [[12, 15, 'ASSAY']]}], ['[Ref list]Prediction of Gestational Diabetes through NMR Metabolomics of Maternal Blood.', {'entities': [[53, 56, 'ASSAY']]}], ['[Ref list]Metabolic Characterization of Advanced Liver Fibrosis in HCV Patients as Studied by Serum 1H-NMR Spectroscopy.', {'entities': [[100, 106, 'ASSAY']]}], ['[Ref list]Application of 1H NMR-based metabonomics in the study of metabolic profiling of human hepatocellular carcinoma and liver cirrhosis.', {'entities': [[25, 31, 'ASSAY']]}], ['Ultra-performance liquid chromatography-triple-quadrupole-mass spectrometry (QqQ/MS) was used to make a full assessment of the metabolites.', {'entities': [[77, 80, 'ASSAY'], [81, 83, 'ASSAY']]}], ['The analyses were conducted on an Agilent 1290 UHPLC chromatograph coupled to a triple quadrupole (QqQ)', {'entities': [[34, 41, 'ASSAY'], [99, 102, 'ASSAY']]}], ['6495 Series with an ESI interface (Agilent Technologies, Waldbronn, Germany).', {'entities': [[20, 23, 'ASSAY'], [35, 42, 'ASSAY']]}], ['2.1 mm x 100mm, 1.8 μm, Waters, Milford MA, USA).', {'entities': [[24, 30, 'ASSAY']]}], ['Mass spectrometric detection was performed by multiple reaction monitoring (MRM) in positive ESI mode and several transitions for each compound were acquired.', {'entities': [[93, 96, 'ASSAY']]}], ['The ESI parameters were as follows:', {'entities': [[4, 7, 'ASSAY']]}], ['The analyses were conducted on an Agilent 1290 UHPLC chromatograph coupled to a triple quadrupole (QqQ)', {'entities': [[34, 41, 'ASSAY'], [99, 102, 'ASSAY']]}], ['6495 Series instrument, with an ESI interface (Agilent Technologies, Waldbronn, Germany).', {'entities': [[32, 35, 'ASSAY'], [47, 54, 'ASSAY']]}], ['Mass spectrometric detection was performed by multiple reactions monitoring (MRM) in negative ESI mode and three transitions for each compound were acquired at their specific retention time.', {'entities': [[94, 97, 'ASSAY']]}], ['The ESI parameters were as follows:', {'entities': [[4, 7, 'ASSAY']]}], ['Sabatine MS, Liu E, Morrow DA, Heller E, McCarroll R, Wiegand R, et al.', {'entities': [[9, 11, 'ASSAY']]}], ['Basak T, Varshney S, Hamid Z, Ghosh S, Seth S, Sengupta S. Identification of metabolic markers in coronary artery disease using an untargeted LC-MS based metabolomic approach.', {'entities': [[142, 147, 'ASSAY']]}], [\"Development and validation of an LC-MS/MS method for quantification of cyclic guanosine 3’,5'-monophosphate\", {'entities': [[33, 41, 'ASSAY']]}], ['Yi X-Y, Liao D-X, Wang C, Cheng W, Fu X-Q, Zhang B. Cytochrome P450 Genetic Variants and Their Metabolite Levels', {'entities': [[76, 82, 'ASSAY']]}], ['Sabatine MS, Liu E, Morrow DA, Heller E, McCarroll R, Wiegand R, Berriz GF, Roth FP, Gerszten RECirculation.', {'entities': [[9, 11, 'ASSAY']]}], ['[Ref list]Identification of metabolic markers in coronary artery disease using an untargeted LC-MS based metabolomic approach.', {'entities': [[93, 98, 'ASSAY']]}], [\"[Ref list]Development and validation of an LC-MS/MS method for quantification of cyclic guanosine 3',5'-monophosphate (cGMP) in clinical applications: a comparison with a EIA method.\", {'entities': [[43, 51, 'ASSAY']]}], ['P450 Genetic Variants and Their Metabolite Levels Associated with Plaque Stability in Patients with Ischemic Stroke.', {'entities': [[13, 19, 'ASSAY']]}], ['26565633Prognosis Biomarkers of Severe Sepsis and Septic Shock by 1H NMR Urine Metabolomics in the Intensive Care UnitMonica Garcia-Simon,# 1 Jose M. Morales,# 2 Vicente Modesto-Alapont, 3 Vannina Gonzalez-Marrachelli, 4 Rosa Vento-Rehues, 1 Angela Jorda-Miñana, 1 Jose Blanquer-Olivas, 1 and Daniel Monleon 4 ,', {'entities': [[66, 72, 'ASSAY']]}], ['Fig: Average 1H NMR spectrum of urine-0h samples obtained at the admission to the ICU from septic patients.', {'entities': [[13, 19, 'ASSAY']]}], ['We aimed to identify metabolomic biomarkers of sepsis in urine by 1H-NMR spectroscopy to assess the severity and to predict outcomes.', {'entities': [[69, 85, 'ASSAY']]}], ['Urine samples were collected from 64 patients with severe sepsis or septic shock in the ICU for a 1H NMR spectra acquisition.', {'entities': [[98, 104, 'ASSAY']]}], ['Our results indicate that NMR metabolic profiling might be helpful for determining the metabolomic phenotype of worst-prognosis septic patients in an early stage.', {'entities': [[26, 29, 'ASSAY']]}], ['The application of Nuclear Magnetic Resonance (NMR) metabolomics in critical septic patients would result in the simultaneous identification of a vast array of potential new biomarkers.', {'entities': [[47, 50, 'ASSAY']]}], ['[8–11].The aim of the present study was to identify metabolomic biomarkers of sepsis in urine by 1H NMR spectroscopy to assess the severity and to predict outcomes.', {'entities': [[100, 116, 'ASSAY']]}], ['Immediately after collection, samples were frozen at—80°C until the NMR measurements.1H NMR Spectroscopy and metabolite concentration profilingForty microliters of a phosphate buffer solution (0.06M Na2HPO4/ 0.04M NaH2PO4, pH 7) and 40μl of sodium-3´-trimethylsilylpropionate-2,2,3,3-d4', {'entities': [[68, 71, 'ASSAY'], [85, 91, 'ASSAY']]}], ['in deuterium oxide were mixed with 420μl of urine and placed in a 5mm high-resolution NMR tube.', {'entities': [[86, 89, 'ASSAY']]}], ['NMR spectra were acquired using a standard one-dimensional pulse sequence with water suppression in a Bruker Avance DRX 600MHz spectrometer (Bruker Biospin GmbH, Rheinstetten, Germany).', {'entities': [[0, 3, 'ASSAY'], [102, 108, 'ASSAY'], [141, 147, 'ASSAY']]}], ['A total of 64 FIDs (free induction decay) were collected into 64k data points with a spectral width of 14 ppm at 310K.Water pre-saturation for 1 s along the recycling delay was used for solvent signal suppression.', {'entities': [[14, 17, 'ASSAY']]}], ['Principal component analysis (PCA) and projection to latent structures for discriminant analysis (PLS-DA) were applied to NMR spectral datasets.', {'entities': [[122, 125, 'ASSAY']]}], ['The results of cross validation were evaluated by the Q2 (R2CV) and RMSCV parameters.', {'entities': [[69, 71, 'ASSAY']]}], ['Root Mean Square Error of Cross-Validation (RMSCV) was calculated as an adequate measurement of over fitting.', {'entities': [[45, 47, 'ASSAY']]}], ['Additional detail about NMR data processing and the construction and validation of multivariable metabolomic models is provided in the Supporting Information section.', {'entities': [[24, 27, 'ASSAY']]}], ['Typical examples of 1H NMR urinary spectra from survivor and non-survivor patients are shown in S1 Fig.', {'entities': [[20, 26, 'ASSAY']]}], ['To our knowledge, this is the first study in adults that describe a metabolic model prediction of mortality in septic patients using 1H NMR spectroscopy in urine samples.', {'entities': [[136, 152, 'ASSAY']]}], ['However, NMR measurements only require 0.5 ml of urine.', {'entities': [[9, 12, 'ASSAY']]}], ['Go to:ConclusionIn conclusion, urine NMR metabolomics analysis can be a useful tool to describe and interpret the pathophysiological processes that occur in sepsis.', {'entities': [[37, 40, 'ASSAY']]}], ['For these reasons, guided by judicious clinical assessment, NMR metabonomics could be a complementary tool to classical methods used until now to stratify better the prognosis in septic shock and severe sepsis.', {'entities': [[60, 63, 'ASSAY']]}], ['Promising results and its numerous advantages (non-invasive, cheap, fast and low-cost measure), suggest that NMR metabonomics could be included in the clinical routine in the near future.', {'entities': [[109, 112, 'ASSAY']]}], ['1H NMR spectrum of urine-0h samples obtained at the admission to the ICU from septic patients.', {'entities': [[0, 6, 'ASSAY']]}], ['Is 1H NMR metabolomics becoming the promising early biomarker for neonatal sepsis and for monitoring the antibiotic toxicity?.', {'entities': [[3, 9, 'ASSAY']]}], ['Dellinger RP, Levy MM, Rhodes A, Annane D, Gerlach H, Opal SM, et al.', {'entities': [[10, 12, 'ASSAY']]}], ['ESICM/ACCP/ATS/SIS International Sepsis Definitions Conference.', {'entities': [[0, 3, 'ASSAY']]}], ['750 MHz 1H NMR spectroscopy characterisation of the complex metabolic pattern of urine from patients with inborn errors of metabolism:', {'entities': [[11, 27, 'ASSAY']]}], ['A metabolomicapproach to early prognostic evaluation of experimental sepsis by 1H NMR and pattern recognition.', {'entities': [[79, 85, 'ASSAY']]}], ['NMR Biomed 2009; 22', {'entities': [[0, 3, 'ASSAY']]}], ['[Ref list]Surviving Sepsis Campaign: international guidelines for management of severe sepsis and septic shock, 2012.Dellinger RP, Levy MM, Rhodes A, Annane D, Gerlach H, Opal SM, Sevransky JE, Sprung CL, Douglas IS, Jaeschke R, Osborn TM, Nunnally ME, Townsend SR, Reinhart K, Kleinpell RM, Angus DC, Deutschman CS, Machado FR, Rubenfeld GD, Webb S, Beale RJ, Vincent JL, Moreno R, Surviving Sepsis Campaign Guidelines Committee including The Pediatric Subgroup.', {'entities': [[127, 129, 'ASSAY']]}], ['ESICM/ACCP/ATS/SIS International Sepsis Definitions Conference.', {'entities': [[0, 3, 'ASSAY']]}], ['ESICM/ACCP/ATS/', {'entities': [[0, 3, 'ASSAY']]}], ['[Ref list]750 MHz 1H NMR spectroscopy characterisation of the complex metabolic pattern of urine from patients with inborn errors of metabolism:', {'entities': [[21, 37, 'ASSAY']]}], ['[Ref list]A metabonomic approach to early prognostic evaluation of experimental sepsis by (1)H NMR and pattern recognition.', {'entities': [[95, 98, 'ASSAY']]}], ['Lin ZY, Xu PB, Yan SK, Meng HB, Yang GJ, Dai WX, Liu XR, Li JB, Deng XM, Zhang WDNMR', {'entities': [[81, 84, 'ASSAY']]}], ['Typical LC-MS/MS trace of PGE-M (A) and 8-iso-', {'entities': [[8, 16, 'ASSAY']]}], ['Urine samples were collected at each visit and analyzed by liquid chromatography-tandem mass spectrometry for PGE-M, 8-iso-PGF2α, and cotinine.', {'entities': [[81, 105, 'ASSAY']]}], ['Therefore, in the study reported here, cigarette smokers used varenicline (Chantix) as a smoking cessation aid, and visited our clinic regularly over a 12-week period to provide urine samples which were analyzed for PGE-M, 8-iso-PGF2α and cotinine by validated liquid chromatography-tandem mass spectrometry methods.', {'entities': [[283, 307, 'ASSAY']]}], ['Bond Elut C18 96 square-well plates (100 mg) from Agilent (Cat.', {'entities': [[50, 57, 'ASSAY']]}], ['The ethyl acetate was evaporated to dryness using a Speedvac concentrator (Thermo Fisher), and the plates were capped and stored in a -20°C freezer until LC-MS analysis.', {'entities': [[75, 88, 'ASSAY'], [154, 159, 'ASSAY']]}], ['Ten μL of the reconstituted sample was injected onto the LC-MS/MS system for SRM analysis on a TSQ Quantum Discovery Max instrument (', {'entities': [[57, 65, 'ASSAY']]}], ['Thermo Fisher Scientific, Waltham, MA) operated in the negative ion APCI mode.', {'entities': [[0, 13, 'ASSAY']]}], ['Chromatography was performed on an Agilent 1100 HPLC system (Agilent Technologies, Inc., Santa Clara, CA) with a 50 x 3.0 mm,', {'entities': [[35, 42, 'ASSAY'], [61, 68, 'ASSAY']]}], ['Other MS/MS parameters were as follows: collision energy for transitions; 15 V for m/z 315', {'entities': [[6, 11, 'ASSAY']]}], ['A typical LC-MS/MS trace of PGE-M is provided in S1 Fig.', {'entities': [[10, 18, 'ASSAY']]}], ['Bond Elut C18 96 square-well plates (100 mg) from Agilent (Cat.', {'entities': [[50, 57, 'ASSAY']]}], ['# A396011C) were used for solid phase extraction (SPE).', {'entities': [[50, 53, 'ASSAY']]}], ['The ethyl acetate was evaporated to dryness using a Speedvac concentrator (Thermo Fisher), and the plates were capped and stored at -20°C until LC ESI—MS/MS analysis.', {'entities': [[75, 88, 'ASSAY'], [147, 150, 'ASSAY'], [151, 156, 'ASSAY']]}], ['0.15% NH4OH.Sample analysis was performed using LC ESI—MS/MS on a Vantage triple quadrupole mass spectrometer (Thermo Scientific, Pittsburgh PA) using negative electrospray mode,with a Dionex Ultimate 3000 Rapid separation (RSLC) HPLC system (Thermo Scientific, Pittsburgh PA).', {'entities': [[51, 54, 'ASSAY'], [55, 60, 'ASSAY'], [92, 109, 'ASSAY'], [111, 117, 'ASSAY'], [243, 249, 'ASSAY']]}], ['An XBridge BEH C18, 2.5 μm, 50 mm x 1.0 mm column (Waters Corp Cat.', {'entities': [[51, 57, 'ASSAY']]}], ['Peak areas were quantified using Thermo Xcaliber Version 2.2 Quan browser.', {'entities': [[33, 39, 'ASSAY']]}], ['A typical LC-MS/MS trace of 8-iso-PGF2α is provided in S1 Fig.', {'entities': [[10, 18, 'ASSAY']]}], ['Analysis of cotinineThis was performed by LC-MS/MS as described previously.[21]', {'entities': [[42, 50, 'ASSAY']]}], ['carried out a study of 23 smokers who received nicotine replacement therapy and abstained from smoking for 24h with placebo or oral administration of a gamma-tocopherol rich mixture.[19] 8-iso-PGF2α was determined by LC-MS/MS.', {'entities': [[217, 225, 'ASSAY']]}], ['evaluated the effects of smoking cessation on 8-iso-PGF2α, determined by LC-MS/MS, in the longitudinal Wisconsin Smokers Health Study-2, which was designed to examine the natural history of smoking and smoking cessation.[15]', {'entities': [[73, 81, 'ASSAY']]}], ['They observed a significant decrease of 21% for 8-iso-PGF2α, but not in PGE-M.Thus, with the exception of the studies by Pilz et al., which did not use LC-MS/MS analysis, and that of Mah et al.', {'entities': [[152, 160, 'ASSAY']]}], ['file.(14K, docx)S1 FigTypical LC-MS/MS trace of PGE-M (A) and', {'entities': [[30, 38, 'ASSAY']]}], ['Mass spectrometry was carried out in the Analytical Biochemistry Core Facility of the Masonic Cancer Center, supported in part by National Cancer Institute Cancer Center Support Grant (CA-077598).', {'entities': [[0, 17, 'ASSAY']]}], ['LC-MS/MS.', {'entities': [[0, 8, 'ASSAY']]}], ['LC-MS/MS Reveals Alterations in the Metabolic Profile after Smoking Cessation.', {'entities': [[0, 8, 'ASSAY']]}], ['Liquid chromatography-tandem mass spectrometric quantification of the dehydration product of tetranor PGE-M, the major urinary metabolite of prostaglandin E(2) in human urine.', {'entities': [[0, 21, 'ASSAY']]}], ['Goettel M, Niessner R, Mueller D, Scherer M, Scherer G, Pluym N. Metabolomic Fingerprinting in Various Body Fluids of a Diet-Controlled Clinical Smoking Cessation Study Using a Validated GC-TOF-MS Metabolomics Platform.', {'entities': [[190, 193, 'ASSAY'], [194, 196, 'ASSAY']]}], ['[Ref list]Liquid chromatography-tandem mass spectrometric quantification of the dehydration product of tetranor PGE-M, the major urinary metabolite of prostaglandin E(2) in human urine.', {'entities': [[10, 31, 'ASSAY']]}], ['[Ref list]Quantitation of isoprostane isomers in human urine from smokers and nonsmokers by LC-MS/MS.Yan W, Byrd GD, Ogden MWJ Lipid Res.', {'entities': [[92, 100, 'ASSAY']]}], ['LC-MS/MS Reveals Alterations in the Metabolic Profile after Smoking Cessation.', {'entities': [[0, 8, 'ASSAY']]}], ['[Ref list]Metabolomic Fingerprinting in Various Body Fluids of a Diet-Controlled Clinical Smoking Cessation Study Using a Validated GC-TOF-MS Metabolomics Platform.', {'entities': [[135, 138, 'ASSAY'], [139, 141, 'ASSAY']]}], ['Liquid Chromatography/Mass Spectrometry (LC/MS) based metabolomics are applied for the mechanism of disease and further supporting the research of diagnosis and management in recent years.', {'entities': [[41, 46, 'ASSAY']]}], ['MethodsSerum samples of 25 recurrent abortion women with APS and 25 healthy women were collected and analyzed by LC/MS in this study.', {'entities': [[113, 118, 'ASSAY']]}], ['Nuclear magnetic resonance (NMR) and mass spectrometry are two analytical tools that have been well applied in metabolomics-based studies [6, 7].', {'entities': [[28, 31, 'ASSAY']]}], ['Wide dynamic range and chemical diversity coverage are the main advantages of LC/MS.', {'entities': [[78, 83, 'ASSAY']]}], ['In this study, LC/MS based metabolomics method was applied to investigate the metabolic profile change between recurrent abortion women with APS and healthy women.', {'entities': [[15, 20, 'ASSAY']]}], ['The filtered samples were finally analyzed by LC/MS.Metabolomics analysisAgilent 1200 rapid resolution liquid chromatography (RRLC) coupled with Agilent 6520 Q-TOF mass spectrometer (Agilent Technologies, USA) was used for the LC/MS analysis.', {'entities': [[46, 51, 'ASSAY'], [73, 80, 'ASSAY'], [145, 152, 'ASSAY'], [160, 163, 'ASSAY'], [164, 181, 'ASSAY'], [183, 190, 'ASSAY'], [227, 232, 'ASSAY']]}], ['Sample volume was 10μL. Column: Agilent Eclipse Plus C18, 3.5μm, 2.1mm×150mm.', {'entities': [[32, 39, 'ASSAY']]}], ['Mass spectrometry condition:', {'entities': [[0, 17, 'ASSAY']]}], ['Electrospray ionization (ESI) source; Positive mode:', {'entities': [[0, 23, 'ASSAY'], [25, 28, 'ASSAY']]}], ['gas temperature, 350°C; drying gas, 9L/min; nebulizer, 40psig; fragmentor, 150V; skimmer, 60V; capillary, 3500V. Centroid mode for data collection, from m/z 50 to 1000.Pattern recognition and pathway analysisData normalization was performed and then imported the data files into the statistic software (Mass Profiler Professional, Agilent Technologies in USA).', {'entities': [[331, 338, 'ASSAY']]}], ['GBU)50.8–26.21114.562.148.330.12132.354.381.249.83144.560.256.428.84130.830.176.235.95142.368.366.126.26146.245.348.248.57112.852.446.350.88118.939.182.133.19136.447.886.529.710140.755.358.350.211119.449.869.228.912128.343.277.344.313125.650.772.539.414131.933.882.336.515121.444.555.442.116113.250.260.137.317142.154.573.544.418133.447.964.735.719127.350.271.238.920134.339.178.240.421129.357.149.843.322122.748.553.248.623127.351.158.731.324128.644.466.934.625133.350.177.433.7Open in a separate windowMetabolic profiling analysisUsing Agilent RRLC/', {'entities': [[538, 545, 'ASSAY']]}], ['Q-TOF/MS', {'entities': [[2, 5, 'ASSAY'], [6, 8, 'ASSAY']]}], ['The accurate molecular ion masses were used for the potential composition calculation and the MS/MS data were collected for structure deduction.', {'entities': [[94, 99, 'ASSAY']]}], ['At last, the validation of the identification results were performed by comparing the MS and MS/MS data of the available standards.', {'entities': [[86, 88, 'ASSAY'], [93, 98, 'ASSAY']]}], ['RRLC-QTOF-MS were used for comprehensive metabolomics analysis and provide the overall metabolites change.', {'entities': [[5, 9, 'ASSAY'], [10, 12, 'ASSAY']]}], ['In present study, the LC-MS based metabolomics method was applied to analyze the metabolic profile changes between recurrent abortion women with APS and control subjects.', {'entities': [[22, 27, 'ASSAY']]}], ['It is proved that the LC/MS metabolomics method is useful for APS mechanism study.', {'entities': [[22, 27, 'ASSAY']]}], ['Wyss M, Kaddurah-Daouk RPhysiol Rev. 2000 Jul; 80(3):1107-213.[PubMed]', {'entities': [[23, 25, 'ASSAY']]}], ['PMC3024392PMID: 21283746Serum Lipidomics Meets Cardiac Magnetic Resonance Imaging:', {'entities': [[30, 40, 'ASSAY']]}], ['Analyzed the data: MSA JK JL MO.', {'entities': [[19, 21, 'ASSAY']]}], ['MSA JK TH MO.Received 2010 Sep 23;', {'entities': [[0, 2, 'ASSAY']]}], ['in a separate windowWe applied Ultra Performance Liquid Chromatography coupled to mass spectrometry (UPLC/MS™) to study molecular lipids as previously described [10].', {'entities': [[103, 108, 'ASSAY']]}], ['The heart surface models were also used to derive 86 MRI parameters that describe anatomical and physiological features of the heart, as described earlier [9].LipidomicsThe lipidome was analyzed as described previously [19].', {'entities': [[159, 169, 'ASSAY']]}], ['Another standard mixture containing three labeled lipid species was added to the extracts and the lipids were analyzed on a Waters Q-Tof Premier mass spectrometer combined with an Acquity Ultra Performance LC™ (UPLC).', {'entities': [[124, 130, 'ASSAY'], [145, 162, 'ASSAY'], [188, 208, 'ASSAY']]}], ['Katajamaa M, Oresic M. Processing methods for differential analysis of LC/MS profile data.', {'entities': [[71, 76, 'ASSAY']]}], ['Kärkkäinen S, Reissell E, Heliö T, Kaartinen M, Tuomainen P, Toivonen L, Kuusisto J, Kupari M, Nieminen MS, Laakso M, Peuhkurinen KHeart.', {'entities': [[104, 106, 'ASSAY']]}], ['Koikkalainen JR, Antila M, Lötjönen JM, Heliö T, Lauerma K, Kivistö SM, Sipola P, Kaartinen MA, Kärkkäinen ST, Reissell E, Kuusisto J, Laakso M, Oresic M, Nieminen MS, Peuhkurinen KJRadiology.', {'entities': [[164, 166, 'ASSAY']]}], ['Kärkkäinen S, Heliö T, Miettinen R, Tuomainen P, Peltola P, Rummukainen J, Ylitalo K, Kaartinen M, Kuusisto J, Toivonen L, Nieminen MS, Laakso M, Peuhkurinen KEur Heart J. 2004 May; 25(10):885-93.[PubMed]', {'entities': [[132, 134, 'ASSAY']]}], ['[Ref list]Processing methods for differential analysis of LC/MS profile data.', {'entities': [[58, 63, 'ASSAY']]}], ['E-mail: ed.etirahc@bawhcs.naj (JMS); ed.etirahc@sseurp.dlarah (', {'entities': [[32, 34, 'ASSAY']]}], ['(MS).DesignMatched-pairs study in University hospital Neurology department.', {'entities': [[1, 3, 'ASSAY']]}], ['PatientsBased on clinical (relapse frequency) and paraclinical (MRI lesions, contrast enhancement) criteria, 10 pairs of age- and sex-matched patients with relapsing-remitting MS were assigned either to a group with highly active or less active MS.', {'entities': [[176, 178, 'ASSAY'], [245, 247, 'ASSAY']]}], ['Lipid mediators were quantified in serum and cerebrospinal fluid using LC-MS/MS-based lipidomics.', {'entities': [[71, 79, 'ASSAY']]}], ['In the patient group with highly active MS, 15-HETE and PGE2 were increased, which are products of the 15-lipoxygenase and cyclooxygenase pathways.', {'entities': [[40, 42, 'ASSAY']]}], ['LXA4 levels were not increased in patients with highly active MS.ConclusionsLipid mediator pathways are regulated differentially in the cerebrospinal fluid of MS patients, depending on disease severity.', {'entities': [[62, 64, 'ASSAY'], [159, 161, 'ASSAY']]}], ['Non-exhaustive or possibly ‘delayed’ resolution pathways may suggest a defective resolution program in patients with highly active MS.', {'entities': [[131, 133, 'ASSAY']]}], ['[1]–[3] Early interventional MS trials supplementing ω-3 and ω-6 were based on socio-epidemiological studies demonstrating that the risk of MS is high in countries with a high intake of saturated fatty acids and low in countries with a high intake of poly-unsaturated fatty acids (PUFAs).', {'entities': [[29, 31, 'ASSAY'], [140, 142, 'ASSAY']]}], ['[5] Elevated levels of PGE2 were detected in EAE lesions and in the cerebrospinal fluid (CSF) of MS patients.', {'entities': [[97, 99, 'ASSAY']]}], ['[2].However, the role of these molecules in MS pathology remains elusive.', {'entities': [[44, 46, 'ASSAY']]}], ['We therefore used LC-MS mass spectrometry to investigate the synthesis of the ω-3 and ω-6-derived resolution agonists in CSF and in serum samples of MS patients with highly and less active disease.', {'entities': [[18, 23, 'ASSAY'], [149, 151, 'ASSAY']]}], ['Go to:MethodsPatients20 selected patients with a diagnosis of relapsing-remitting MS or clinically isolated syndrome according to the revised McDonald criteria from 2011 [9] were age- and sex-matched for highly or less active disease severity and enrolled in this cohort study.', {'entities': [[82, 84, 'ASSAY']]}], ['According to the German Multiple Sclerosis Consensus Group [MSTKG] severe disease is considered with ≥6 lesions in the initial MRI or a severe clinical presentation at disease onset defined by an Expanded Disability Status Scale (EDSS) of ≥3 (because patients with a clinically isolated syndrome and EDSS ≥3 had a high probability to develop definite MS).', {'entities': [[60, 62, 'ASSAY'], [351, 353, 'ASSAY']]}], ['Patients with a sum score ≤6 were classified as having less active MS and patients with >6 points as highly active MS.', {'entities': [[67, 69, 'ASSAY'], [115, 117, 'ASSAY']]}], ['Lipid autacoids were extracted by solid phase using SampliQ ODS-C18 cartridges (Agilent Technologies, Santa Clara, CA).', {'entities': [[80, 87, 'ASSAY']]}], ['Endogenous levels of the following lipid mediators were identified and quantified with LC/MS/MS-based lipidomics (figure 1):', {'entities': [[87, 92, 'ASSAY']]}], ['In brief, we analyzed extracted samples by a triple-quadrupole linear ion trap LC/MS/MS system (MDS SCIEX 3200 QTRAP) equipped with a Kinetex C18 mini-bore column.', {'entities': [[79, 84, 'ASSAY'], [100, 105, 'ASSAY']]}], ['MS/MS analyses were carried out in negative ion mode and prominent fatty acid metabolites were quantified by multiple reaction monitoring (MRM mode) using established transitions (table S1 and www.lipidmaps.org).', {'entities': [[0, 5, 'ASSAY']]}], ['MS/MS analyses using enhanced product ion mode with appropriate selection of the parent ion in quadrupole 1.Open in a separate windowFigure 1LC/MS/MS Lipid Mediator Profiling.', {'entities': [[0, 5, 'ASSAY'], [141, 146, 'ASSAY']]}], ['Representative MRM chromatogram of 15-HETE, PGE2 and RvD1 from patients with less active MS (black line) and with highly active MS (red line).', {'entities': [[89, 91, 'ASSAY'], [128, 130, 'ASSAY']]}], ['Structures were confirmed for selected samples by MS/MS analyses using enhanced product ion mode with appropriate selection of the parent ion in quadrupole 1.Statistical AnalysisThe serum and CSF levels of lipid mediators (mean ± SEM) were compared between the two groups.', {'entities': [[50, 55, 'ASSAY']]}], ['The age (mean ± SEM) in the ‘less active’ MS group (31.4 years ±2.5) was not statistically different from the ‘highly active’ group (34.0±2.3).', {'entities': [[42, 44, 'ASSAY']]}], ['Patients with highly active MS revealed higher (p = 0.0019; Mann-Whitney, two-tailed) numbers of CSF cells (40.2/3±8.6/3 per µl) compared to the less active group (9.4/3±1.97/3 per µl).', {'entities': [[28, 30, 'ASSAY']]}], ['Furthermore, the number of MS lesions in MRI was about three times higher (p = 0.0015) in the ‘highly active’ group (12.8±3.8) than in the ‘less active’ group (', {'entities': [[27, 29, 'ASSAY']]}], ['3.4±0.67).Mass spectrometry analysis of serum and CSF samples was performed blinded and revealed significant differences in lipid mediator levels of both AA and DHA derived lipid mediators (figure 2).', {'entities': [[10, 27, 'ASSAY']]}], ['In the active MS group the intermediate, AA-derived molecule 15-hydroxyeicosatetraenoic acid (15-HETE) reached significantly higher levels (', {'entities': [[14, 16, 'ASSAY']]}], ['‘less active’ MS.Released AA is converted into prostaglandins, leukotrienes, thromboxane, lipoxins, and hydroxy-eicosatetraenoic acids (HETEs), collectively termed eicosanoids.', {'entities': [[14, 16, 'ASSAY']]}], ['(A) Mass spectrometry detection in lumbar CSF suggests that the differences in the brain are very robust as the molecules are produced locally and become diluted during CSF flow.', {'entities': [[4, 21, 'ASSAY']]}], ['PGE2, 15-HETE and the proresolution agonist RvD1 were significantly increased in patients with highly active MS, and NPD1 was only detectable in this group.', {'entities': [[109, 111, 'ASSAY']]}], ['Synopsis of lipid mediator synthesis pathways and disease severity-dependent differences in CSF of MS patients.', {'entities': [[99, 101, 'ASSAY']]}], ['The levels of the precursors AA and DHA correlated closely, suggesting parallel regulation of the lipid mediator metabolism, but they did not correlate with active MS.', {'entities': [[164, 166, 'ASSAY']]}], ['In contrast, increased CSF levels of PGE2, 15-HETE and RvD1 correlated with active MS.', {'entities': [[83, 85, 'ASSAY']]}], ['Thus, lipid mediator production in active MS is not regulated by the release of precursor molecules but rather likely by the activation/expression of targeted enzymatic oxidation pathways as a specific disease response or component of disease progression.', {'entities': [[42, 44, 'ASSAY']]}], ['In contrast, the DHA-derived resolvin D1 (RvD1) was increased in CSF of patients with highly active MS (0.89 pg/ml ±0.41 vs. 0.12 pg/ml ±0.05, P = 0.014).', {'entities': [[100, 102, 'ASSAY']]}], ['Also, the DHA-derived neuroprotectin D1 (NPD1) reached detectable levels only in highly active MS patients.', {'entities': [[95, 97, 'ASSAY']]}], ['we report that patients with ‘highly active’ and ‘less active’ MS not only had a different CSF expression of pro-inflammatory mediators, but also differed in their synthesis of proresolution lipid mediators.', {'entities': [[63, 65, 'ASSAY']]}], ['We were able to corroborate the elevated PGE2 and 15-HETE CSF levels that were recently reported in MS patients as a quantitative CSF marker of neuroinflammation and oxidative stress.', {'entities': [[100, 102, 'ASSAY']]}], ['This finding supports the well-established concept of compartmentalized effects in MS, as seen for several biomarkers [', {'entities': [[83, 85, 'ASSAY']]}], ['[17] Another biological explanation would be that in the ‘highly active’ MS cohort increased CSF cell numbers would represent a higher synthesis capacity of the lipid mediator resolution agonists.', {'entities': [[73, 75, 'ASSAY']]}], ['In contrast to RvD1, the AA-derived LXA4 was not significantly upregulated in the CSF of ‘highly active’ MS patients, suggesting a differentially regulated resolution program defining a ω-3 PUFA versus a ω-6 PUFA hierarchy in response to an inflammatory stimulus within a spatio-temporal resolutive context.', {'entities': [[105, 107, 'ASSAY']]}], ['Furthermore, since endogenous LXA4 synthesis is limited despite sufficient substrate (15-HETE) which is in line with a defective “lipid mediator class switch” [18], it might be worthwhile analyzing whether aspirin-triggered epimeric LXA4 levels can also be boosted in the CSF of MS patients, as has been demonstrated in the serum of healthy subjects taking low-dose aspirin.', {'entities': [[279, 281, 'ASSAY']]}], ['[7] Disruption of 12/15-lipoxygenase (which produces LXA4) results in a break of immunological self-tolerance [19], which is impaired in MS.', {'entities': [[137, 139, 'ASSAY']]}], ['The clinical outcome of MS patients might not be determined solely by the levels of pro-inflammatory mediators, which have conceptually dominated diagnostic and interventional approaches in past, but also by a milieu of resolution agonists at the lesion site.', {'entities': [[24, 26, 'ASSAY']]}], ['PMC4368747PMID: 25793887Validation of a Rapid and Sensitive LC-MS/MS Method for Determination of Exemestane and Its Metabolites, 17β-Hydroxyexemestane and 17β-', {'entities': [[60, 68, 'ASSAY']]}], ['(LC-MS/MS) method was developed and validated for the evaluation of exemestane pharmacokinetics and its metabolites, 17β-dihydroexemestane (active metabolite) and 17β-', {'entities': [[1, 9, 'ASSAY']]}], ['Chromatographic separation of analytes was achieved using Thermo Fisher BDS Hypersil C18 analytic HPLC column (100 × 2.1 mm, 5 μm).', {'entities': [[58, 71, 'ASSAY']]}], ['The column effluents were detected by API 4000 triple quadrupole mass spectrometer using electrospray ionisation (ESI) and monitored by multiple reaction monitoring (MRM) in positive mode.', {'entities': [[65, 82, 'ASSAY'], [114, 117, 'ASSAY']]}], ['[18] and the other is a LC-MS/MS method with LLOQ of 0.2 ng/ml for Exe and 0.1 ng/ml for 17DhExe.[19]', {'entities': [[24, 32, 'ASSAY']]}], ['Compared to HPLC-UV method, the LC-MSMS method demonstrated superior sensitivity and specificity which are critical to accurately quantify analytes of interest.', {'entities': [[32, 37, 'ASSAY'], [37, 39, 'ASSAY']]}], ['In order to support a clinical trial of Exe in breast cancer patients in Singapore National University Hospital (NUH), a simple, novel and sensitive LC-MS/MS method was developed for determination of plasma concentrations of these three compounds simultaneously.', {'entities': [[149, 157, 'ASSAY']]}], ['ConditionsLC-MS/MS analysis was performed using an API 4000 triple quadrupole mass spectrometer (Applied Biosystems, MDS SCIEX, Ontario, Canada).', {'entities': [[10, 18, 'ASSAY'], [78, 95, 'ASSAY'], [121, 126, 'ASSAY']]}], ['The HPLC system was composed of Agilent 1100 series gradient pump, degasser, autosampler and column oven (Agilent Technologies, Germany).', {'entities': [[32, 39, 'ASSAY'], [106, 113, 'ASSAY']]}], ['The data was acquired and processed using Analyst software (Version 1.4.2, Applied Biosystems, MDS SCIEX, Ontario, Canada).The chromatographic separation of analytes from endogenous compounds was performed on a BDS Hypersil C18 column (100 × 2.1 mm, 5 μm, Thermo Fisher Scientific, Waltham, Massachusetts, United States), which was preceded by a SecurityGuard cartridge (4.0 × 3.0 mm, Phenomenex, CA, United States).', {'entities': [[99, 104, 'ASSAY'], [256, 269, 'ASSAY']]}], ['C.The mass spectrometer was equipped with an electrospray ionisation source and it was operated in the positive ion detection mode in the multiple reaction monitoring (MRM).Preparation of Stock Solutions, Calibration and Quality Control StandardsStock solutions of Exe, Exe-d3, 17DhExe, 17DhExe-d3, Exe17Oglu and Exe17Oglu-d3', {'entities': [[6, 23, 'ASSAY']]}], ['A volume of 30 μL was injected per run for quantitative analysis by LC-MS/MS.Construction of Standard Calibration CurveThe standard calibration curves were constructed using six concentrations.', {'entities': [[68, 76, 'ASSAY']]}], ['to:Results and DiscussionLC-MS/MS optimizationMass spectrometric parameters were optimised and the product ion mass spectra of each analyte under optimised conditions were as follows.', {'entities': [[25, 33, 'ASSAY']]}], ['A total of three reversed-phase HPLC columns were investigated for chromatographic separation of analytes from endogeneous interferences and among analytes.', {'entities': [[17, 36, 'ASSAY']]}], ['#Expressed as percentage of the standard deviation divided by the meanApplication of Validated LC-MS/MS MethodThe developed method for the simultaneous quantification of Exe and its metabolites, 17DhExe, and Exe17Oglu was successfully applied in a pharmacokinetics/dynamics study, in which all the plasma samples taken from one typical patient were quantified for the concentrations of Exe, 17DhExe, and Exe17Oglu over 24 hours after exemestane ingestion at dose of 25 mg.', {'entities': [[95, 103, 'ASSAY']]}], ['ConclusionsA simple, novel and specific LC-MS/MS assay for the quantification of Exe and', {'entities': [[40, 48, 'ASSAY']]}], ['(NMRC/CSA/021/2010 and NMRC/NIG/MH95:03/1-36/2012).', {'entities': [[1, 4, 'ASSAY'], [23, 26, 'ASSAY']]}], ['Breda M, Pianezzola E, Benedetti MS.', {'entities': [[33, 35, 'ASSAY']]}], ['A liquid chromatography-tandem mass spectrometry method for the simultaneous determination of exemestane and its metabolite 17-dihydroexemestane in human plasma.', {'entities': [[24, 48, 'ASSAY']]}], ['Breda M, Pianezzola E, Benedetti MSJ Chromatogr.', {'entities': [[33, 35, 'ASSAY']]}], ['[Ref list]A liquid chromatography-tandem mass spectrometry method for the simultaneous determination of exemestane and its metabolite 17-dihydroexemestane in human plasma.', {'entities': [[34, 58, 'ASSAY']]}], [': PMC3929651PMID: 24586615Serum 1H-NMR Metabolomic Fingerprints of Acute-On-Chronic Liver Failure in Intensive Care Unit Patients with Alcoholic CirrhosisRoland Amathieu,# 1 , 2 , * Mohamed N. Triba,# 2 Pierre Nahon, 3 Nadia Bouchemal, 2 Walid Kamoun, 1 Hakim Haouache, 1 Jean-Claude Trinchet, 3 Philippe Savarin, 2 Laurence Le Moyec, 4 and Gilles Dhonneur 1', {'entities': [[32, 38, 'ASSAY']]}], ['Our study assessed the metabolomic profile of serum using proton nuclear magnetic resonance (1H-NMR) spectroscopy to identify metabolic changes related to acute-on-chronic liver failure.', {'entities': [[93, 99, 'ASSAY']]}], ['Blood samples were drawn at admission, and sera were separated and stored at –80°C until 1H-NMR spectral analysis.', {'entities': [[89, 95, 'ASSAY']]}], ['ConclusionA serum metabolite fingerprint for acute-on-chronic liver failure, obtained with 1H-NMR, was identified.', {'entities': [[91, 97, 'ASSAY']]}], ['1H-NMR) spectroscopy, when applied to liver disease, has shown a close relationship between metabolic abnormalities and the severity of the disease in sera and tissues [6], [7], [8].Jimenez et', {'entities': [[0, 6, 'ASSAY']]}], ['al. have shown that using 1H-NMR to assess the metabolomics of sera from patients with cirrhosis could discriminate between patients with minimal hepatic encephalopathy and those with no encephalopathy [9].', {'entities': [[26, 32, 'ASSAY']]}], ['Another study that assessed sera using 1H-NMR showed the potential of the metabolomic profile to evaluate patients with fulminant hepatic failure and to predict an unfavorable outcome [14].', {'entities': [[39, 45, 'ASSAY']]}], ['However, none of these studies have focused on the changes in metabolism that occur in patients with cirrhosis and ACLF.We hypothesize that patients with cirrhosis and hospitalized in ICUs have acute changes to their metabolism that can be identified using 1H-NMR to evaluate their sera.', {'entities': [[257, 263, 'ASSAY']]}], ['Thus, the purpose of our study was to assess whether the metabolomic profiles of sera, obtained by 1H-NMR spectroscopy, were modified in patients with alcohol-induced cirrhosis and hospitalized in an ICU for an acute event known to cause ACLF, when compared to patients with compensated or decompensated, but stable, cirrhosis.', {'entities': [[102, 118, 'ASSAY']]}], ['1H-NMR SpectroscopyFor NMR analysis, samples were thawed at room temperature.', {'entities': [[0, 6, 'ASSAY'], [23, 26, 'ASSAY']]}], ['The proton spectra were acquired at 500 MHz on a Varian Unity Inova', {'entities': [[49, 55, 'ASSAY']]}], ['The Carr-Purcell-Meiboom-Gill (CPMG) sequence is frequently used by others [9] to suppress the broad protein signal according to its short T2 relaxation time.', {'entities': [[4, 29, 'ASSAY'], [31, 35, 'ASSAY']]}], ['Associations were considered to be statistically significant at two-tailed α-values of 0.05.Multivariate AnalysesA principal component analysis (PCA) was first performed to detect any group separation based on NMR-signal variability.', {'entities': [[210, 213, 'ASSAY']]}], ['Each point in the score-plot represents the projection of an NMR spectrum (and thus a patient’s sample) on the predictive (horizontal axis) and the first orthogonal component of the model (vertical axis).', {'entities': [[61, 64, 'ASSAY']]}], ['For each NMR variable, distribution of the regression coefficient was also estimated.', {'entities': [[9, 12, 'ASSAY']]}], ['The buckets are labeled according to metabolite assignment, as presented in Table 3.Table 3Discriminant metabolites observed by 1H-NMR spectroscopy.', {'entities': [[131, 147, 'ASSAY']]}], ['The chemical shift and multiplicity correspond to those found in the 1H-NMR spectra of the patients’ sera.$ppm:', {'entities': [[69, 75, 'ASSAY']]}], ['In the 1H-NMR spectra of sera, the lipids were detected as broad resonances of fatty-acid methyl and methylene moieties at 0.8 and 1.24 ppm, respectively, and the N-trimethyl moiety of choline was included in phospholipids at 3.2 ppm.', {'entities': [[7, 13, 'ASSAY']]}], ['Dunn WB, Broadhurst DI, Atherton HJ, Goodacre R, Griffin JL (', {'entities': [[20, 22, 'ASSAY']]}], ['Metabolomic approach by 1H NMR spectroscopy of serum for the assessment of chronic liver failure in patients with cirrhosis.', {'entities': [[27, 43, 'ASSAY']]}], ['Metabolic profile of chronic liver disease by NMR spectroscopy of human biopsies.', {'entities': [[46, 62, 'ASSAY']]}], ['Comparison of the metabolic profiling of hepatitis B virus-infected cirrhosis and alcoholic cirrhosis patients by using (1) H NMR-based metabonomics.', {'entities': [[126, 129, 'ASSAY']]}], ['(1)H NMR-based serum metabolic profiling in compensated and decompensated cirrhosis.', {'entities': [[5, 8, 'ASSAY']]}], ['1H NMR spectroscopic study of blood serum for the assessment of liver function in liver transplant patients.', {'entities': [[0, 6, 'ASSAY']]}], ['1H NMR spectroscopy for the prediction of therapeutic outcome in patients with fulminant hepatic failure.', {'entities': [[3, 19, 'ASSAY']]}], ['NMR in biomedicine 19: 521–526.', {'entities': [[0, 3, 'ASSAY']]}], ['Dellinger RP, Levy MM, Carlet JM, Bion J, Parker MM, et al.', {'entities': [[10, 12, 'ASSAY']]}], ['NMR spectroscopy of human blood plasma.', {'entities': [[0, 16, 'ASSAY']]}], ['1994) High-resolution 1H-NMR spectroscopy of blood plasma for metabolic studies.', {'entities': [[25, 41, 'ASSAY']]}], ['Identification of serum proton NMR metabolomic fingerprints associated with hepatocellular carcinoma in patients with alcoholic cirrhosis.', {'entities': [[31, 34, 'ASSAY']]}], ['Statistical total correlation spectroscopy: an exploratory approach for latent biomarker identification from metabolic 1H NMR data sets.', {'entities': [[119, 125, 'ASSAY']]}], ['Sen S, Davies NA, Mookerjee RP, Cheshire LM, Hodges SJ, et al.', {'entities': [[28, 30, 'ASSAY']]}], ['Dunn WB, Broadhurst DI, Atherton HJ, Goodacre R,', {'entities': [[20, 22, 'ASSAY']]}], ['[Ref list]Metabolomic approach by 1H NMR spectroscopy of serum for the assessment of chronic liver failure in patients with cirrhosis.', {'entities': [[37, 53, 'ASSAY']]}], ['[Ref list]Metabolic profile of chronic liver disease by NMR spectroscopy of human biopsies.', {'entities': [[56, 72, 'ASSAY']]}], ['Martínez-Granados B, Morales JM, Rodrigo JM, Del Olmo J, Serra MA, Ferrández A, Celda B, Monleón DInt J Mol Med.', {'entities': [[97, 99, 'ASSAY']]}], ['[Ref list]Comparison of the metabolic profiling of hepatitis B virus-infected cirrhosis and alcoholic cirrhosis patients by using (1) H NMR-based metabonomics.', {'entities': [[136, 139, 'ASSAY']]}], ['[Ref list]¹H NMR-based serum metabolic profiling in compensated and decompensated cirrhosis.', {'entities': [[13, 16, 'ASSAY']]}], ['NMR spectroscopic study of blood serum for the assessment of liver function in liver transplant patients.', {'entities': [[0, 3, 'ASSAY']]}], ['Serkova NJ, Zhang Y, Coatney JL, Hunter L, Wachs ME, Niemann CU, Mandell MSTransplantation.', {'entities': [[73, 75, 'ASSAY']]}], ['NMR spectroscopy for the prediction of therapeutic outcome in patients with fulminant hepatic failure.', {'entities': [[0, 16, 'ASSAY']]}], ['Saxena V, Gupta A, Nagana Gowda GA, Saxena R, Yachha SK, Khetrapal CLNMR', {'entities': [[69, 72, 'ASSAY']]}], ['2008.Dellinger RP, Levy MM, Carlet JM, Bion J, Parker MM, Jaeschke R, Reinhart K, Angus DC, Brun-Buisson C, Beale R, Calandra T, Dhainaut JF, Gerlach H, Harvey M, Marini JJ, Marshall J, Ranieri M, Ramsay G, Sevransky J, Thompson BT, Townsend S, Vender JS, Zimmerman JL, Vincent JL, International Surviving Sepsis Campaign Guidelines Committee.', {'entities': [[15, 17, 'ASSAY']]}], ['NMR spectroscopy of human blood plasma.', {'entities': [[0, 16, 'ASSAY']]}], ['[Ref list]High-resolution 1H-NMR spectroscopy of blood plasma for metabolic studies.', {'entities': [[29, 45, 'ASSAY']]}], ['[Ref list]Identification of serum proton NMR metabolomic fingerprints associated with hepatocellular carcinoma in patients with alcoholic cirrhosis.', {'entities': [[41, 44, 'ASSAY']]}], ['[Ref list]Statistical total correlation spectroscopy: an exploratory approach for latent biomarker identification from metabolic 1H NMR data sets.', {'entities': [[129, 135, 'ASSAY']]}], ['Sen S, Davies NA, Mookerjee RP, Cheshire LM, Hodges SJ, Williams R, Jalan RLiver Transpl.', {'entities': [[28, 30, 'ASSAY']]}], ['Conceived and designed the experiments: RP PE JAD KV BM.', {'entities': [[40, 42, 'ASSAY']]}], ['Performed the experiments: RP PE HL BM.', {'entities': [[27, 29, 'ASSAY']]}], ['RP PE HL JAD CMC DK PA KV BM.', {'entities': [[0, 2, 'ASSAY']]}], ['RP PE HL JAD CMC DK PA KV BM.', {'entities': [[0, 2, 'ASSAY']]}], ['Serum levels and 24h urinary excretion of p-cresyl sulfate, p-cresyl glucuronide, indoxyl sulfate, trimethylamine N-oxide and phenylacetylglutamine were determined at each time point using liquid chromatography—tandem mass spectrometry.', {'entities': [[211, 235, 'ASSAY']]}], ['After enrollment (performed by RP), patients were randomized by the study nurse, blinded from both the investigator and study participant.', {'entities': [[31, 33, 'ASSAY']]}], ['Serum levels of PCS, PCG, IS, PAG and TMAO were quantified using a dedicated ultra-performance liquid chromatography—tandem mass spectrometry (UPLC—MS/MS)', {'entities': [[117, 141, 'ASSAY'], [148, 153, 'ASSAY']]}], ['Xevo TQS, Waters, Zellik, Belgium) method.', {'entities': [[10, 16, 'ASSAY']]}], ['IS-d4, TMAO-d9 and PAG-d5) and 150 μl acetonitrile were thoroughly mixed in 96-well Ostro plates (Waters, Zellik, Belgium).', {'entities': [[98, 104, 'ASSAY']]}], ['Supernatants were collected, dried and after dilution with 1000 μl MQ water, injected on the UPLC-MS/MS system.', {'entities': [[95, 103, 'ASSAY']]}], ['Chromatographic separation was performed on an Acquity CSHFluoroPhenyl column (Waters, Zellik, Belgium).', {'entities': [[79, 85, 'ASSAY']]}], ['file.(114K, docx)Go to:AcknowledgmentsRP is the recipient of a Ph.D. fellowship of the Research Foundation—Flanders (FWO)', {'entities': [[38, 40, 'ASSAY']]}], ['Funding StatementRP is the recipient of a Ph.D. fellowship of the Research Foundation - Flanders (FWO)', {'entities': [[17, 19, 'ASSAY']]}], ['Salmean YA, Segal MS, Palii SP, Dahl WJ.', {'entities': [[18, 20, 'ASSAY']]}], ['Wu IW, Hsu KH, Lee CC, Sun CY, Hsu HJ, Tsai CJ, Tzen CY, Wang YC, Lin CY, Wu MSNephrol Dial Transplant.', {'entities': [[77, 79, 'ASSAY']]}], ['Powe NRPLoS', {'entities': [[6, 8, 'ASSAY']]}], ['Salmean YA, Segal MS, Palii SP, Dahl WJJ Ren Nutr.', {'entities': [[18, 20, 'ASSAY']]}], ['PMC3655955PMID: 23696887Urinary Metabolic Biomarker and Pathway Study of Hepatitis B Virus Infected Patients Based on UPLC-MS SystemAihua Zhang, Hui Sun, Ying Han, Guangli Yan, and Xijun Wang', {'entities': [[118, 125, 'ASSAY']]}], ['(UPLC-Q-TOF-HDMS) to determine metabolite alterations in HBV patients.', {'entities': [[8, 11, 'ASSAY'], [14, 16, 'ASSAY']]}], ['This paper was designed to investigate a comprehensive metabolome of patients HBV-induced cirrhosis by UPLC-Q-TOF-HDMS combined with pattern recognition methods to identify urine biomarkers for HBV, explore the diagnostic possibilities, and enhance the understanding of its mechanisms.', {'entities': [[110, 113, 'ASSAY'], [116, 118, 'ASSAY']]}], ['Fractions (50 mL) of the urine supernatants were then stored at − 80°C until UPLC-Q-TOF-HDMS analysis.', {'entities': [[84, 87, 'ASSAY'], [90, 92, 'ASSAY']]}], ['Thawed urine samples were collected after centrifugation at 13,000 rpm for 10 minutes at 4°C, and the supernatant was then filtered through a syringe filter (0.22 µm), 5 µL of the supernatant were injected into the UPLC-Q-TOF-HDMS.Metabolic profiling Chromatographic condition', {'entities': [[222, 225, 'ASSAY'], [228, 230, 'ASSAY']]}], ['The human urine metabolite profiling was performed with a Waters ACQUITY UPLC system (Waters Corp., Milford, USA) coupled with TOF-MS.', {'entities': [[58, 64, 'ASSAY'], [86, 92, 'ASSAY'], [127, 130, 'ASSAY'], [131, 133, 'ASSAY']]}], ['A Waters Acquity UPLC BEH C18 (2.1 mm i.d. ×100 mm ACQUITY) column packed with 1.7 mm beads was used to separate the molecules in the biofluids set.', {'entities': [[2, 8, 'ASSAY']]}], ['TOF-MS condition', {'entities': [[0, 3, 'ASSAY'], [4, 6, 'ASSAY']]}], ['MS system was operated using the ESI+ and ESI− mode and the mass range was set at 100–1000 m', {'entities': [[0, 2, 'ASSAY'], [33, 36, 'ASSAY'], [42, 45, 'ASSAY']]}], ['Leucine enkaphalin was used as the reference compound (positive ion mode ([M+H]+ = 556.2771) and [M−H]− = 554.2615) at a concentration of 0.2 ng/mL under a flow rate of 100 µl·min−1, to ensure accuracy and reproducibility during the MS analysis.', {'entities': [[233, 235, 'ASSAY']]}], ['Multivariate data analysis Multivariate data was analyzed by EZinfo software (Waters Corp., Milford, USA).', {'entities': [[78, 84, 'ASSAY']]}], ['Biomarkers identificationMassFragment™ application manager (Waters Corp., Milford, USA) was used to facilitate the MS/MS fragment ion analysis process by way of chemically intelligent peak-matching algorithms.', {'entities': [[60, 66, 'ASSAY'], [115, 120, 'ASSAY']]}], ['MS based metabolomic approach was carried out to differentiate HBV patients from controls.', {'entities': [[0, 2, 'ASSAY']]}], [', UPLC-HDMS provided the retention time, precise molecular mass and MS/MS data for the identification of biomarkers.', {'entities': [[9, 11, 'ASSAY'], [68, 73, 'ASSAY']]}], ['Panel D shows the combination of S- and VIP-score plots constructed from the supervised OPLS analysis of urine (ESI+ mode).', {'entities': [[112, 115, 'ASSAY']]}], [\"Panel D shows the combination of S- and VIP-score plots constructed from the supervised OPLS analysis of urine (ESI− mode).Open in a separate windowFigure 3Heatmap visualization and hierarchical clustering analysis with MetaboAnalyst's data annotation tools constructed based on the differential metabolites of importance for the urine of HBV (A).Rows: samples;\", {'entities': [[112, 115, 'ASSAY']]}], ['At the defined operational conditions, all of the metabolites in a total ion chromatogram were extracted and aligned using the EZinfo software 2.0 (Waters Corp., Milford, USA), 11 discriminant metabolites were identified (table 1).', {'entities': [[148, 154, 'ASSAY']]}], ['In our study, an UPLC–TOF/MS-based urine metabolomics approach coupled with multivariate statistical methods provide a powerful approach to clearly differentiate patients with HBV from matched control subjects and identify the potential biomarkers.', {'entities': [[22, 25, 'ASSAY'], [26, 28, 'ASSAY']]}], ['Exploratory urinary metabolic biomarkers and pathways using UPLC-Q-TOF-HDMS coupled with pattern recognition approach.', {'entities': [[67, 70, 'ASSAY'], [73, 75, 'ASSAY']]}], ['Walsh BH, Broadhurst DI, Mandal R, Wishart DS, Boylan GB, et al.', {'entities': [[21, 23, 'ASSAY']]}], ['Potential drug targets on insomnia and intervention effects of Jujuboside A through metabolic pathway analysis as revealed by UPLC/ESI-SYNAPT-HDMS coupled with pattern recognition approach.', {'entities': [[131, 134, 'ASSAY'], [144, 146, 'ASSAY']]}], ['Wedemeyer H, Yurdaydìn C, Dalekos GN, Erhardt A, Çakaloğlu Y, Değertekin H, Gürel S, Zeuzem S, Zachou K, Bozkaya H, Koch A, Bock T, Dienes HP, Manns MP, HIDIT Study Group.', {'entities': [[155, 157, 'ASSAY']]}], ['Zheng D, Zhang S, Zhou X, Wang C, Xiang P, Zheng Q, Xu JRPLoS', {'entities': [[56, 58, 'ASSAY']]}], ['[Ref list]Exploratory urinary metabolic biomarkers and pathways using UPLC-Q-TOF-HDMS coupled with pattern recognition approach.', {'entities': [[77, 80, 'ASSAY'], [83, 85, 'ASSAY']]}], ['[Ref list]Potential drug targets on insomnia and intervention effects of Jujuboside A through metabolic pathway analysis as revealed by UPLC/ESI-SYNAPT-HDMS coupled with pattern recognition approach.', {'entities': [[141, 144, 'ASSAY'], [154, 156, 'ASSAY']]}], ['/L; p value 0.008) and lower HDL-C (1.16 ± 0.34 vs. 1.35 ± 0.54 mmol/L; p value 0.04), reflected in 1H NMR serum analysis that showed differences in the baseline lipid content between the two groups.', {'entities': [[100, 106, 'ASSAY']]}], ['Participants were recruited for a 12 month period from Dec 2012 to Dec 2013.Aspartame (100mg, equivalent to that consumed in a can of diet soda and less than the Acceptable Daily Intake maximum (ADI) recommended by FDA of 40mg/kg body weight/day) was incorporated into cold pressed cereal bars (Campden BRI, United Kingdom).', {'entities': [[196, 198, 'ASSAY']]}], ['Serum CRP was measured by the high-sensitivity method on a Beckman DXC analyser.', {'entities': [[7, 9, 'ASSAY']]}], ['Plasma L-Tyr and L-Phe were measured simultaneously by LC-MS.', {'entities': [[55, 60, 'ASSAY']]}], ['Beck Depression Inventory (BDI); Hospital Anxiety and Depression Scale (HADS); Perceived Stress Severity Scale (PSS); life events in the past 12 months (SRRI); Toronto Alexithymia Scale (TAS-20); Whiteley-7 Somatisation Index; State Trait Anxiety Inventory (STAI).', {'entities': [[28, 30, 'ASSAY']]}], ['Metabonomic analysisMetabonomic analysis of serum and urine utilised high throughput nuclear magnetic resonance (NMR) spectroscopy, and liquid chromatography coupled to mass spectrometry (UPLC-MS)', {'entities': [[113, 116, 'ASSAY'], [188, 195, 'ASSAY']]}], ['NMR data AcquisitionA Bruker Avance III', {'entities': [[0, 3, 'ASSAY'], [22, 28, 'ASSAY']]}], ['600 MHz spectrometer (Bruker Analytische GmbH, Rheinstetten, Germany), employing an inverse detection probe of 5 mm width with z-gradients, at 300 K was used.', {'entities': [[22, 28, 'ASSAY']]}], ['1D NOESY pulse sequence with presaturation (25 Hz) for water signal suppression collecting 128 scans in 64-k data points over a spectral width of 12019.23 Hz, using a relaxation delay of 2 s, an acquisition time of 6.83 s and a mixing time of 0.01 s. Serum samples: Carr-Purcell-Meiboom-Gill (CPMG) utilising a T2 relaxation time filter to suppress the broad lipoprotein signals and presaturation (25 Hz) for water signal suppression, collecting 128 scans in 64-k data points over a spectral width of 7194.2 Hz, using a total delay of 6.55 s between pulse cycles.', {'entities': [[266, 291, 'ASSAY'], [293, 297, 'ASSAY']]}], ['NMR data were visually inspected, imported in Matlab, phased, baseline corrected and referenced to TSP (urine) and glucose (serum) using in-house routines.', {'entities': [[0, 3, 'ASSAY']]}], ['Analysis was blinded to aspartame consumption and subject group (AS or NS).LCMS Lipidomics', {'entities': [[77, 79, 'ASSAY'], [80, 90, 'ASSAY']]}], ['acquisitionA subset of samples from 12 randomly picked individuals from each coded group, were analysed for lipids in +ESI and -ESI.', {'entities': [[119, 122, 'ASSAY'], [128, 131, 'ASSAY']]}], ['XCMS peak picking [29]', {'entities': [[2, 4, 'ASSAY']]}], ['(BDI)Depression (HADS)Perceived Stress SeverityLife Events12mFeelings', {'entities': [[2, 4, 'ASSAY']]}], ['studiesThe 1H NMR analysis indicated a metabolic difference in the blood serum samples attributed to variation in the serum lipid content between the two groups (AS vs NS).', {'entities': [[11, 17, 'ASSAY']]}], ['3A and 3B, where the OPLS-DA scores plot along with the respective loadings plot in the form of a pseudo-NMR spectrum depicts these findings.', {'entities': [[105, 108, 'ASSAY']]}], ['Confirmation of the serum 1H NMR data was obtained using UPLC-MS lipidomics on a sample subset chosen randomly from each group (n = 12 per group).', {'entities': [[26, 32, 'ASSAY'], [57, 64, 'ASSAY']]}], ['NMR human serum baseline (pre-bar consumption) data from sensitive and control participants (A), and female and male participants (C).3A. OPLS-DA scores plot demonstrating that metabolic variation is dominated by inter-group (sensitive versus control) variation; 3B. OPLS-DA regression coefficient loadings plot (in the form of pseudo-NMR spectrum) of sensitive vs.', {'entities': [[0, 3, 'ASSAY'], [335, 338, 'ASSAY']]}], ['The signal orientation indicates a higher concentration of characteristic lipid metabolites in the sensitive population; 3C. OPLS-DA scores plot demonstrating clear separation of male (red symbols) from female (black symbols) subjects; 3D. OPLS-DA regression coefficient loadings plot (in the form of pseudo-NMR spectrum) male vs female confirming discriminatory peaks of interest identified by analysis of the raw spectra.', {'entities': [[308, 311, 'ASSAY']]}], ['For the loadings plots in B and D, the color-coded scale reflects the significance of the correlation of each metabolite with the separation (red indicates highest significance and blue indicates no significance).No significant differences were found between the baseline samples from the AS and NS individuals using 1H NMR of the urine, nor were any significant differences found in the samples from the AS and NS individuals before or after aspartame or control bars.', {'entities': [[317, 323, 'ASSAY']]}], ['(2005) Evaluation of the orthogonal projection on latent structure model limitations caused by chemical shift variability and improved visualization of biomarker changes in 1H NMR spectroscopic metabonomic studies.', {'entities': [[173, 179, 'ASSAY']]}], ['XCMS:', {'entities': [[2, 4, 'ASSAY']]}], ['[Ref list]Evaluation of the orthogonal projection on latent structure model limitations caused by chemical shift variability and improved visualization of biomarker changes in 1H NMR spectroscopic metabonomic studies.', {'entities': [[176, 182, 'ASSAY']]}], ['[Ref list]XCMS: processing mass spectrometry data for metabolite profiling using nonlinear peak alignment, matching, and identification.', {'entities': [[12, 14, 'ASSAY']]}], ['Genetic Testing Registry GenBank Reference Sequences Gene Expression Omnibus Genome Data Viewer Human Genome Mouse', {'entities': []}], ['Genome Influenza Virus Primer-BLAST Sequence Read Archive NCBI Information About NCBI Research at NCBI NCBI News & Blog NCBI', {'entities': []}], ['FTP Site NCBI on Facebook NCBI on Twitter NCBI on YouTube Privacy Policy External link.', {'entities': []}], ['Please review our privacy policy.', {'entities': []}], ['NLM NIH DHHS USA.gov National Center for Biotechnology Information, U.S. National Library of Medicine 8600', {'entities': []}], ['Rockville Pike, Bethesda MD, 20894 USA Policies and Guidelines | Contact PreferencesTurn offExternal link.', {'entities': []}], ['Please review our privacy policy.', {'entities': []}], ['External link.', {'entities': []}], ['Please review our privacy policy.', {'entities': []}], ['External link.', {'entities': []}], ['Please review our privacy policy.', {'entities': []}], ['External link.', {'entities': []}], ['Please review our privacy policy.', {'entities': []}], ['External link.', {'entities': []}], ['Please review our privacy policy.', {'entities': []}], ['External link.', {'entities': []}], ['Please review our privacy policy.', {'entities': []}], ['External link.', {'entities': []}], ['Please review our privacy policy.', {'entities': []}], ['External link.', {'entities': []}], ['Please review our privacy policy.', {'entities': []}], ['External link.', {'entities': []}], ['Please review our privacy policy.', {'entities': []}], ['External link.', {'entities': []}], ['Please review our privacy policy.', {'entities': []}], ['External link.', {'entities': []}], ['Please review our privacy policy.', {'entities': []}], ['External link.', {'entities': []}], ['Please review our privacy policy.', {'entities': []}], ['External link.', {'entities': []}], ['Please review our privacy policy.', {'entities': []}], ['External link.', {'entities': []}], ['Please review our privacy policy.', {'entities': []}], ['External link.', {'entities': []}], ['Please review our privacy policy.', {'entities': []}], ['External link.', {'entities': []}], ['Please review our privacy policy.', {'entities': []}], ['External link.', {'entities': []}], ['Please review our privacy policy.', {'entities': []}], ['External link.', {'entities': []}], ['Please review our privacy policy.', {'entities': []}], ['External link.', {'entities': []}], ['Please review our privacy policy.', {'entities': []}], ['External link.', {'entities': []}], ['Please review our privacy policy.', {'entities': []}], ['External link.', {'entities': []}], ['Please review our privacy policy.', {'entities': []}], ['External link.', {'entities': []}], ['Please review our privacy policy.', {'entities': []}], ['External link.', {'entities': []}], ['Please review our privacy policy.', {'entities': []}], ['External link.', {'entities': []}], ['Please review our privacy policy.', {'entities': []}]]\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "BVCd3WFTjkNY"
      },
      "source": [
        ""
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "WU3M-xglZIx5",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 991
        },
        "outputId": "2f27d20f-b383-4c15-b8a3-8b74077e6f47"
      },
      "source": [
        "#6\n",
        "# Traning additional entity type using Spacy\n",
        "# Because such a traning would need a lot of formating data, it was done using \n",
        "# a only one example, the NMR assay. Bigger and formated data set would be needed for a better model\n",
        "#!/usr/bin/env python\n",
        "# coding: utf8\n",
        "\"\"\"Example of training an additional entity type\n",
        "\n",
        "This script shows how to add a new entity type to an existing pretrained NER\n",
        "model. To keep the example short and simple, only four sentences are provided\n",
        "as examples. In practice, you'll need many more — a few hundred would be a\n",
        "good start. You will also likely need to mix in examples of other entity\n",
        "types, which might be obtained by running the entity recognizer over unlabelled\n",
        "sentences, and adding their annotations to the training set.\n",
        "\n",
        "The actual training is performed by looping over the examples, and calling\n",
        "`nlp.entity.update()`. The `update()` method steps through the words of the\n",
        "input. At each word, it makes a prediction. It then consults the annotations\n",
        "provided on the GoldParse instance, to see whether it was right. If it was\n",
        "wrong, it adjusts its weights so that the correct action will score higher\n",
        "next time.\n",
        "\n",
        "After training your model, you can save it to a directory. We recommend\n",
        "wrapping models as Python packages, for ease of deployment.\n",
        "\n",
        "For more details, see the documentation:\n",
        "* Training: https://spacy.io/usage/training\n",
        "* NER: https://spacy.io/usage/linguistic-features#named-entities\n",
        "\n",
        "Compatible with: spaCy v2.1.0+\n",
        "Last tested with: v2.2.4\n",
        "\"\"\"\n",
        "from __future__ import unicode_literals, print_function\n",
        "\n",
        "import plac\n",
        "import random\n",
        "import warnings\n",
        "from pathlib import Path\n",
        "import spacy\n",
        "from spacy.util import minibatch, compounding\n",
        "from spacy import displacy\n",
        "\n",
        "# new entity label\n",
        "LABEL = \"ASSAY\"\n",
        "\n",
        "# training data\n",
        "# Note: If you're using an existing model, make sure to mix in examples of\n",
        "# other entity types that spaCy correctly recognized before. Otherwise, your\n",
        "# model might learn the new type, but \"forget\" what it previously knew.\n",
        "# https://explosion.ai/blog/pseudo-rehearsal-catastrophic-forgetting\n",
        "\n",
        "# Train_data= [ (\"tekst1\",{\"entities\":[(start_char,end_char, LABEL)]},) , (\"tekst2\",{\"entities\":[(start_char,end_char, LABEL)]},) , ... ]\n",
        "# In Word, do a search/replace, replacing all spaces with paragraph breaks (^p). Select all the text and then copy/paste it into Excel.\n",
        "\n",
        "TRAIN_DATA = data_train  # data train is the one from the opened json file\n",
        "              \n",
        "#en_core_web_sm\n",
        "@plac.annotations(\n",
        "    model=(\"Model name. Defaults to blank 'en' model.\", \"option\", \"m\", str),\n",
        "    new_model_name=(\"New model name for model meta.\", \"option\", \"nm\", str),\n",
        "    output_dir=(\"Optional output directory\", \"option\", \"o\", Path),\n",
        "    n_iter=(\"Number of training iterations\", \"option\", \"n\", int),\n",
        ")\n",
        "def main(model=\"en_core_web_sm\", new_model_name=\"ASSAY\", output_dir=None, n_iter=35):\n",
        "    \"\"\"Set up the pipeline and entity recognizer, and train the new entity.\"\"\"\n",
        "    random.seed(0)\n",
        "    if model is not None:\n",
        "        nlp = spacy.load(model)  # load existing spaCy model\n",
        "        print(\"Loaded model '%s'\" % model)\n",
        "    else:\n",
        "        nlp = spacy.blank(\"en\")  # create blank Language class\n",
        "        print(\"Created blank 'en' model\")\n",
        "    # Add entity recognizer to model if it's not in the pipeline\n",
        "    # nlp.create_pipe works for built-ins that are registered with spaCy\n",
        "    if \"ner\" not in nlp.pipe_names:\n",
        "        ner = nlp.create_pipe(\"ner\")\n",
        "        nlp.add_pipe(ner)\n",
        "    # otherwise, get it, so we can add labels to it\n",
        "    else:\n",
        "        ner = nlp.get_pipe(\"ner\")\n",
        "\n",
        "    ner.add_label(LABEL)  # add new entity label to entity recognizer\n",
        "    # Adding extraneous labels shouldn't mess anything up\n",
        "    \n",
        "    if model is None:\n",
        "        optimizer = nlp.begin_training()\n",
        "    else:\n",
        "        optimizer = nlp.resume_training()\n",
        "    move_names = list(ner.move_names)\n",
        "    \n",
        "    # get names of other pipes to disable them during training\n",
        "    pipe_exceptions = [\"ner\", \"trf_wordpiecer\", \"trf_tok2vec\"]\n",
        "    other_pipes = [pipe for pipe in nlp.pipe_names if pipe not in pipe_exceptions]\n",
        "    # only train NER\n",
        "    with nlp.disable_pipes(*other_pipes), warnings.catch_warnings():\n",
        "        # show warnings for misaligned entity spans once\n",
        "        warnings.filterwarnings(\"once\", category=UserWarning, module='spacy')\n",
        "\n",
        "        sizes = compounding(1.0, 4.0, 1.001)\n",
        "        # batch up the examples using spaCy's minibatch\n",
        "        for itn in range(n_iter):\n",
        "            random.shuffle(TRAIN_DATA)\n",
        "            batches = minibatch(TRAIN_DATA, size=sizes)\n",
        "            losses = {}\n",
        "            for batch in batches:\n",
        "                texts, annotations = zip(*batch)\n",
        "                nlp.update(texts, annotations, sgd=optimizer, drop=0.35, losses=losses)\n",
        "            print(\"Losses\", losses)\n",
        "\n",
        "    # test the trained model\n",
        "    test_text = ('NMR Metabolomics is used to determine the metabolic profile of biological samples, identify specific biomarkers,' \n",
        "       'and explore possible metabolic pathways. It has been used during drug development [1], '\n",
        "       'and in clinical disease research [2, 3], pathology [4], toxicology [5] and nutrition studies [6].' \n",
        "       'Metabolomics mainly utilizes NMR spectroscopy [7], liquid chromatography (LC)–mass spectrometry [8] and ' \n",
        "       'gas chromatography–mass spectrometry [9] to analyze and evaluate biological specimens. '\n",
        "       'Each analytical technique has its own advantages and shortcomings. none of them can be used individually' \n",
        "       'to systematically and accurately identify metabolites in complex biological matrices. '\n",
        "       'Since accurate metabolite identification directly determines the usefulness of the metabolomic analysis,' \n",
        "       'metabolite identification has gained increased attention from the metabolomics research community. '\n",
        "       '1H NMR spectroscopy is often used for metabolomics research. As all 1H nucleuses have the same sensitivity,' \n",
        "       'the reproducibility of NMR spectroscopy is typically high.')  \n",
        "    doc = nlp(test_text)\n",
        "    print(\"Entities in '%s'\" % test_text)\n",
        "    #ents = [ent for ent in list(doc.ents) if ent.label_ == 'ASSAY'] # trying to disable other entities but not working\n",
        "    for ent in doc.ents:        \n",
        "          print(ent.label_, ent.text)\n",
        "    displacy.render(doc, style=\"ent\",jupyter=True)\n",
        "    # save model to output directory\n",
        "    if output_dir is not None:\n",
        "        output_dir = Path(output_dir)\n",
        "        if not output_dir.exists():\n",
        "            output_dir.mkdir()\n",
        "        nlp.meta[\"name\"] = new_model_name  # rename model\n",
        "        nlp.to_disk(output_dir)\n",
        "        print(\"Saved model to\", output_dir)\n",
        "\n",
        "        # test the saved model\n",
        "        print(\"Loading from\", output_dir)\n",
        "        nlp2 = spacy.load(output_dir)\n",
        "        # Check the classes have loaded back consistently\n",
        "        assert nlp2.get_pipe(\"ner\").move_names == move_names\n",
        "        doc2 = nlp2(test_text)\n",
        "        for ent in doc2.ents:\n",
        "            print(ent.label_, ent.text)\n",
        "\n",
        "if __name__ == \"__main__\":\n",
        "    x=main()\n",
        "    "
      ],
      "execution_count": 63,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Loaded model 'en_core_web_sm'\n",
            "Losses {'ner': 9407.517137901705}\n",
            "Losses {'ner': 8713.402289576357}\n",
            "Losses {'ner': 8242.427278756033}\n",
            "Losses {'ner': 8192.706165627867}\n",
            "Losses {'ner': 8020.404950931668}\n",
            "Losses {'ner': 8008.3627757281065}\n",
            "Losses {'ner': 8011.043619979304}\n",
            "Losses {'ner': 7894.566299546277}\n",
            "Losses {'ner': 7839.690645961091}\n",
            "Losses {'ner': 7930.880325779319}\n",
            "Losses {'ner': 7899.402653617552}\n",
            "Losses {'ner': 7817.054569201544}\n",
            "Losses {'ner': 7819.733863943897}\n",
            "Losses {'ner': 7866.980376549065}\n",
            "Losses {'ner': 7776.290685646236}\n",
            "Losses {'ner': 7753.479310523719}\n",
            "Losses {'ner': 7571.179981552908}\n",
            "Losses {'ner': 7870.194521646481}\n",
            "Losses {'ner': 7737.129519006936}\n",
            "Losses {'ner': 7631.79074918438}\n",
            "Losses {'ner': 7570.854939341545}\n",
            "Losses {'ner': 7807.016919463174}\n",
            "Losses {'ner': 7835.121870201256}\n",
            "Losses {'ner': 7725.883611432335}\n",
            "Losses {'ner': 7775.05771268066}\n",
            "Losses {'ner': 7796.743461849401}\n",
            "Losses {'ner': 7608.068307725116}\n",
            "Losses {'ner': 7800.594547480345}\n",
            "Losses {'ner': 7795.206630493514}\n",
            "Losses {'ner': 7617.337821315974}\n",
            "Losses {'ner': 7767.633104834706}\n",
            "Losses {'ner': 7597.534856164886}\n",
            "Losses {'ner': 7767.296204882208}\n",
            "Losses {'ner': 7582.043026044965}\n",
            "Losses {'ner': 7727.78337183781}\n",
            "Entities in 'NMR Metabolomics is used to determine the metabolic profile of biological samples, identify specific biomarkers,and explore possible metabolic pathways. It has been used during drug development [1], and in clinical disease research [2, 3], pathology [4], toxicology [5] and nutrition studies [6].Metabolomics mainly utilizes NMR spectroscopy [7], liquid chromatography (LC)–mass spectrometry [8] and gas chromatography–mass spectrometry [9] to analyze and evaluate biological specimens. Each analytical technique has its own advantages and shortcomings. none of them can be used individuallyto systematically and accurately identify metabolites in complex biological matrices. Since accurate metabolite identification directly determines the usefulness of the metabolomic analysis,metabolite identification has gained increased attention from the metabolomics research community. 1H NMR spectroscopy is often used for metabolomics research. As all 1H nucleuses have the same sensitivity,the reproducibility of NMR spectroscopy is typically high.'\n",
            "ASSAY NMR\n",
            "ASSAY NMR spectroscopy\n",
            "ASSAY NMR spectroscopy\n",
            "ASSAY NMR spectroscopy\n"
          ],
          "name": "stdout"
        },
        {
          "output_type": "display_data",
          "data": {
            "text/html": [
              "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
              "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
              "    NMR\n",
              "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; text-transform: uppercase; vertical-align: middle; margin-left: 0.5rem\">ASSAY</span>\n",
              "</mark>\n",
              " Metabolomics is used to determine the metabolic profile of biological samples, identify specific biomarkers,and explore possible metabolic pathways. It has been used during drug development [1], and in clinical disease research [2, 3], pathology [4], toxicology [5] and nutrition studies [6].Metabolomics mainly utilizes \n",
              "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
              "    NMR spectroscopy\n",
              "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; text-transform: uppercase; vertical-align: middle; margin-left: 0.5rem\">ASSAY</span>\n",
              "</mark>\n",
              " [7], liquid chromatography (LC)–mass spectrometry [8] and gas chromatography–mass spectrometry [9] to analyze and evaluate biological specimens. Each analytical technique has its own advantages and shortcomings. none of them can be used individuallyto systematically and accurately identify metabolites in complex biological matrices. Since accurate metabolite identification directly determines the usefulness of the metabolomic analysis,metabolite identification has gained increased attention from the metabolomics research community. 1H \n",
              "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
              "    NMR spectroscopy\n",
              "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; text-transform: uppercase; vertical-align: middle; margin-left: 0.5rem\">ASSAY</span>\n",
              "</mark>\n",
              " is often used for metabolomics research. As all 1H nucleuses have the same sensitivity,the reproducibility of \n",
              "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
              "    NMR spectroscopy\n",
              "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; text-transform: uppercase; vertical-align: middle; margin-left: 0.5rem\">ASSAY</span>\n",
              "</mark>\n",
              " is typically high.</div></span>"
            ],
            "text/plain": [
              "<IPython.core.display.HTML object>"
            ]
          },
          "metadata": {
            "tags": []
          }
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "t_JFxj49a1VN"
      },
      "source": [
        "\n",
        "# -----------------------------------------------\n",
        "# Make text into sentences the hard way but\n",
        "# Computationally LESS expensive than Spacy \n",
        "# -----------------------------------------------\n",
        "# \n",
        "# -*- coding: utf-8 -*-\n",
        "import re\n",
        "alphabets= \"([A-Za-z])\"\n",
        "prefixes = \"(Mr|St|Mrs|Ms|Dr|Prof)[.]\"\n",
        "suffixes = \"(Inc|Ltd|Jr|Sr|Co)\"\n",
        "starters = \"(Mr|Mrs|Ms|Dr|He\\s|She\\s|It\\s|They\\s|Their\\s|Our\\s|We\\s|But\\s|However\\s|That\\s|This\\s|Wherever)\"\n",
        "acronyms = \"([A-Z][.][A-Z][.](?:[A-Z][.])?)\"\n",
        "websites = \"[.](com|net|org|io|gov|me|ai|edu)\"\n",
        "\n",
        "def split_into_sentences(text):\n",
        "    text = \" \" + text + \"  \"\n",
        "    text = text.replace(\"\\n\",\" \")\n",
        "    text = re.sub(prefixes,\"\\\\1<prd>\",text)\n",
        "    text = re.sub(websites,\"<prd>\\\\1\",text)\n",
        "    if \"Ph.D\" in text: text = text.replace(\"Ph.D.\",\"Ph<prd>D<prd>\")\n",
        "    text = re.sub(\"\\s\" + alphabets + \"[.] \",\" \\\\1<prd> \",text)\n",
        "    text = re.sub(acronyms+\" \"+starters,\"\\\\1<stop> \\\\2\",text)\n",
        "    text = re.sub(alphabets + \"[.]\" + alphabets + \"[.]\" + alphabets + \"[.]\",\"\\\\1<prd>\\\\2<prd>\\\\3<prd>\",text)\n",
        "    text = re.sub(alphabets + \"[.]\" + alphabets + \"[.]\",\"\\\\1<prd>\\\\2<prd>\",text)\n",
        "    text = re.sub(\" \"+suffixes+\"[.] \"+starters,\" \\\\1<stop> \\\\2\",text)\n",
        "    text = re.sub(\" \"+suffixes+\"[.]\",\" \\\\1<prd>\",text)\n",
        "    text = re.sub(\" \" + alphabets + \"[.]\",\" \\\\1<prd>\",text)\n",
        "    if \"”\" in text: text = text.replace(\".”\",\"”.\")\n",
        "    if \"\\\"\" in text: text = text.replace(\".\\\"\",\"\\\".\")\n",
        "    if \"!\" in text: text = text.replace(\"!\\\"\",\"\\\"!\")\n",
        "    if \"?\" in text: text = text.replace(\"?\\\"\",\"\\\"?\")\n",
        "    text = text.replace(\".\",\".<stop>\")\n",
        "    text = text.replace(\"?\",\"?<stop>\")\n",
        "    text = text.replace(\"!\",\"!<stop>\")\n",
        "    text = text.replace(\"<prd>\",\".\")\n",
        "    sentences = text.split(\"<stop>\")\n",
        "    sentences = sentences[:-1]\n",
        "    sentences = [s.strip() for s in sentences]\n",
        "    return sentences\n",
        "\n"
      ],
      "execution_count": 24,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 108
        },
        "id": "9lcAzj5Xoxk_",
        "outputId": "9721d58a-e950-4d5e-cbdf-9b2e88f32bea"
      },
      "source": [
        "#2\n",
        "import os\n",
        "from bs4 import BeautifulSoup\n",
        "#import urllib2\n",
        "content=[]\n",
        "train=[]\n",
        "#url = \"/content/drive/My Drive/Imperial college London/data/PMC3908470.html\"\n",
        "path = r\"/content/drive/My Drive/Imperial college London/data\"\n",
        "for filename in os.listdir(path):\n",
        "    if filename.endswith(\".json\"):\n",
        "       fullpath = os.path.join(path, filename)\n",
        "       with open(fullpath, \"r\") as read_file:   # read json file \n",
        "        data_train = json.load(read_file) \n",
        "        temp=data_train.get(\"paragraphs\")\n",
        "        for i in range(0,len(temp)):        #temp is list of dictiionaries under the key \"Paragraphs\"\n",
        "          if (temp[i].get(\"section_heading\")==\"Materials and Methods\"):  # PMC specific \n",
        "            train.append(temp[i].get(\"body\")) #train is a list of all body sections\n",
        "\n",
        "#print(data_train,type(data_train))\n",
        "print((train))  \n",
        "'''\n",
        "# Turn to STRING and add to section 4\n",
        "'''\n",
        "qw=split_into_sentences('NMR Metabolomics is used to determine the metabolic profile of biological samples, identify specific biomarkers,' \n",
        "       'and explore possible metabolic pathways. It has been used during drug development [1], '\n",
        "       'and in clinical disease research [2, 3], pathology [4], toxicology [5] and nutrition studies [6].' \n",
        "       'Metabolomics mainly utilizes NMR spectroscopy [7], liquid chromatography (LC)–mass spectrometry [8] and ' \n",
        "       'gas chromatography–mass spectrometry [9] to analyze and evaluate biological specimens. '\n",
        "       'Each analytical technique has its own advantages and shortcomings. none of them can be used individually' \n",
        "       'to systematically and accurately identify metabolites in complex biological matrices. '\n",
        "       'Since accurate metabolite identification directly determines the usefulness of the metabolomic analysis,' \n",
        "       'metabolite identification has gained increased attention from the metabolomics research community. '\n",
        "       '1H NMR spectroscopy is often used for metabolomics research. As all 1H nucleuses have the same sensitivity,' \n",
        "       'the reproducibility of NMR spectroscopy is typically high.')\n",
        "'''\n",
        "#so huggin face proba\n",
        "#!pip install spacy-transformers\n",
        "#!python3 -m spacy download en_trf_bertbaseuncased_lg\n",
        "import en_trf_bertbaseuncased_lg\n",
        "import spacy\n",
        "nlp = en_trf_bertbaseuncased_lg.load()\n",
        "doc = nlp(\"Apple shares rose on the news. Apple pie is delicious.\")\n",
        "\n",
        "print(doc[0].similarity(doc[7]))\n",
        "print(doc._.trf_last_hidden_state.shape)\n",
        "'''\n",
        "#print(type(qw))"
      ],
      "execution_count": 76,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "['The DHS is a family-based study examining risk for macrovascular and other complications in T2D. Briefly, the DHS includes siblings concordant for T2D but without advanced renal insufficiency. When possible, unaffected siblings were also recruited. T2D was clinically defined as diabetes developing after the age of 35 years and initially treated with oral agents and/or diet and exercise, in the absence of historical evidence of ketoacidosis. Diagnoses were confirmed by measurement of fasting blood glucose and glycosylated hemoglobin (HbA1c). Extensive measurements of cardiovascular disease (CVD) risk factors were obtained during baseline exams, which occurred from 1998 to 2006. Ascertainment and recruitment have been previously described in detail [, ].', 'The DHS MIND is an ancillary study to the DHS that included a cognitive testing component and brain magnetic resonance imaging (MRI). The purpose was to investigate the relationships between cognitive function, brain imaging, and vascular disease in T2D. Participants from the original DHS investigation were re-examined on average 6.7 ± 1.6 years after their initial visit. Participant examinations were conducted in the General Clinical Research Center of the Wake Forest Baptist Medical Center. Study protocols were approved by the Institutional Review Board at Wake Forest School of Medicine and all study procedures were carried out in accordance with the Declaration of Helsinki. All participants provided written informed consent before participation.', \"Total serum AGEs were measured using a competitive enzyme linked immunosorbant assay (ELISA) (Lifeome Biolabs; Oceanside, CA). The ELISA was run according to the manufacturer's recommendations using stored serum. The ELISA uses a monoclonal antibody that is specific to AGEs. This ELISA has a minimum detectable dose of AGEs less than 35.2 ng/ml. Intra-assay and inter-assay precision was 2.0% and 22.5%, respectively.\", 'Additional SNPs, predominately low-frequency and rare coding SNPs, were also captured by the Illumina HumanExome BeadChips v.1.0 (Illumina Inc., San Diego, CA) for which genotype data was available in the DHS []. For DHS exome chip data, genotype calling was completed using Genome Studio Software v1.9.4 (Illumina). Samples failing to meet a minimum acceptable call rate of 98% (n=3) were excluded from further analyses. An additional 58 samples were included as blind duplicates within the genotyping set to serve as QC samples; the concordance rate for blind duplicates was 99.9 ±0.0001% (mean± standard deviation (SD)). Exclusion criteria for SNP performance included call rate < 95% (n=972), SNPs with 5 or fewer observances (n=204,273) and Hardy-Weinberg Equilibrium (HWE) p-value <1×10_−6  (n=26); 41,961 SNPs were retained for analysis. Additional QC of exome chip data set was completed to exclude samples with poor quality genotype calls, gender errors, or unclear/unexpected sibling relationships.', 'Genomic DNA was purified from whole-blood samples obtained from subjects using the PUREGENE DNA isolation kit (Gentra Systems., Minneapolis, MN). DNA was quantitated using standardized fluorometric readings on a Hoefer DyNA Quant 200 fluorometer (Hoefer Pharmacia Biotech, Inc., San Francisco, CA).', 'A genome-wide association study (GWAS) was completed using the Affymetrix Genome-wide Human SNP Array 5.0 (Affymetrix, CA, USA) as reported []. Genotype calling was completed using the BRLLM-P algorithm in Genotyping Console v4.0 (Affymetrix). Samples failing to meet an intensity quality control (QC) threshold and those failing to meet a minimum acceptable call rate of 95% were excluded from further analyses (n=7). An additional 39 samples were included as blind duplicates within the genotyping set to serve as QC samples; the concordance rate for these blind duplicates was 99.0 ±0.72% (mean± SD). Exploratory analyses of genotype data were performed using PLINK v. 1.07 (http://pngu.mgh.harvard.edu/purcell/plink/) and samples with poor quality genotype calls, gender errors, or unclear/unexpected sibling relationships were excluded from further analysis. Exclusion criteria for SNP performance included call rate < 95% (n=11,085), HWE p-value < 1×10_−6  (n=332), and minor allele frequency (MAF) < 0.01 (n=57,382); 371,951 SNPs were retained for analysis.', 'Heritability estimates for AGEs were assessed in 506 related individuals from 245 families that had AGE measures. The AGE measurements were natural log transformed to approximate the normality assumptions of the analysis. To determine the contribution of genetic factors to AGE levels, the data in family members were analyzed using Sequential Oligogenic Linkage Analysis Routines (SOLAR) v6.3.4 (Texas Biomedical Research Institute, San Antonio, TX, USA). SOLAR performs a variance components analysis of family data where the total phenotypic variation is partitioned into genetic and nongenetic sources of variation. This approach has been used previously in the DHS [, ]. To minimize the bias associated with shared environmental factors, the estimates of heritability (h_2 ) were based on all available family data and were controlled for covariates related to AGEs including age, gender, BMI, eGFR, T2D affected status, and smoking status (current or former). Five models were developed that incorporated an increasing number of covariates to determine the extent that genetic factors contribute to variation in AGE levels independent of other confounding variables. The first model was unadjusted. The second model was adjusted for age and gender. The third model was adjusted for age, gender, and BMI. The fourth was adjusted for age, gender, BMI, and eGFR. The fifth model was the fully adjusted model and included age, gender, BMI, eGFR, T2D-affected status, and a history of smoking. The significance of the heritability estimates was obtained by likelihood ratio tests.', 'Genetic association was run using SOLAR v6.3.4 (Texas Biomedical Research Institute, San Antonio, TX) which uses variance component methods to account for family structure. AGE measures were natural log transformed prior to analysis to approximate conditional normality. Covariates used in the analysis were age, gender, BMI, T2D affected status, eGFR, and smoking history. All analyses were run using the additive model of inheritance. The Bonferroni corrected p - value for significance for the 121 candidate gene SNPs was set at 0.00041 (α=0.05/121). Similarly, the p - value for significance for the GWAS and exome chip associations was set at 1.21×10_−7  after correcting for multiple comparisons based upon the number of SNPs (413,912) investigated.']\n"
          ],
          "name": "stdout"
        },
        {
          "output_type": "execute_result",
          "data": {
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "string"
            },
            "text/plain": [
              "'\\n#so huggin face proba\\n#!pip install spacy-transformers\\n#!python3 -m spacy download en_trf_bertbaseuncased_lg\\nimport en_trf_bertbaseuncased_lg\\nimport spacy\\nnlp = en_trf_bertbaseuncased_lg.load()\\ndoc = nlp(\"Apple shares rose on the news. Apple pie is delicious.\")\\n\\nprint(doc[0].similarity(doc[7]))\\nprint(doc._.trf_last_hidden_state.shape)\\n'"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 76
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "0lh39HM1_YKK"
      },
      "source": [
        ""
      ],
      "execution_count": null,
      "outputs": []
    }
  ]
}